<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Topic</th>
      <th>Title</th>
      <th>Link</th>
      <th>EDAM topic</th>
      <th>EDAM operation</th>
      <th>Creation</th>
      <th>Last modification</th>
      <th>Version</th>
      <th>Tutorial</th>
      <th>Slides</th>
      <th>Video</th>
      <th>Workflows</th>
      <th>Tools</th>
      <th>Servers with precise tool versions</th>
      <th>Servers with tool but different versions</th>
      <th>Feedback number</th>
      <th>Feedback mean note</th>
      <th>Visitors</th>
      <th>Page views</th>
      <th>Visit duration</th>
      <th>Video views</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy on the Cloud</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/cloudbursting/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-01-31</td>
      <td>2024-12-19</td>
      <td>7</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314392</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Monitoring with Telegraf and Grafana</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/monitoring/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-31</td>
      <td>2024-12-18</td>
      <td>47</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2726093</td>
      <td>7319677</td>
      <td>3.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Docker and Galaxy</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/galaxy-docker/slides.html</td>
      <td></td>
      <td></td>
      <td>2016-09-29</td>
      <td>2022-10-18</td>
      <td>16</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314392</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Monitoring</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/general-monitoring/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-04-26</td>
      <td>2022-03-01</td>
      <td>5</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314392</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Distributed Object Storage</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/object-store/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2025-03-24</td>
      <td>28</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2724590</td>
      <td>7316490</td>
      <td>4.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Installation with Ansible</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/ansible-galaxy/tutorial.html</td>
      <td>Database management, Informatics, Data management</td>
      <td></td>
      <td>2019-01-28</td>
      <td>2025-04-08</td>
      <td>102</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2748440</td>
      <td>7361435</td>
      <td>4.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Running Jobs on Remote Resources with Pulsar</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/pulsar/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2025-04-08</td>
      <td>49</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2725727</td>
      <td>7318485</td>
      <td>5.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Reference Data with Data Managers</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/reference-genomes/tutorial.html</td>
      <td></td>
      <td>Sequence alignment, Generation, Genome indexing, Read mapping</td>
      <td>2019-01-28</td>
      <td>2023-04-21</td>
      <td>7</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>bwa_mem_index_builder_data_manager, bwa, data_manager_fetch_genome_all_fasta_dbkey</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723793</td>
      <td>7315094</td>
      <td>7.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galactic Database</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/database/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2022-06-16</td>
      <td>5</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314392</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>uWSGI</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/uwsgi/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2022-03-01</td>
      <td>9</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314392</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Troubleshooting</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/troubleshooting/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2024-06-25</td>
      <td>15</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314392</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Data Libraries</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/data-library/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-01-14</td>
      <td>2024-06-14</td>
      <td>32</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2706612</td>
      <td>7250780</td>
      <td>4.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Tool Management with Ephemeris</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/tool-management/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-27</td>
      <td>2025-04-03</td>
      <td>39</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2725005</td>
      <td>7316900</td>
      <td>5.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy from an administrator's point of view</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-12</td>
      <td>2024-06-25</td>
      <td>6</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314392</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Monitoring with gxadmin</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/gxadmin/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2024-08-08</td>
      <td>27</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2724111</td>
      <td>7315739</td>
      <td>5.116667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>External Authentication</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/external-auth/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2022-10-18</td>
      <td>14</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723946</td>
      <td>7315484</td>
      <td>3.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Setting up Celery Workers for Galaxy</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/celery/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-07-16</td>
      <td>2025-04-08</td>
      <td>12</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1868224</td>
      <td>4493811</td>
      <td>4.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Reference Data with CVMFS</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/cvmfs/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-27</td>
      <td>2025-04-03</td>
      <td>50</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2725312</td>
      <td>7317452</td>
      <td>5.700000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Connecting Galaxy to a compute cluster</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/connect-to-compute-cluster/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-01-07</td>
      <td>2025-04-08</td>
      <td>59</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2727212</td>
      <td>7321259</td>
      <td>5.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Gearing towards production</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/production/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2025-04-08</td>
      <td>10</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314392</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Server Maintenance: Cleanup, Backup, and Restoration</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/backup-cleanup/tutorial.html</td>
      <td>Database management, Informatics, Data management</td>
      <td></td>
      <td>2019-01-31</td>
      <td>2024-08-09</td>
      <td>5</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723790</td>
      <td>7315107</td>
      <td>11.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Server: Other</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/stuff/slides.html</td>
      <td></td>
      <td></td>
      <td>2018-01-07</td>
      <td>2023-02-17</td>
      <td>6</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314392</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Administrator Time Burden and Technology Usage</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/poll-ssa/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-07-10</td>
      <td>2024-07-10</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>710794</td>
      <td>1718477</td>
      <td>2.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>User, Role, Group, Quota, and Authentication managment</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/users-groups-quotas/slides.html</td>
      <td></td>
      <td></td>
      <td>2018-01-07</td>
      <td>2024-08-08</td>
      <td>19</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314392</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Advanced customisation of a Galaxy instance</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/advanced-galaxy-customisation/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-27</td>
      <td>2023-04-17</td>
      <td>18</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314392</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Empathy</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/empathy/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-01-31</td>
      <td>2021-04-06</td>
      <td>4</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314392</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Interactive Tools</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/interactive-tools/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-02-27</td>
      <td>2025-04-08</td>
      <td>27</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2725490</td>
      <td>7317769</td>
      <td>5.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Ansible</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/ansible/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-07-11</td>
      <td>2024-08-08</td>
      <td>48</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2729579</td>
      <td>7325743</td>
      <td>4.183333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Controlling Galaxy with systemd or Supervisor</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/systemd-supervisor/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-01-28</td>
      <td>2023-04-16</td>
      <td>5</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314392</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>How I learned to stop worrying and love the systemd</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/stop-worrying-love-systemd/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-07-06</td>
      <td>2023-09-28</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>667</td>
      <td>769</td>
      <td>0.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Customizing the look of Galaxy (Manual)</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/customization-manual/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-04-16</td>
      <td>2025-04-08</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>181</td>
      <td>219</td>
      <td>2.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Performant Uploads with TUS</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/tus/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-03-01</td>
      <td>2024-06-14</td>
      <td>17</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>829</td>
      <td>1221</td>
      <td>5.283333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Reference Data with CVMFS without Ansible</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/cvmfs-manual/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-06-17</td>
      <td>2023-11-23</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>374</td>
      <td>509</td>
      <td>2.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Deploying a Beacon v1 in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/beacon/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-04-16</td>
      <td>2024-12-18</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>136</td>
      <td>154</td>
      <td>2.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Monitoring Galaxy and Pulsar with Sentry</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/sentry/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-04-19</td>
      <td>2024-03-18</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>204</td>
      <td>249</td>
      <td>1.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Upgrading Galaxy</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/upgrading/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-06-08</td>
      <td>2024-11-25</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>667</td>
      <td>1035</td>
      <td>2.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Enable upload via FTP</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/ftp/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-06-18</td>
      <td>2024-02-20</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1211</td>
      <td>1672</td>
      <td>1.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Adding file-sources to Galaxy</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/file_sources/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-01-20</td>
      <td>2025-01-20</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>42</td>
      <td>58</td>
      <td>3.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Database schema</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/database-schema/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2016-07-20</td>
      <td>2025-04-08</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2999</td>
      <td>4923</td>
      <td>1.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Monitoring with Reports</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/reports/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-31</td>
      <td>2025-04-08</td>
      <td>43</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>484</td>
      <td>648</td>
      <td>2.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Training Infrastructure as a Service (TIaaS)</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/tiaas/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-02-12</td>
      <td>2024-12-18</td>
      <td>38</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>527</td>
      <td>708</td>
      <td>3.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Automation with Jenkins</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/jenkins/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-08-06</td>
      <td>2022-10-18</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>8207</td>
      <td>16262</td>
      <td>0.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Configuring the Onedata connectors (remotes, Object Store, BYOS, BYOD)</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/onedata-configuration/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-24</td>
      <td>2025-04-08</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>31</td>
      <td>36</td>
      <td>5.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Use Apptainer containers for running Galaxy jobs</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/apptainer/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-01-08</td>
      <td>2025-04-08</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1007</td>
      <td>1282</td>
      <td>2.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Customizing the look of Galaxy</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/customization/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-04-16</td>
      <td>2025-04-08</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>548</td>
      <td>676</td>
      <td>4.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Mapping Jobs to Destinations using TPV</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/job-destinations/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-01-17</td>
      <td>2024-06-14</td>
      <td>36</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1620</td>
      <td>2551</td>
      <td>3.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Alternative Celery Deployment for Galaxy</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/celeryless/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-11-07</td>
      <td>2024-11-07</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>45</td>
      <td>50</td>
      <td>4.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Create a subdomain for your community on UseGalaxy.eu</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/subdomain/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-11-27</td>
      <td>2024-12-13</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>21</td>
      <td>26</td>
      <td>6.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy Installation on Kubernetes</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/k8s-deploying-galaxy/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-07-02</td>
      <td>2024-06-30</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1585</td>
      <td>2630</td>
      <td>2.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Managing Galaxy on Kubernetes</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/k8s-managing-galaxy/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-07-02</td>
      <td>2024-12-19</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>652</td>
      <td>962</td>
      <td>2.183333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Galaxy usage on SURF Research Cloud</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/surf-research-cloud-galaxy/tutorial.html</td>
      <td>Informatics, Data management</td>
      <td></td>
      <td>2024-12-18</td>
      <td>2025-08-05</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>62</td>
      <td>81</td>
      <td>2.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Pulsar usage on SURF Research Cloud</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/surf-research-cloud-pulsar/tutorial.html</td>
      <td>Informatics, Data management</td>
      <td></td>
      <td>2024-12-18</td>
      <td>2024-12-19</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>46</td>
      <td>54</td>
      <td>0.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Deploying Tailscale/Headscale for private mesh networking</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/wireguard-headscale/tutorial.html</td>
      <td>Data security, Software engineering</td>
      <td></td>
      <td>2022-09-21</td>
      <td>2024-12-19</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2328</td>
      <td>3011</td>
      <td>0.450000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Deploying Wireguard for private mesh networking</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/wireguard/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-21</td>
      <td>2024-12-19</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1091</td>
      <td>1245</td>
      <td>0.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Server administration</td>
      <td>Deploying a compute cluster in OpenStack via Terraform</td>
      <td>https://training.galaxyproject.org//topics/admin/tutorials/terraform/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-10-27</td>
      <td>2024-12-19</td>
      <td>15</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2739063</td>
      <td>7344142</td>
      <td>3.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Image analysis using Deep Learning</td>
      <td>BioImage Model Zoo: Advanced models in one-click</td>
      <td>https://training.galaxyproject.org//topics/ai4life/tutorials/getting-started/slides.html</td>
      <td></td>
      <td></td>
      <td>2023-04-18</td>
      <td>2023-05-24</td>
      <td>4</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1421756</td>
      <td>3342015</td>
      <td>2.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Image analysis using Deep Learning</td>
      <td>Contribute to the BioImage.IO models</td>
      <td>https://training.galaxyproject.org//topics/ai4life/tutorials/contribute-materials/slides.html</td>
      <td></td>
      <td></td>
      <td>2023-05-18</td>
      <td>2023-05-24</td>
      <td>3</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1360539</td>
      <td>3206243</td>
      <td>2.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Image analysis using Deep Learning</td>
      <td>Welcome to the BioImage Model Zoo</td>
      <td>https://training.galaxyproject.org//topics/ai4life/tutorials/welcome-bioimageio/slides.html</td>
      <td></td>
      <td></td>
      <td>2023-05-11</td>
      <td>2023-05-18</td>
      <td>3</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1374832</td>
      <td>3238619</td>
      <td>2.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>An Introduction to Genome Assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/general-introduction/tutorial.html</td>
      <td></td>
      <td>Statistical calculation, Formatting, Sequence composition calculation, De-novo assembly, Sequencing quality control, Visualisation, Validation, Sequence assembly validation</td>
      <td>2017-05-23</td>
      <td>2024-09-12</td>
      <td>29</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>quast, fastq_paired_end_interlacer, fastqc, multiqc, velveth, velvetg</td>
      <td>Galaxy@AuBi, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2757784</td>
      <td>7371314</td>
      <td>5.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Large genome assembly and polishing</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/largegenome/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity</td>
      <td>Variant calling, Box-Whisker plot plotting, Sequence contamination filtering, Sequence assembly, De-novo assembly, Pairwise sequence alignment, Validation, Sequence assembly validation, Scatter plot plotting, Sequence assembly visualisation, Genome assembly, Statistical calculation, Cross-assembly, Mapping assembly, Sequence composition calculation, Sequencing quality control, Visualisation, Base-calling</td>
      <td>2023-03-06</td>
      <td>2025-06-04</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>fastp, barchart_gnuplot, quast, busco, fastqc, multiqc, fasta-stats, genomescope, minimap2, medaka_consensus_pipeline, nanoplot, meryl, bandage_image, flye, racon</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>6776</td>
      <td>8846</td>
      <td>2.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Decontamination of a genome assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/assembly-decontamination/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity</td>
      <td>Data handling, Taxonomic classification</td>
      <td>2024-09-04</td>
      <td>2025-06-04</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Cut1, gfastats, ncbi_blastn_wrapper, parse_mito_blast, kraken2, tp_cat, tp_sed_tool, ncbi_dustmasker_wrapper, Filter1</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1417</td>
      <td>1769</td>
      <td>2.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of a bacterial genome (MRSA) sequenced using Illumina MiSeq Data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-illumina/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology</td>
      <td>Sequence assembly visualisation, Genome assembly, Statistical calculation, Sequence contamination filtering, Sequence composition calculation, Sequencing quality control, Read mapping, Visualisation, Sequence assembly validation</td>
      <td>2021-03-24</td>
      <td>2025-03-20</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>fastp, quast, fastqc, shovill, upload1, bandage_info, bandage_image, falco</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>9642</td>
      <td>14954</td>
      <td>3.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Data handling, Read pre-processing, Variant calling, Primer removal, Sequence contamination filtering, Sequence assembly validation, Sequence alignment analysis, Sequence assembly visualisation, Genome assembly, Statistical calculation, Sequence file editing, Formatting, Sequence composition calculation, Sequencing quality control, Read mapping, Visualisation, Sequence trimming, Local alignment</td>
      <td>2025-09-16</td>
      <td>2025-09-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>quast, filter_tabular, bamtools, fastqc, seqtk_subseq, cutadapt, collection_column_join, megahit, coverm_contig, metaspades, bowtie2, ngsutils_bam_filter, bandage_info, tp_cat, random_lines1, bandage_image, megahit_contig2fastg, bg_uniq</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>2</td>
      <td>2</td>
      <td>0.200000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Making sense of a newly assembled genome</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/ecoli_comparison/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Sequence visualisation, Read mapping, Genome visualisation, Sequence alignment, Mapping</td>
      <td>2018-06-14</td>
      <td>2024-06-24</td>
      <td>29</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>join1, Grep1, upload1, tp_sed_tool, datamash_ops, Cut1, jbrowse, circos, circos_aln_to_links, tp_replace_in_line, addValue, bedtools_sortbed, random_lines1, fasta_compute_length, bedtools_complementbed, Filter1, bedtools_intersectbed, tp_grep_tool, circos_interval_to_tiles, cat1, fasta_filter_by_length, tp_replace_in_column, lastz_wrapper_2, tp_cat, tp_sort_header_tool, collapse_dataset, mergeCols1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6655</td>
      <td>12902</td>
      <td>2.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>De Bruijn Graph Assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/debruijn-graph-assembly/tutorial.html</td>
      <td></td>
      <td>Genome assembly, Optimisation and refinement, Sequence assembly</td>
      <td>2017-05-24</td>
      <td>2024-02-01</td>
      <td>25</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>fasta-stats, spades, velvetoptimiser</td>
      <td>Galaxy@AuBi, UseGalaxy.be, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2729515</td>
      <td>7324326</td>
      <td>5.116667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>ERGA post-assembly QC</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/ERGA-post-assembly-QC/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity</td>
      <td>Sequence alignment analysis, Data handling, Sequence assembly visualisation, Data deposition, Genome assembly, Sequence trimming, Sequence assembly validation, Sequence alignment, De-novo assembly, Data parsing, Pairwise sequence alignment, Genotyping, Visualisation, k-mer counting, Phasing, Scaffolding</td>
      <td>2023-06-23</td>
      <td>2025-06-04</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>bg_diamond, meryl, bellerophon, gfastats, pretext_snapshot, CONVERTER_gz_to_uncompressed, genomescope, pretext_map, tp_find_and_replace, bandage_image, interactive_tool_blobtoolkit, merqury, collapse_dataset, busco, bwa_mem2, minimap2, smudgeplot, samtools_stats, blobtoolkit, __EXTRACT_DATASET__</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1792</td>
      <td>2300</td>
      <td>2.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of MRSA from Oxford Nanopore MinION data (and optionally Illumina data)</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-nanopore/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Antimicrobial resistance</td>
      <td>Box-Whisker plot plotting, Sequence contamination filtering, De-novo assembly, Sequence assembly validation, Filtering, Scatter plot plotting, Sequence assembly visualisation, Genome assembly, Statistical calculation, Sequencing error detection, Cross-assembly, Sequence alignment, Mapping assembly, Sequencing quality control, Sequence composition calculation, Read mapping, Visualisation</td>
      <td>2021-03-24</td>
      <td>2024-11-29</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>fastp, quast, filtlong, bwa_mem2, CONVERTER_bz2_to_uncompressed, fastqc, upload1, nanoplot, polypolish, bandage_image, porechop, flye</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>9993</td>
      <td>15024</td>
      <td>3.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Vertebrate genome assembly using HiFi, Bionano and Hi-C data - Step by Step</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/vgp_genome_assembly/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity</td>
      <td>Read pre-processing, Data handling, k-mer counting, Primer removal, Genome assembly, Sequence assembly validation, Sequence alignment, De-novo assembly, Visualisation, Pairwise sequence alignment, Sequence trimming, Phasing, Mapping, Scaffolding, Read binning</td>
      <td>2021-06-04</td>
      <td>2025-06-04</td>
      <td>57</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>quast, bedtools_bamtobed, hifiasm, purge_dups, Convert characters1, gfa_to_fa, yahs, sort1, bionano_scaffold, salsa, cutadapt, meryl, bellerophon, gfastats, tp_grep_tool, pretext_snapshot, cat1, genomescope, pretext_map, tp_find_and_replace, Add_a_column1, merqury, tp_cut_tool, collapse_dataset, busco, bwa_mem2, collection_column_join, minimap2</td>
      <td></td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>15256</td>
      <td>23193</td>
      <td>2.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly Quality Control</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/assembly-quality-control/tutorial.html</td>
      <td>Biodiversity, Sequence assembly, Data quality management</td>
      <td>De-novo assembly, Scaffolding, Visualisation, k-mer counting, Genome assembly, Phasing, Sequence assembly validation</td>
      <td>2022-08-26</td>
      <td>2025-06-04</td>
      <td>6</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>quast, busco, chromeister, meryl, merqury</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1820710</td>
      <td>4360126</td>
      <td>4.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome assembly using PacBio data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/hifi-assembly/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity</td>
      <td>De-novo assembly, Visualisation, Genome assembly, Cross-assembly, Mapping assembly, Sequence assembly validation</td>
      <td>2021-11-29</td>
      <td>2025-06-04</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>quast, busco, hifiasm, fasta-stats, gfa_to_fa, flye</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>539</td>
      <td>633</td>
      <td>1.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Unicycler Assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/unicycler-assembly/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Aggregation, Genome assembly, Statistical calculation, Coding region prediction, Gene prediction, Sequence composition calculation, Sequencing quality control, Visualisation, Validation, Genome annotation, Sequence assembly validation</td>
      <td>2017-10-11</td>
      <td>2025-05-15</td>
      <td>24</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>quast, fastqc, multiqc, unicycler, prokka</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2736025</td>
      <td>7336307</td>
      <td>4.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Assembly of the mitochondrial genome from PacBio HiFi reads</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mitochondrion-assembly/tutorial.html</td>
      <td>Genomics, Sequence assembly, Biodiversity, Mitochondria</td>
      <td></td>
      <td>2024-09-03</td>
      <td>2025-06-04</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>mitohifi, compress_file</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>757</td>
      <td>936</td>
      <td>2.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Unicycler assembly of SARS-CoV-2 genome with preprocessing to remove human genome reads</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/assembly-with-preprocessing/tutorial.html</td>
      <td></td>
      <td>Data handling, Scatter plot plotting, Box-Whisker plot plotting, Sequence assembly visualisation, Aggregation, Genome assembly, Sequence contamination filtering, Read mapping, Sequencing quality control, Pairwise sequence alignment, Sequence file editing, Validation</td>
      <td>2020-04-21</td>
      <td>2025-05-15</td>
      <td>14</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>fastp, collapse_dataset, fasterq_dump, samtools_fastx, multiqc, fasta_filter_by_length, minimap2, nanoplot, bowtie2, __ZIP_COLLECTION__, bandage_info, samtools_view, bandage_image, samtools_stats, seqtk_sample, unicycler</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>2728915</td>
      <td>7323915</td>
      <td>5.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Using the VGP workflows to assemble a vertebrate genome with HiFi and Hi-C data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/vgp_workflow_training/tutorial.html</td>
      <td>Sequence assembly, Chromosome conformation capture</td>
      <td>Visualisation, Sequence assembly validation</td>
      <td>2022-04-06</td>
      <td>2025-09-09</td>
      <td>26</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>quast</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3331</td>
      <td>4647</td>
      <td>2.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Chloroplast genome assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/chloroplast-assembly/tutorial.html</td>
      <td></td>
      <td>Scatter plot plotting, Genome annotation, Box-Whisker plot plotting, Sequence assembly visualisation, Genome visualisation, Genome assembly, Read alignment, Cross-assembly, Sequence alignment, Sequence assembly, Generation, Genome indexing, Read mapping, De-novo assembly, Mapping assembly, Coding region prediction, Gene prediction, Analysis</td>
      <td>2020-12-04</td>
      <td>2025-05-15</td>
      <td>21</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, upload1, fasta-stats, nanoplot, pilon, bandage_info, bwa_mem, bandage_image, flye, prokka</td>
      <td>UseGalaxy.eu</td>
      <td>GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>7691</td>
      <td>11923</td>
      <td>2.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome assembly and assembly QC - Introduction short version</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/assembly_and_qc_short/slides.html</td>
      <td></td>
      <td></td>
      <td>2025-04-22</td>
      <td>2025-04-23</td>
      <td>2</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>263512</td>
      <td>605412</td>
      <td>2.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Deeper look into Genome Assembly algorithms</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/algorithms-introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2021-01-12</td>
      <td>2021-01-12</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2702522</td>
      <td>7245856</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>An introduction to get started in genome assembly and annotation</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/get-started-genome-assembly/slides.html</td>
      <td></td>
      <td></td>
      <td>2021-11-30</td>
      <td>2023-10-23</td>
      <td>7</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2241753</td>
      <td>5689338</td>
      <td>2.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Functionally Assembled Terrestrial Ecosystem Simulator (FATES)</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/fates/tutorial.html</td>
      <td></td>
      <td>Visualisation</td>
      <td>2020-10-25</td>
      <td>2025-05-10</td>
      <td>18</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>ctsm_fates, interactive_tool_panoply, xarray_select, tp_find_and_replace, ggplot2_point, __EXTRACT_DATASET__, xarray_metadata_info</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2724701</td>
      <td>7316687</td>
      <td>4.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Pangeo ecosystem 101 for everyone - Introduction to Xarray Galaxy Tools</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/pangeo/tutorial.html</td>
      <td></td>
      <td>Visualisation</td>
      <td>2022-02-18</td>
      <td>2025-05-10</td>
      <td>6</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>graphicsmagick_image_montage, xarray_mapplot, regexColumn1, climate_stripes, __MERGE_COLLECTION__, xarray_select, xarray_coords_info, ggplot2_point, xarray_netcdf2netcdf, __EXTRACT_DATASET__, xarray_metadata_info</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2116387</td>
      <td>5289452</td>
      <td>5.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Ocean's variables study</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/ocean-variables/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-11-29</td>
      <td>2025-05-10</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>interactive_tool_odv, divand_full_analysis, interactive_tool_divand</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>199</td>
      <td>239</td>
      <td>2.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Sentinel 5P data visualisation</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/sentinel5_data/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-01-26</td>
      <td>2025-05-10</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>interactive_tool_panoply, interactive_tool_copernicus, interactive_tool_copernicus_notebook</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>283</td>
      <td>329</td>
      <td>0.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Ocean's variables study</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/ocean_qcv_analysis/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-08-08</td>
      <td>2025-08-08</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>harmonize_insitu_to_netcdf, interactive_tool_odv, tool_odv, tool_odv_history, tool_biogeochemical_calibration</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>16</td>
      <td>24</td>
      <td>4.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Pangeo Notebook in Galaxy - Introduction to Xarray</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/pangeo-notebook/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-02-18</td>
      <td>2025-05-10</td>
      <td>11</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>interactive_tool_pangeo_notebook</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2116593</td>
      <td>5289941</td>
      <td>4.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Getting your hands-on climate data</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/climate-101/tutorial.html</td>
      <td></td>
      <td>Visualisation</td>
      <td>2020-04-30</td>
      <td>2025-05-10</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Grep1, cds_essential_variability, psy_maps, interactive_tool_panoply, climate_stripes, tp_awk_tool, ggplot2_point, datamash_ops</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1015</td>
      <td>1412</td>
      <td>1.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Analyse Argo data</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/argo_pangeo/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-24</td>
      <td>2025-05-10</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>interactive_tool_odv, argo_getdata, xarray_coords_info, timeseries_extraction, xarray_metadata_info</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>274</td>
      <td>334</td>
      <td>1.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Introduction to climate data</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2020-10-11</td>
      <td>2025-05-10</td>
      <td>2</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314392</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Collaboration with JupyterGIS</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/jupytergis_collaboration/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-04-03</td>
      <td>2025-05-10</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>155</td>
      <td>197</td>
      <td>0.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Ocean Data View (ODV)</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/ocean-data-view/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-11-27</td>
      <td>2025-05-10</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_odv</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>370</td>
      <td>449</td>
      <td>1.200000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Visualize Climate data with Panoply netCDF viewer</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/panoply/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-03-05</td>
      <td>2025-05-10</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_panoply</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6224</td>
      <td>10183</td>
      <td>0.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Functionally Assembled Terrestrial Ecosystem Simulator (FATES) with Galaxy Climate JupyterLab</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/fates-jupyterlab/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-10-25</td>
      <td>2025-05-10</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_jupyter_notebook</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>539</td>
      <td>945</td>
      <td>1.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Getting your hands-on earth data</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/earth_system/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-10-04</td>
      <td>2025-05-10</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>obis_data, argo_getdata, c3s, spocc_occ</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>84</td>
      <td>115</td>
      <td>3.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Community Building</td>
      <td>Make your tools available on your subdomain</td>
      <td>https://training.galaxyproject.org//topics/community/tutorials/tools_subdomains/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-12-03</td>
      <td>2024-12-20</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>36</td>
      <td>51</td>
      <td>12.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Community Building</td>
      <td>Creating community content</td>
      <td>https://training.galaxyproject.org//topics/community/tutorials/community_content/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-12-20</td>
      <td>2024-12-20</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>33</td>
      <td>44</td>
      <td>0.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Community Building</td>
      <td>What's a Special Interest Group?</td>
      <td>https://training.galaxyproject.org//topics/community/tutorials/sig_define/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-07-08</td>
      <td>2024-12-20</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>69</td>
      <td>99</td>
      <td>4.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Community Building</td>
      <td>Creation of an interactive Galaxy tools table for your community</td>
      <td>https://training.galaxyproject.org//topics/community/tutorials/community-tool-table/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-13</td>
      <td>2024-12-20</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>64</td>
      <td>81</td>
      <td>3.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Galaxy Community Building</td>
      <td>Creating a Special Interest Group</td>
      <td>https://training.galaxyproject.org//topics/community/tutorials/sig_create/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-07-08</td>
      <td>2024-12-20</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>90</td>
      <td>99</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>Protein-ligand docking</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/cheminformatics/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-10-19</td>
      <td>2023-11-09</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ctb_online_data_fetch, ctb_compound_convert, docking, prepare_receptor, ctb_remDuplicates, ctb_chemfp_butina_clustering, sdf_to_tab, tp_grep_tool, openbabel_compound_convert, tp_cat, tp_find_and_replace, ctb_np-likeness-calculator, prepare_box, ctb_filter, collapse_dataset, ctb_remIons, fpocket, ctb_chemfp_nxn_clustering, chembl, get_pdb, ctb_chemfp_mol2fps, openbabel_svg_depiction, ctb_obgrep, ctb_change_title</td>
      <td></td>
      <td>UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>18776</td>
      <td>32164</td>
      <td>1.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>Analysis of molecular dynamics simulations</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/analysis-md-simulations/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-06-03</td>
      <td>2023-11-09</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mdanalysis_distance, bio3d_rmsf, bio3d_rmsd, mdanalysis_ramachandran_plot, mdanalysis_rdf, mdanalysis_cosine_analysis, mdanalysis_dihedral, bio3d_pca</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>32107</td>
      <td>57083</td>
      <td>1.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>Data management in Medicinal Chemistry</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/med-chem-data/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-01-08</td>
      <td>2024-09-13</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>openbabel_compound_convert, chembl, ctb_silicos_qed, md_converter, get_pdb, openbabel_svg_depiction, ctb_im_rxn_maker</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>413</td>
      <td>500</td>
      <td>1.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>Running molecular dynamics simulations using GROMACS</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/md-simulation-gromacs/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-06-03</td>
      <td>2023-11-09</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>gmx_em, gmx_editconf, tp_grep_tool, gmx_sim, gmx_setup, gmx_solvate, get_pdb</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>26126</td>
      <td>47102</td>
      <td>1.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>Running molecular dynamics simulations using NAMD</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/md-simulation-namd/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-06-03</td>
      <td>2023-11-09</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>setup, namd_nvt, namd_npt, minimizer</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>4158</td>
      <td>7000</td>
      <td>1.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>Protein target prediction of a bioactive ligand with Align-it and ePharmaLib</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/zauberkugel/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-02-15</td>
      <td>2024-06-14</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>openbabel_addh, rdconf, tp_cat, ctb_alignit, sort1, split_file_to_collection</td>
      <td>UseGalaxy.cz</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1185</td>
      <td>1725</td>
      <td>2.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>High Throughput Molecular Dynamics and Analysis</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/htmd-analysis/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-05-20</td>
      <td>2025-04-08</td>
      <td>19</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>gmx_em, gmx_editconf, bio3d_rmsf, tp_grep_tool, bio3d_rmsd, bio3d_pca, openbabel_compound_convert, gmx_solvate, gmx_merge_topology_files, gmx_energy, vmd_hbonds, gmx_sim, ambertools_acpype, mdanalysis_cosine_analysis, gmx_setup, md_converter, bio3d_pca_visualize, get_pdb</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>10491</td>
      <td>18559</td>
      <td>2.450000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>Virtual screening of the SARS-CoV-2 main protease with rxDock and pose scoring</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/covid19-docking/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-03-27</td>
      <td>2023-11-09</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>xchem_transfs_scoring, collapse_dataset, xchem_pose_scoring, rxdock_rbcavity, openbabel_compound_convert, enumerate_charges, rdock_rbdock, sucos_max_score, ctb_frankenstein_ligand, rxdock_rbdock, rdock_rbcavity, split_file_to_collection</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3130</td>
      <td>5171</td>
      <td>3.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Computational chemistry</td>
      <td>Setting up molecular systems</td>
      <td>https://training.galaxyproject.org//topics/computational-chemistry/tutorials/setting-up-molecular-systems/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-06-03</td>
      <td>2023-11-09</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1752</td>
      <td>2368</td>
      <td>1.700000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Contributing to the Galaxy Training Network with GitHub</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/github-contribution/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-06-12</td>
      <td>2025-03-14</td>
      <td>2</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723336</td>
      <td>7314485</td>
      <td>3.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Overview of the Galaxy Training Material</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-25</td>
      <td>2025-03-14</td>
      <td>2</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314393</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Teaching Python</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/python/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-19</td>
      <td>2023-11-23</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>238</td>
      <td>531</td>
      <td>1.200000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>FAIR-by-Design methodology</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/fair-by-design/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-10-17</td>
      <td>2024-10-17</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>26</td>
      <td>31</td>
      <td>0.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Creating Interactive Galaxy Tours</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/create-new-tutorial-tours/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-06-25</td>
      <td>2025-04-08</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>287</td>
      <td>445</td>
      <td>2.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Including a new topic</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/create-new-topic/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-06-04</td>
      <td>2025-03-14</td>
      <td>32</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1202</td>
      <td>1659</td>
      <td>0.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Updating diffs in admin training</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/admin-knitting/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-06-07</td>
      <td>2023-11-23</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>55</td>
      <td>97</td>
      <td>0.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Generating PDF artefacts of the website</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/generating-pdf/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-06-04</td>
      <td>2023-11-09</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>72</td>
      <td>89</td>
      <td>3.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Contributing with GitHub via its interface</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/github-interface-contribution/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-06-12</td>
      <td>2023-11-09</td>
      <td>17</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>676</td>
      <td>1027</td>
      <td>2.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Tools, Data, and Workflows for tutorials</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/create-new-tutorial-technical/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-06-25</td>
      <td>2025-04-08</td>
      <td>26</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>540</td>
      <td>731</td>
      <td>1.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Adding Quizzes to your Tutorial</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/create-new-tutorial-quiz/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-05-19</td>
      <td>2022-10-18</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>97</td>
      <td>111</td>
      <td>0.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Design and plan session, course, materials</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/design/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2025-03-11</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>563</td>
      <td>753</td>
      <td>2.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Adding auto-generated video to your slides</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/slides-with-video/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-10-20</td>
      <td>2023-11-09</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>259</td>
      <td>339</td>
      <td>4.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Principles of learning and how they apply to training and teaching</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/learning-principles/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2025-03-11</td>
      <td>17</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>5308</td>
      <td>6334</td>
      <td>0.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Preview the GTN website as you edit your training material</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/rendering_gtn/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-14</td>
      <td>2025-07-02</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>297</td>
      <td>419</td>
      <td>5.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>GTN Metadata</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/schemas/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-06-02</td>
      <td>2024-10-15</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>256</td>
      <td>342</td>
      <td>2.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Creating a new tutorial</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/create-new-tutorial/tutorial.html</td>
      <td></td>
      <td>Data handling, Read pre-processing, Primer removal, Sequence file editing, Sequence trimming, Read mapping</td>
      <td>2018-06-20</td>
      <td>2025-03-14</td>
      <td>32</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Cut1, tp_sorted_uniq, Filter1, bam_to_sam, seqtk_subseq, cutadapt, c3s, bowtie2, tp_cat, random_lines1, tp_sed_tool, tp_cut_tool, fastq_to_tabular</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1738</td>
      <td>2584</td>
      <td>5.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>FAIR Galaxy Training Material</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/fair-gtn/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-09-16</td>
      <td>2025-09-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Single Cell Publication - Data Analysis</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/meta-analysis-data/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-11-07</td>
      <td>2024-11-07</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>36</td>
      <td>38</td>
      <td>0.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Single Cell Publication - Data Plotting</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/meta-analysis-plot/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-11-07</td>
      <td>2024-11-07</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>32</td>
      <td>32</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Creating content in Markdown</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/create-new-tutorial-content/tutorial.html</td>
      <td></td>
      <td>Validation, Sequencing quality control</td>
      <td>2017-06-25</td>
      <td>2025-04-08</td>
      <td>85</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>multiqc, upload1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3662</td>
      <td>9659</td>
      <td>2.116667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Creating Slides</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/create-new-tutorial-slides/slides.html</td>
      <td></td>
      <td></td>
      <td>2018-06-13</td>
      <td>2024-11-08</td>
      <td>17</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314393</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Contributing to the Galaxy Training Material</td>
      <td>Updating tool versions in a tutorial</td>
      <td>https://training.galaxyproject.org//topics/contributing/tutorials/updating_tutorial/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-11-29</td>
      <td>2024-11-29</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>82</td>
      <td>101</td>
      <td>5.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Removal of human reads from SARS-CoV-2 sequencing data</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/human-reads-removal/tutorial.html</td>
      <td></td>
      <td>Genome indexing, Data handling, Read mapping, Sequence file editing, Sequence alignment, Generation</td>
      <td>2021-08-04</td>
      <td>2023-11-09</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>__UNZIP_COLLECTION__, trimmomatic, seqtk_subseq, Grep1, __ZIP_COLLECTION__, tp_replace_in_line, bwa_mem, samtools_fastx</td>
      <td>UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3947</td>
      <td>5455</td>
      <td>2.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Automating Galaxy workflows using the command line</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/workflow-automation/tutorial.html</td>
      <td></td>
      <td>Variant classification, Tree-based sequence alignment</td>
      <td>2021-06-08</td>
      <td>2025-04-08</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>collapse_dataset, compose_text_param, bcftools_consensus, snpSift_filter, pangolin</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>MISSISSIPPI, UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2802</td>
      <td>3407</td>
      <td>2.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Submitting sequence data to ENA</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/upload-data-to-ena/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-08-10</td>
      <td>2024-12-11</td>
      <td>10</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>ena_consensus_submit, ena_upload</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2402421</td>
      <td>6223814</td>
      <td>3.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>From NCBI's Sequence Read Archive (SRA) to Galaxy: SARS-CoV-2 variant analysis</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/sars-cov-2/tutorial.html</td>
      <td></td>
      <td>Data handling, Sequence contamination filtering, Sequence alignment, Formatting, Sequencing quality control, Generation, Genome indexing, Read mapping, SNP detection, Validation</td>
      <td>2020-06-24</td>
      <td>2024-06-14</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>fastp, picard_MarkDuplicates, lofreq_call, Filter1, lofreq_indelqual, snpSift_extractFields, fasterq_dump, Grep1, snpeff_sars_cov_2, multiqc, bwa_mem, samtools_stats, lofreq_viterbi, tp_cut_tool</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>MISSISSIPPI, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>13544</td>
      <td>20425</td>
      <td>2.883333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Mutation calling, viral genome reconstruction and lineage/clade assignment from SARS-CoV-2 sequencing data</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.html</td>
      <td></td>
      <td>Variant classification, Tree-based sequence alignment, Methylation analysis, Variant calling</td>
      <td>2021-06-30</td>
      <td>2025-03-28</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>nextclade, pangolin, fasta_regex_finder, upload1, datamash_ops, Filter1</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>7221</td>
      <td>12991</td>
      <td>3.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>SRA Aligned Read Format to Speed Up SARS-CoV-2 data Analysis</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/ncbi-sarf/tutorial.html</td>
      <td></td>
      <td>Data handling, SNP detection</td>
      <td>2021-05-31</td>
      <td>2024-06-14</td>
      <td>12</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>jq, fasterq_dump, query_tabular, fastq_dump, snpeff_sars_cov_2, tp_cat, fastq_to_fasta_python</td>
      <td>UseGalaxy.be, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2503917</td>
      <td>6559803</td>
      <td>4.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>SARS-CoV-2 Viral Sample Alignment and Variant Visualization</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.html</td>
      <td></td>
      <td>Genome indexing, Sequencing quality control, Read mapping, Genome visualisation, Statistical calculation, Sequence alignment, Sequence composition calculation, Generation</td>
      <td>2023-07-10</td>
      <td>2024-02-29</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>jbrowse, bwa_mem, fastqc</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, Galaxy@Pasteur, GalaxyTrakr, MISSISSIPPI, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1188</td>
      <td>1543</td>
      <td>4.200000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Data Manipulation Olympics</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/data-manipulation-olympics/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-09-16</td>
      <td>2025-09-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>join1, {{version_cut_columns}}, tp_sorted_uniq, wc_gnu, upload1, {{version_split}}, {{version_join}}, {{version_wc}}, datamash_ops, {{version_replace_text_line}}, Cut1, {{version_filter}}, {{version_datamash}}, {{version_cat}}, {{version_paste}}, Filter1, Show beginning1, {{version_remove_beginning}}, cat1, {{version_sort}}, tp_cat, tp_sort_header_tool, tp_split_on_column, Add_a_column1, {{version_cut_advanced}}, Count1, {{version_unique}}, Grouping1, Remove beginning1, {{version_replace_text_column}}, regexColumn1, {{version_select_first}}, tabular_to_csv, {{version_compute}}, {{version_remove_columns_by_header}}, {{version_count}}</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>A (very) brief history of genomics</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/gnmx-lecture1/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-01-09</td>
      <td>2024-02-20</td>
      <td>2</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>999560</td>
      <td>2413362</td>
      <td>3.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Learning about one gene across biological resources and formats</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/online-resources-gene/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-09-07</td>
      <td>2023-11-09</td>
      <td>3</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>ncbi_blastp_wrapper</td>
      <td>Galaxy@AuBi, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@Pasteur, GalaxyTrakr, MISSISSIPPI, PepSimili, PhagePromotor, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>1158293</td>
      <td>2763714</td>
      <td>5.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>One protein along the UniProt page</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/online-resources-protein/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-09-07</td>
      <td>2024-12-05</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>408</td>
      <td>434</td>
      <td>0.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Basics of using Git from the Command Line</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/git-cli/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-06-16</td>
      <td>2023-11-16</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>128</td>
      <td>373</td>
      <td>0.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Version Control with Git</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/bash-git/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-07-06</td>
      <td>2023-10-23</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_rstudio</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>266</td>
      <td>497</td>
      <td>0.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>SQL with Python</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/sql-python/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-10-11</td>
      <td>2023-11-16</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>499</td>
      <td>681</td>
      <td>0.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Warm-up for statistics and machine learning</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-warmup-stat-ml/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-11</td>
      <td>2025-03-11</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>86</td>
      <td>100</td>
      <td>0.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>CLI Educational Game - Bashcrawl</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/cli-bashcrawl/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-09-30</td>
      <td>2024-10-15</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1965</td>
      <td>2240</td>
      <td>0.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Variant Calling Workflow</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/bash-variant-calling/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-07-07</td>
      <td>2023-10-23</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>8564</td>
      <td>11207</td>
      <td>2.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Advanced CLI in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/cli-advanced/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-09-30</td>
      <td>2024-10-15</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>923</td>
      <td>1181</td>
      <td>2.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>R basics in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/r-basics/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-10-08</td>
      <td>2024-06-14</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>interactive_tool_rstudio</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2623</td>
      <td>3547</td>
      <td>2.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Introduction to Python</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-basics/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-11-08</td>
      <td>2024-10-15</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1265</td>
      <td>1652</td>
      <td>1.900000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Make &amp; Snakemake</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/snakemake/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-11-18</td>
      <td>2025-07-09</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3283</td>
      <td>4245</td>
      <td>0.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>CLI basics</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/cli-basics/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-09-30</td>
      <td>2024-10-15</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1112</td>
      <td>1577</td>
      <td>2.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Introduction to SQL</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/sql-basic/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-10-11</td>
      <td>2024-12-13</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>541</td>
      <td>665</td>
      <td>1.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Math</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-math/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>901</td>
      <td>1126</td>
      <td>1.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Advanced SQL</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/sql-advanced/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-10-11</td>
      <td>2024-12-13</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1409</td>
      <td>1772</td>
      <td>0.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>SQL with R</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/sql-r/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-10-11</td>
      <td>2023-11-16</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>593</td>
      <td>754</td>
      <td>0.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>SQL Educational Game - Murder Mystery</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/sql-game/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-10-11</td>
      <td>2023-11-23</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>661</td>
      <td>715</td>
      <td>0.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Advanced Python</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-advanced-np-pd/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-11-08</td>
      <td>2024-10-15</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>505</td>
      <td>683</td>
      <td>1.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Functions</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-functions/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>944</td>
      <td>1111</td>
      <td>0.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Data visualisation Olympics - Visualization in R</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/data-manipulation-olympics-viz-r/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-06-16</td>
      <td>2024-05-07</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>590</td>
      <td>668</td>
      <td>1.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Advanced R in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/r-advanced/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-12-09</td>
      <td>2024-10-15</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>interactive_tool_rstudio</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1492</td>
      <td>2346</td>
      <td>2.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Introduction to sequencing with Python (part one)</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/gnmx-lecture2/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-01-23</td>
      <td>2024-02-20</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>227</td>
      <td>288</td>
      <td>2.116667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Introduction to sequencing with Python (part two)</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/gnmx-lecture3/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-01-23</td>
      <td>2024-02-20</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>104</td>
      <td>110</td>
      <td>0.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Data Manipulation Olympics - SQL</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/data-manipulation-olympics-sql/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-06-16</td>
      <td>2024-05-07</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>761</td>
      <td>854</td>
      <td>0.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>dplyr &amp; tidyverse for data processing</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/r-dplyr/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-10-20</td>
      <td>2024-10-15</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>581</td>
      <td>764</td>
      <td>0.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Basic Types &amp; Type Conversion</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-types/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>381</td>
      <td>659</td>
      <td>6.133333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Plotting in Python</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-plotting/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-11-08</td>
      <td>2024-10-15</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>547</td>
      <td>669</td>
      <td>1.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Lists &amp; Strings &amp; Dictionaries</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-iterables/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2024-06-25</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2051</td>
      <td>2430</td>
      <td>0.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Data Manipulation Olympics - JQ</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/data-manipulation-olympics-jq/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-06-16</td>
      <td>2024-05-07</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>305</td>
      <td>349</td>
      <td>0.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Introduction to sequencing with Python (part three)</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/gnmx-lecture4/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-02-06</td>
      <td>2024-02-20</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>105</td>
      <td>114</td>
      <td>0.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Flow Control</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-flow/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>254</td>
      <td>305</td>
      <td>0.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Introduction to sequencing with Python (part four)</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/gnmx-lecture5/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-02-13</td>
      <td>2024-02-20</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>111</td>
      <td>122</td>
      <td>0.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Data manipulation with Pandas</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/gnmx-lecture6/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-02-20</td>
      <td>2025-02-27</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>37</td>
      <td>43</td>
      <td>1.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Loops</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-loops/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>365</td>
      <td>440</td>
      <td>1.700000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Versioning your code and data with git</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/gnmx-lecture7/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-12</td>
      <td>2024-03-12</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>7</td>
      <td>7</td>
      <td>0.133333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Files &amp; CSV</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-files/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>526</td>
      <td>615</td>
      <td>0.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Try &amp; Except</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-exceptions/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>561</td>
      <td>643</td>
      <td>0.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Introductory Graduation</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-basics-recap/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>223</td>
      <td>265</td>
      <td>1.700000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Testing</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-testing/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-19</td>
      <td>2023-02-13</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>214</td>
      <td>237</td>
      <td>0.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Type annotations</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-typing/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-19</td>
      <td>2023-02-13</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>604</td>
      <td>640</td>
      <td>0.183333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Conda Environments For Software Development</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-conda/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-19</td>
      <td>2023-05-18</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>621</td>
      <td>671</td>
      <td>0.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Argparse</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-argparse/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-01-24</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1296</td>
      <td>1411</td>
      <td>0.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Virtual Environments For Software Development</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-venv/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-19</td>
      <td>2023-05-18</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>168</td>
      <td>190</td>
      <td>1.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Globbing</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-glob/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>510</td>
      <td>552</td>
      <td>0.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Subprocess</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-subprocess/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-25</td>
      <td>2023-02-13</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1141</td>
      <td>1326</td>
      <td>0.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Coding Style</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-linting/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-11-03</td>
      <td>2023-05-18</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>170</td>
      <td>183</td>
      <td>0.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Foundations of Data Science</td>
      <td>Python - Multiprocessing</td>
      <td>https://training.galaxyproject.org//topics/data-science/tutorials/python-multiprocessing/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-19</td>
      <td>2023-11-16</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>836</td>
      <td>900</td>
      <td>0.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>JavaScript plugins</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/visualization-charts/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-06-22</td>
      <td>2025-04-08</td>
      <td>20</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723941</td>
      <td>7315216</td>
      <td>3.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Tool Dependencies and Conda</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/conda/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-20</td>
      <td>2023-04-14</td>
      <td>33</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314397</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Galaxy Interactive Tours</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/interactive-tours/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-01-12</td>
      <td>2021-07-27</td>
      <td>14</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314397</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Tool Shed: sharing Galaxy tools</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/toolshed/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-12</td>
      <td>2022-10-18</td>
      <td>15</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314397</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Galaxy Code Architecture</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/architecture/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-20</td>
      <td>2024-06-14</td>
      <td>34</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314397</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Galaxy Webhooks</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/webhooks/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-04-03</td>
      <td>2025-04-08</td>
      <td>16</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723439</td>
      <td>7314643</td>
      <td>3.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Prerequisites for building software/conda packages</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/conda_sys/slides.html</td>
      <td></td>
      <td></td>
      <td>2018-02-15</td>
      <td>2021-05-28</td>
      <td>6</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314397</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Scripting Galaxy using the API and BioBlend</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/bioblend-api/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-01-17</td>
      <td>2023-11-16</td>
      <td>22</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723934</td>
      <td>7315284</td>
      <td>4.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Galaxy Interactive Environments</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/interactive-environments/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-02-05</td>
      <td>2023-02-17</td>
      <td>16</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314397</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>ToolFactory: Generating Tools From Simple Scripts</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/tool-generators/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-05-26</td>
      <td>2024-06-14</td>
      <td>10</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2511644</td>
      <td>6585094</td>
      <td>4.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Tool development and integration into Galaxy</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/tool-integration/slides.html</td>
      <td></td>
      <td></td>
      <td>2016-09-30</td>
      <td>2025-04-08</td>
      <td>30</td>
      <td>external</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723259</td>
      <td>7314401</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Generic plugins</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/visualization-generic/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2016-09-30</td>
      <td>2023-11-09</td>
      <td>21</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723579</td>
      <td>7314868</td>
      <td>4.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Tool Dependencies and Containers</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/containers/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-20</td>
      <td>2022-10-18</td>
      <td>14</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314397</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Debugging Galaxy</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/debugging/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-06-15</td>
      <td>2024-06-14</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>576</td>
      <td>1180</td>
      <td>2.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Contributing to BioBlend as a developer</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/bioblend-dev/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-05-25</td>
      <td>2023-11-09</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>212</td>
      <td>299</td>
      <td>3.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Creating Galaxy tools from Conda Through Deployment</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/tool-from-scratch/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-06-15</td>
      <td>2024-10-08</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2685</td>
      <td>4356</td>
      <td>2.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Data source integration</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/data-source-integration/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2016-09-30</td>
      <td>2022-10-18</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>218</td>
      <td>327</td>
      <td>3.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Contributing a New Feature to Galaxy Core</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/core-contributing/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-06-08</td>
      <td>2025-03-14</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>691</td>
      <td>1045</td>
      <td>2.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Setting up a dev Onedata instance</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/onedata-dev-instance/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-24</td>
      <td>2025-04-08</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>17</td>
      <td>20</td>
      <td>4.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Writing Automated Tests for Galaxy</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/writing_tests/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-07-15</td>
      <td>2024-06-25</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>250</td>
      <td>678</td>
      <td>2.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>ToolFactory: Generating Tools From More Complex Scripts</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/tool-generators-advanced/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-05-26</td>
      <td>2024-09-27</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>395</td>
      <td>479</td>
      <td>1.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Adding and updating best practice metadata for Galaxy tools using the bio.tools registry</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/tool-annotation/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-06</td>
      <td>2024-11-06</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>230</td>
      <td>274</td>
      <td>2.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Development in Galaxy</td>
      <td>Galaxy Interactive Tools</td>
      <td>https://training.galaxyproject.org//topics/dev/tutorials/interactive-tools/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-03-02</td>
      <td>2024-12-03</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1141</td>
      <td>1526</td>
      <td>2.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Digital Humanities</td>
      <td>Text-Mining Differences in Chinese Newspaper Articles</td>
      <td>https://training.galaxyproject.org//topics/digital-humanities/tutorials/text_mining_chinese/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-02-28</td>
      <td>2025-09-09</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Cut1, tp_replace_in_line, Add_a_column1, wordcloud, datamash_ops, sort1, Filter1, diff</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>124</td>
      <td>148</td>
      <td>1.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Digital Humanities</td>
      <td>Introduction to Digital Humanities in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/digital-humanities/tutorials/introduction_to_dh/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-09-11</td>
      <td>2025-09-15</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_grep_tool, wc_gnu, Remove beginning1, tp_replace_in_line, tp_find_and_replace, wordcloud, diff</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>7</td>
      <td>10</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Species distribution modeling</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/species-distribution-modeling/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-07-04</td>
      <td>2025-06-03</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Filter1, tabular_to_csv, interactive_tool_wallace, tp_find_and_replace, spocc_occ, tp_cut_tool</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3083</td>
      <td>4691</td>
      <td>1.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>RAD-Seq to construct genetic maps</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/genetic-map-rad-seq/tutorial.html</td>
      <td></td>
      <td>Data handling</td>
      <td>2017-02-14</td>
      <td>2025-06-03</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>stacks_denovomap, stacks_genotypes</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>863</td>
      <td>1217</td>
      <td>1.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Ecoregionalization workflow tutorial</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/Ecoregionalization_tutorial/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-09-16</td>
      <td>2025-06-03</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ecoregion_taxa_seeker, ecoregion_GeoNearestNeighbor, ecoregion_clara_cluster, filter_tabular, interactive_tool_jupyter_notebook, unzip, regexColumn1, ecoregion_brt_analysis, ecoregion_cluster_estimate, mergeCols1, ecoregion_eco_map, tp_cut_tool</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>214</td>
      <td>299</td>
      <td>3.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>RAD-Seq de-novo data analysis</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/de-novo-rad-seq/tutorial.html</td>
      <td></td>
      <td>Statistical calculation, Data handling, Sequence composition calculation, Sequencing quality control</td>
      <td>2017-02-14</td>
      <td>2025-06-03</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>stacks_populations, stacks_denovomap, stacks_procrad, fastqc</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2608</td>
      <td>4187</td>
      <td>2.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Regional GAM</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/regionalGAM/tutorial.html</td>
      <td>Ecology</td>
      <td>Visualisation</td>
      <td>2019-07-04</td>
      <td>2025-06-03</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>regionalgam_gls, csv2tab_R, regionalgam_glmmpql, Remove beginning1, flight-curve, regionalgam_autocor_acf, regionalgam_flight_curve, regexColumn1, tab2csv_R, ggplot2_point, mergeCols1, regionalgam_ab_index, tp_awk_tool, Paste1, regionalgam_plot_trend, Filter1, Count1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>329</td>
      <td>443</td>
      <td>3.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Marine Omics identifying biosynthetic gene clusters</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/marine_omics_bgc/tutorial.html</td>
      <td></td>
      <td>Protein feature detection, Genome annotation, Sequence motif recognition</td>
      <td>2024-08-19</td>
      <td>2025-06-03</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>interproscan, sanntis_marine, regex1, prodigal, SanntiS_marine</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>151</td>
      <td>177</td>
      <td>2.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Compute and analyze biodiversity metrics with PAMPA toolsuite</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/PAMPA-toolsuite-tutorial/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-11-19</td>
      <td>2025-09-03</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Filter1, tp_easyjoin_tool, pampa_glmcomm, regex1, pampa_glmsp, pampa_plotglm, regexColumn1, datamash_transpose, tp_cat, mergeCols1, tp_sort_header_tool, pampa_communitymetrics, pampa_presabs, tp_cut_tool, Count1</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1071</td>
      <td>1416</td>
      <td>1.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Preparing genomic data for phylogeny reconstruction</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/phylogeny-data-prep/tutorial.html</td>
      <td></td>
      <td>Genetic variation analysis, Multiple sequence alignment, Sequence clustering, Genome annotation, Sequence analysis, Sequence assembly validation</td>
      <td>2022-10-21</td>
      <td>2025-06-03</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>glimmer_gbk_to_orf, phykit_alignment_based, collapse_dataset, clustalw, busco, proteinortho_grab_proteins, regex1, proteinortho, clipkit, repeatmasker_wrapper, tp_replace_in_line, funannotate_predict, Filter1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2110</td>
      <td>2895</td>
      <td>2.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Phylodiversity analysis quick tutorial</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/phylodiversity_workflow/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-06-06</td>
      <td>2025-06-06</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>crsconverter, phylogenetic_occupancy_matcher, estimate_endem, phylo_index</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>56</td>
      <td>80</td>
      <td>1.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Taxonomic Analysis of eDNA</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/eDNA-taxonomic-analysis/tutorial.html</td>
      <td></td>
      <td>Sequence contamination filtering, Sequencing quality control</td>
      <td>2025-06-02</td>
      <td>2025-06-03</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>fastp, cshl_fastq_to_fasta, ncbi_blastn_wrapper, Count1</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, MISSISSIPPI, UseGalaxy.be, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>273</td>
      <td>348</td>
      <td>0.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Metabarcoding/eDNA through Obitools</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/Obitools-metabarcoding/tutorial.html</td>
      <td></td>
      <td>Sequencing quality control, Statistical calculation, Sequence analysis, Sequence composition calculation</td>
      <td>2022-02-28</td>
      <td>2025-06-03</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>join1, fastq_groomer, Cut1, obi_grep, obi_ngsfilter, wc_gnu, ncbi_blastn_wrapper, fastqc, obi_clean, obi_stat, obi_illumina_pairend, unzip, obi_annotate, obi_tab, obi_uniq, seq_filter_by_id, Filter1</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>4312</td>
      <td>6010</td>
      <td>1.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Biodiversity data exploration</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/biodiversity-data-exploration/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-03-01</td>
      <td>2025-06-03</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Remove beginning1, ecology_beta_diversity, regexColumn1, ecology_presence_abs_abund, ecology_stat_presence_abs, ecology_link_between_var, tool_anonymization, ecology_homogeneity_normality, tp_cut_tool</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1562</td>
      <td>2196</td>
      <td>2.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Visualization of Climate Data using  NetCDF xarray Map Plotting</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/x-array-map-plot/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-26</td>
      <td>2024-06-27</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>graphicsmagick_image_montage, xarray_mapplot, cdo_operations, xarray_coords_info, xarray_metadata_info</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2312</td>
      <td>2678</td>
      <td>0.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Checking expected species and contamination in bacterial isolate</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/bacterial-isolate-species-contamination-checking/tutorial.html</td>
      <td>Whole genome sequencing, Genomics, Microbiology, Microbial ecology</td>
      <td>Cross-assembly, Statistical calculation, Expression analysis, Taxonomic classification</td>
      <td>2024-03-04</td>
      <td>2024-06-27</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>est_abundance, recentrifuge, kraken2, upload1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>761</td>
      <td>1021</td>
      <td>3.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>From NDVI data with OpenEO to time series visualisation with Holoviews</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/ndvi_openeo/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-01-24</td>
      <td>2024-06-29</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>interactive_tool_holoviz, interactive_tool_copernicus_notebook</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>171</td>
      <td>209</td>
      <td>1.283333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Sentinel 2 biodiversity</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/remote-sensing/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-04-13</td>
      <td>2025-06-03</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>srs_diversity_maps, srs_preprocess_s2, srs_spectral_indices, srs_pca, srs_global_indices, srs_process_data</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>571</td>
      <td>740</td>
      <td>1.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>RAD-Seq Reference-based data analysis</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/ref-based-rad-seq/tutorial.html</td>
      <td></td>
      <td>Data handling, Statistical calculation, Genome indexing, Sequence alignment, Read mapping, Sequencing quality control, Sequence composition calculation, Generation, Validation</td>
      <td>2017-02-14</td>
      <td>2025-06-03</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>stacks_populations, bwa, Cut1, stacks_refmap, bwa_wrapper, Summary_Statistics1, fastqc, Grep1, cat1, multiqc, Convert characters1, addValue, tp_replace_in_line, regex_replace, tp_sort_header_tool, stacks_procrad, Filter1, Count1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2885</td>
      <td>4561</td>
      <td>1.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Obis marine indicators</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/obisindicators/tutorial.html</td>
      <td>Biodiversity, Ecology, Data mining</td>
      <td></td>
      <td>2023-04-11</td>
      <td>2025-06-03</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>obis_data, tp_cut_tool, obisindicators</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>293</td>
      <td>371</td>
      <td>2.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Cleaning GBIF data for the use in Ecology</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/gbif_cleaning/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-28</td>
      <td>2025-06-03</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Summary_Statistics1, gdal_ogr2ogr, spocc_occ, Filter1, Count1</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>604</td>
      <td>752</td>
      <td>0.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Creating FAIR Quality assessment reports and draft of Data Papers from EML metadata with MetaShRIMPS</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/Metashrimps_tutorial/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-01-22</td>
      <td>2025-06-03</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_metashrimps</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>93</td>
      <td>100</td>
      <td>2.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Life Traits Ecoregionalization workflow</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/ecoregiolifetraits/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-06-02</td>
      <td>2025-06-10</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>Filter1, ecoregion_GeoNearestNeighbor, ecoregion_taxa_seeker, unzip, regexColumn1, ClaraGuess, ecoregion_brt_analysis, WormsMeasurements, ecoregion_eco_map, column_remove_by_header, tp_cut_tool</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>22</td>
      <td>24</td>
      <td>4.883333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Champs blocs indicators</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/champs-blocs/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-01-04</td>
      <td>2025-06-03</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>cb_div, cb_qecb, cb_eco, cb_ivr, cb_dissim</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>160</td>
      <td>308</td>
      <td>1.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Data submission using ENA upload Tool</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/ENA_Biodiv_submission/tutorial.html</td>
      <td></td>
      <td>Read pre-processing, Visualisation, Primer removal, Read mapping, Sequence trimming, Validation, Sequencing quality control</td>
      <td>2024-12-13</td>
      <td>2025-03-20</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>fastq_groomer, ncbi_blastn_wrapper, regex1, cutadapt, ab1_fastq_converter, collection_element_identifiers, __RELABEL_FROM_FILE__, multiqc, __FILTER_EMPTY_DATASETS__, fastq_filter, falco, Paste1, fastq_to_fasta_python, bg_uniq</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>161</td>
      <td>167</td>
      <td>0.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Creating metadata using Ecological Metadata Language (EML) standard with EML Assembly Line functionalities</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/MetaShARK_tutorial/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-04</td>
      <td>2024-09-10</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_metashark</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>170</td>
      <td>211</td>
      <td>2.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Cleaning GBIF data using OpenRefine</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/openrefine_gbif/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-01-21</td>
      <td>2025-06-03</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>regex1, interactive_tool_openrefine</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>126</td>
      <td>153</td>
      <td>0.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>QGIS Web Feature Services</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/QGIS_Web_Feature_Services/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-10-11</td>
      <td>2025-06-03</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_qgis</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>254</td>
      <td>292</td>
      <td>1.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Visualize EBV cube data with Panoply netCDF viewer</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/panoply_ebv/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-10-28</td>
      <td>2024-06-27</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_panoply</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>213</td>
      <td>276</td>
      <td>1.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Foundational Aspects of Machine Learning using Python</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/intro-to-ml-with-python/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-11</td>
      <td>2025-05-19</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>327</td>
      <td>385</td>
      <td>1.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Pretraining a Large Language Model (LLM) from Scratch on DNA Sequences</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/genomic-llm-pretraining/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-04-17</td>
      <td>2025-05-22</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>579</td>
      <td>674</td>
      <td>1.700000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Motivation and Demotivation</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/motivation_demotivation/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2025-03-11</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1858</td>
      <td>2200</td>
      <td>0.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Training techniques to enhance learner participation and engagement</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/learner_participation_engagement/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2025-03-11</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1562</td>
      <td>1845</td>
      <td>0.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Train-the-Trainer: putting it all together</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/workshop-final-exercise/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-06-12</td>
      <td>2025-03-11</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>75</td>
      <td>87</td>
      <td>0.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Assessment and feedback in training and teachings</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/assessment/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2025-03-11</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1217</td>
      <td>1413</td>
      <td>0.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Fine-tuning a LLM for DNA Sequence Classification</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/genomic-llm-finetuning/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-04-17</td>
      <td>2025-05-22</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>376</td>
      <td>437</td>
      <td>1.283333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Neural networks using Python</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/neural-networks-with-python/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-11</td>
      <td>2025-03-11</td>
      <td>1</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>342973</td>
      <td>797496</td>
      <td>2.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Predicting Mutation Impact with Zero-shot Learning using a pretrained DNA LLM</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/genomic-llm-zeroshot-prediction/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-04-17</td>
      <td>2025-05-22</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>91</td>
      <td>94</td>
      <td>2.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Deep Learning (without Generative Artificial Intelligence) using Python</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/deep-learning-without-gai-with-python/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-11</td>
      <td>2025-03-11</td>
      <td>1</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>342974</td>
      <td>797499</td>
      <td>2.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Generating Artificial Yeast DNA Sequences using a DNA LLM</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/genomic-llm-sequence-generation/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-04-17</td>
      <td>2025-05-22</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>86</td>
      <td>96</td>
      <td>0.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Optimizing DNA Sequences for Biological Functions using a DNA LLM</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/genomic-llm-sequence-optimization/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-04-17</td>
      <td>2025-05-22</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>13</td>
      <td>15</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Regulations/standards for AI using DOME</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/dome/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-11</td>
      <td>2025-05-19</td>
      <td>3</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>342995</td>
      <td>797533</td>
      <td>4.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>Formation of the Super-Structures on the Inactive X</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/formation_of_super-structures_on_xi/tutorial.html</td>
      <td></td>
      <td>Read pre-processing, Primer removal, Genome visualisation, Statistical calculation, Pathway visualisation, Sequence composition calculation, Sequencing quality control, Gene regulatory network analysis, Enrichment analysis, Read mapping, Structure visualisation, Sequence trimming, Peak calling, Mapping</td>
      <td>2018-09-17</td>
      <td>2025-04-08</td>
      <td>14</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>deeptools_multi_bam_summary, deeptools_plot_heatmap, bbgtobigwig, deeptools_bam_compare, fastqc, deeptools_bam_coverage, cat1, trim_galore, deeptools_plot_correlation, macs2_callpeak, bowtie2, bedtools_sortbed, bedtools_mergebed, deeptools_plot_fingerprint, jbrowse2, deeptools_compute_matrix, Filter1, samtools_idxstats</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2729096</td>
      <td>7326018</td>
      <td>6.200000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>Identification of the binding sites of the T-cell acute lymphocytic leukemia protein 1 (TAL1)</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/tal1-binding-site-identification/tutorial.html</td>
      <td>ChIP-seq</td>
      <td>Statistical calculation, Genome indexing, Sequence alignment, Read mapping, Sequencing quality control, Sequence composition calculation, Generation, Gene regulatory network analysis, Enrichment analysis, Peak calling, Mapping</td>
      <td>2016-12-20</td>
      <td>2025-04-08</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>deeptools_multi_bam_summary, deeptools_plot_heatmap, bwa, bedtools_intersectbed, trimmomatic, deeptools_bam_compare, fastqc, deeptools_plot_correlation, macs2_callpeak, deeptools_compute_gc_bias, deeptools_plot_fingerprint, deeptools_compute_matrix, deeptools_correct_gc_bias, samtools_idxstats</td>
      <td>UseGalaxy.be, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, MISSISSIPPI, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>7052</td>
      <td>12968</td>
      <td>3.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>Infinium Human Methylation BeadChip</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/ewas-suite/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-08-27</td>
      <td>2024-10-15</td>
      <td>21</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>clusterprofiler_bitr, Cut1, clusterprofiler_go, Remove beginning1, minfi_analysis, chipeakanno_annopeaks</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2725280</td>
      <td>7317592</td>
      <td>4.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>CUT&amp;RUN data analysis</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/cut_and_run/tutorial.html</td>
      <td></td>
      <td>Sequence alignment analysis, Data handling, Read pre-processing, Primer removal, Peak calling, Formatting, Sequencing quality control, Gene regulatory network analysis, Read mapping, Enrichment analysis, Visualisation, Sequence trimming, Validation, Mapping</td>
      <td>2024-03-06</td>
      <td>2025-03-19</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>deeptools_plot_heatmap, Extract genomic DNA 1, bedtools_bamtobed, Grep1, multiqc, macs2_callpeak, deeptools_plot_fingerprint, falco, wig_to_bigWig, bowtie2, idr, deeptools_compute_matrix, bamFilter, bedtools_intersectbed, picard_MarkDuplicates, __FLATTEN__, meme_chip, trim_galore, datamash_transpose, pe_histogram, __EXTRACT_DATASET__</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>5381</td>
      <td>6868</td>
      <td>1.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>ATAC-Seq data analysis</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/atac-seq/tutorial.html</td>
      <td></td>
      <td>Sequence alignment analysis, Read pre-processing, Data handling, Primer removal, Statistical calculation, Formatting, Sequencing quality control, Sequence composition calculation, Gene regulatory network analysis, Enrichment analysis, Read mapping, Visualisation, Sequence trimming, Peak calling, Mapping</td>
      <td>2019-09-02</td>
      <td>2024-08-08</td>
      <td>23</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>deeptools_plot_heatmap, bedtools_intersectbed, Cut1, picard_MarkDuplicates, bedtools_bamtobed, wig_to_bigWig, fastqc, cutadapt, Grep1, pygenomeTracks, macs2_callpeak, bamFilter, bowtie2, pe_histogram, tp_sort_header_tool, deeptools_compute_matrix, Filter1, samtools_idxstats</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2785650</td>
      <td>7427433</td>
      <td>4.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>DNA Methylation data analysis</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/methylation-seq/tutorial.html</td>
      <td>Epigenomics</td>
      <td>Visualisation, Read mapping, Statistical calculation, Sequence composition calculation, Sequencing quality control</td>
      <td>2017-02-16</td>
      <td>2025-04-03</td>
      <td>24</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>deeptools_plot_profile, metilene, wig_to_bigWig, fastqc, bwameth, replace_column_with_key_value_file, pileometh, falco, tp_awk_tool, tp_tail_tool, deeptools_compute_matrix</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>14537</td>
      <td>24364</td>
      <td>2.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>Identification of the binding sites of the Estrogen receptor</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/estrogen-receptor-binding-site-identification/tutorial.html</td>
      <td></td>
      <td>Read pre-processing, Primer removal, Statistical calculation, Sequence composition calculation, Sequencing quality control, Gene regulatory network analysis, Enrichment analysis, Read mapping, Sequence trimming, Peak calling</td>
      <td>2017-09-18</td>
      <td>2025-04-08</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>deeptools_multi_bam_summary, deeptools_plot_heatmap, bam_to_sam, deeptools_bam_compare, fastqc, deeptools_bam_coverage, trim_galore, deeptools_plot_correlation, macs2_callpeak, bowtie2, deeptools_compute_gc_bias, deeptools_plot_fingerprint, deeptools_compute_matrix, samtools_idxstats</td>
      <td>UseGalaxy.be</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1883</td>
      <td>2793</td>
      <td>2.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>Hi-C analysis of Drosophila melanogaster cells using HiCExplorer</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/hicexplorer/tutorial.html</td>
      <td></td>
      <td>Visualisation, Formatting, Read mapping</td>
      <td>2018-02-23</td>
      <td>2025-08-11</td>
      <td>30</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>hicexplorer_hicdetectloops, hicexplorer_hicfindrestrictionsites, pygenomeTracks, hicexplorer_hicfindtads, bowtie2, hicexplorer_hicbuildmatrix, hicexplorer_hiccorrectmatrix, hicexplorer_hicinfo, hicexplorer_hicplotmatrix, hicexplorer_hicmergematrixbins, hicexplorer_hicpca</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>7235</td>
      <td>12269</td>
      <td>1.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>Introduction to DNA Methylation data analysis</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/introduction-dna-methylation/slides.html</td>
      <td></td>
      <td></td>
      <td>2016-10-05</td>
      <td>2024-03-05</td>
      <td>2</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314397</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Epigenetics</td>
      <td>Introduction to ChIP-Seq data analysis</td>
      <td>https://training.galaxyproject.org//topics/epigenetics/tutorials/introduction-chip-seq/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-12</td>
      <td>2024-03-05</td>
      <td>2</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314397</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Phylogenetics - Back to basics</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/abc_intro_phylo/tutorial.html</td>
      <td></td>
      <td>Phylogenetic analysis, Sequence analysis, Multiple sequence alignment, Phylogenetic tree generation, Phylogenetic tree reconstruction, Phylogenetic tree generation (from molecular sequences), Phylogenetic tree analysis, Phylogenetic tree generation (maximum likelihood and Bayesian methods)</td>
      <td>2024-05-10</td>
      <td>2024-05-10</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>newick_display, rbc_mafft, iqtree, fasttree</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>4240</td>
      <td>5474</td>
      <td>1.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Tree thinking for tuberculosis evolution and epidemiology</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_phylogeny/tutorial.html</td>
      <td>Genomics, Microbiology, Phylogeny, Infectious disease</td>
      <td>Phylogenetic tree analysis, Sequence analysis</td>
      <td>2022-03-16</td>
      <td>2025-06-04</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>upload1, raxml, interactive_tool_rstudio</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2122</td>
      <td>3073</td>
      <td>2.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Identifying tuberculosis transmission links: from SNPs to transmission clusters</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_transmission/tutorial.html</td>
      <td>Genomics, Microbiology, Infectious disease, DNA polymorphism</td>
      <td>Antimicrobial resistance prediction, Phylogenetic tree visualisation, Variant calling, Phylogenetic tree generation</td>
      <td>2022-03-16</td>
      <td>2025-05-26</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>snippy, Grep1, upload1, __MERGE_COLLECTION__, samtools_view, tb_variant_filter, tp_sed_tool, tp_replace_in_line, addName, tp_grep_tool, tp_cat, tb_profiler_profile, collapse_dataset, trimmomatic, tp_easyjoin_tool, bcftools_consensus, snp_sites, snp_dists, clustering_from_distmat</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2536</td>
      <td>3663</td>
      <td>2.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Phylogenetics - Back to Basics - Introduction</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/abc_intro_phylo-intro/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-05-10</td>
      <td>2024-05-10</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>797991</td>
      <td>1935700</td>
      <td>2.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Phylogenetic analysis for bacterial comparative genomics</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/bacterial-comparative-genomics/tutorial.html</td>
      <td>Genomics, Microbiology, Phylogeny</td>
      <td>Phylogenetic tree generation (from molecular sequences), Phylogenetic tree generation (maximum likelihood and Bayesian methods)</td>
      <td>2025-03-14</td>
      <td>2025-03-14</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>fasttree, upload1</td>
      <td>GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi</td>
      <td>0</td>
      <td></td>
      <td>70</td>
      <td>90</td>
      <td>3.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Phylogenetics - Back to Basics - Terminology</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/abc_intro_phylo-terminology/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-05-10</td>
      <td>2024-05-10</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>797991</td>
      <td>1935700</td>
      <td>2.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Phylogenetics - Back to Basics - Multiple Sequence Alignment</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/abc_intro_phylo-msa/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-05-10</td>
      <td>2024-05-10</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>797991</td>
      <td>1935700</td>
      <td>2.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Phylogenetics - Back to Basics - Building Trees</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/abc_intro_phylo-trees/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-05-10</td>
      <td>2024-05-10</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>797991</td>
      <td>1935700</td>
      <td>2.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Phylogenetics - Back to Basics - Estimating trees from alignments</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/abc_intro_phylo-mle-trees/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-05-10</td>
      <td>2024-05-10</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>797991</td>
      <td>1935700</td>
      <td>2.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Phylogenetics - Back to Basics - Phylogenetic Networks</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/abc_intro_phylo-networks/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-05-10</td>
      <td>2024-05-10</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>797991</td>
      <td>1935700</td>
      <td>2.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Data management in Medicinal Chemistry</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/med-chem-data/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-09-16</td>
      <td>2025-09-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>openbabel_compound_convert, chembl, ctb_silicos_qed, md_converter, get_pdb, openbabel_svg_depiction, ctb_im_rxn_maker</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>An earth-system RO-Crate</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/earth_system_rocrate/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-12-02</td>
      <td>2024-12-02</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>493458</td>
      <td>1166003</td>
      <td>2.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Making clinical datasets FAIR</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-clinical/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-10-24</td>
      <td>2024-09-13</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>276</td>
      <td>364</td>
      <td>1.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Sequence data submission to ENA</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-ena/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-11-01</td>
      <td>2024-06-25</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>476</td>
      <td>562</td>
      <td>0.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>FAIR and its Origins</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-origin/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-26</td>
      <td>2024-03-27</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>120</td>
      <td>136</td>
      <td>0.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Workflow Run RO-Crate Introduction</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/ro-crate-workflow-run-ro-crate/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-11</td>
      <td>2023-05-11</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>323</td>
      <td>415</td>
      <td>1.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>FAIR in a nutshell</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-intro/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-30</td>
      <td>2024-04-30</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>782</td>
      <td>1019</td>
      <td>1.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Introduction to Data Management Plan (DMP) for Peatland Research and PeatDataHub</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/data-management-peatlands/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-09-13</td>
      <td>2024-09-13</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>79</td>
      <td>108</td>
      <td>2.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>RO-Crate - Introduction</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/ro-crate-intro/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-23</td>
      <td>2025-04-08</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>560</td>
      <td>747</td>
      <td>1.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>DataPLANT ARCs</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/dataplant-arcs/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-10-22</td>
      <td>2024-12-11</td>
      <td>3</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>553043</td>
      <td>1310277</td>
      <td>4.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>RO-Crate in Python</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/ro-crate-in-python/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-11</td>
      <td>2025-04-08</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>223</td>
      <td>249</td>
      <td>1.450000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Metadata</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-metadata/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-26</td>
      <td>2024-03-27</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>155</td>
      <td>185</td>
      <td>1.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Exporting Workflow Run RO-Crates from Galaxy</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/ro-crate-in-galaxy/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-11</td>
      <td>2024-08-26</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>257</td>
      <td>314</td>
      <td>2.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>FAIR data management solutions</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/data-management/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-30</td>
      <td>2025-01-16</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>344</td>
      <td>427</td>
      <td>2.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Best practices for workflows in GitHub repositories</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/ro-crate-galaxy-best-practices/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-11</td>
      <td>2025-04-08</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>349</td>
      <td>405</td>
      <td>1.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>FAIR Galaxy Training Material</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-gtn/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-30</td>
      <td>2024-03-27</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>129</td>
      <td>155</td>
      <td>2.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Data Registration</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-data-registration/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-26</td>
      <td>2024-09-27</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>67</td>
      <td>71</td>
      <td>1.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>FAIR Bioimage Metadata</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/bioimage-metadata/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-09-13</td>
      <td>2024-10-15</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>259</td>
      <td>311</td>
      <td>2.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Access</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-access/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-26</td>
      <td>2024-03-27</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>48</td>
      <td>57</td>
      <td>3.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Submitting workflows to LifeMonitor</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/ro-crate-submitting-life-monitor/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-11</td>
      <td>2024-09-26</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>43</td>
      <td>51</td>
      <td>4.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>Persistent Identifiers</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-persistent-identifiers/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-26</td>
      <td>2024-03-27</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>136</td>
      <td>154</td>
      <td>0.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>REMBI - Recommended Metadata for Biological Images – metadata guidelines for bioimaging data</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/bioimage-REMBI/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-09-13</td>
      <td>2024-10-15</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>251</td>
      <td>283</td>
      <td>2.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>FAIR Data, Workflows, and Research</td>
      <td>FAIRification of an RNAseq dataset</td>
      <td>https://training.galaxyproject.org//topics/fair/tutorials/fair-rna/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-03-27</td>
      <td>2024-03-27</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>219</td>
      <td>268</td>
      <td>1.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Workflow Reports</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/workflow-reports/tutorial.html</td>
      <td></td>
      <td>Visualisation</td>
      <td>2022-06-02</td>
      <td>2024-06-14</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Cut1, Grouping1, tp_sorted_uniq, Remove beginning1, upload1, csv_to_tabular, ggplot2_point, datamash_ops</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, MISSISSIPPI, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>831</td>
      <td>1159</td>
      <td>2.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Using Workflow Parameters</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/workflow-parameters/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-06-27</td>
      <td>2024-06-14</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>tp_head_tool</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>MISSISSIPPI</td>
      <td>0</td>
      <td></td>
      <td>1914</td>
      <td>2450</td>
      <td>2.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Creating, Editing and Importing Galaxy Workflows</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/workflow-editor/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-07-17</td>
      <td>2023-11-09</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Show beginning1, Paste1, tp_tac</td>
      <td>Galaxy@AuBi, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>MISSISSIPPI</td>
      <td>0</td>
      <td></td>
      <td>5747</td>
      <td>7169</td>
      <td>3.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>InterMine integration with Galaxy</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/intermine/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-12-09</td>
      <td>2024-02-29</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>galaxy_intermine_exchange, intermine</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>169</td>
      <td>226</td>
      <td>3.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Galaxy workflows in Dockstore</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/galaxy-on-dockstore/slides.html</td>
      <td></td>
      <td></td>
      <td>2022-01-14</td>
      <td>2022-10-27</td>
      <td>4</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2175431</td>
      <td>5478417</td>
      <td>2.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Getting data into Galaxy</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/get-data/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-05-24</td>
      <td>2022-10-18</td>
      <td>4</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314397</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Extracting Workflows from Histories</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/history-to-workflow/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-10-18</td>
      <td>2024-12-02</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2888</td>
      <td>3744</td>
      <td>3.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Downloading and Deleting Data in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/download-delete-data/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-10-17</td>
      <td>2023-11-09</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>7244</td>
      <td>9637</td>
      <td>1.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>JupyterLab in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/jupyterlab/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-03-05</td>
      <td>2025-04-25</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_jupyter_notebook</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2073</td>
      <td>3336</td>
      <td>2.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Using dataset collections</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/collections/tutorial.html</td>
      <td></td>
      <td>Sequence alignment, Generation, Genome indexing, Read mapping</td>
      <td>2016-09-30</td>
      <td>2024-12-05</td>
      <td>20</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>collapse_dataset, lofreq_call, bwa_mem, snpSift_extractFields</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>15384</td>
      <td>21111</td>
      <td>3.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Onedata user-owned storage</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/onedata-byos/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-24</td>
      <td>2025-04-08</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>60</td>
      <td>73</td>
      <td>6.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Understanding Galaxy history system</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/history/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-02-20</td>
      <td>2025-04-28</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>5377</td>
      <td>6945</td>
      <td>2.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Searching Your History</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/search/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-07-15</td>
      <td>2023-11-16</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>394</td>
      <td>600</td>
      <td>2.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Creating high resolution images of Galaxy Workflows</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/workflow-posters/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-02-05</td>
      <td>2025-04-08</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>graphicsmagick_image_convert</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>243</td>
      <td>295</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>RStudio in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/rstudio/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-10-08</td>
      <td>2025-07-04</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>interactive_tool_rstudio</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>4970</td>
      <td>7743</td>
      <td>2.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Getting started with Onedata distributed storage</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/onedata-getting-started/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-24</td>
      <td>2025-04-08</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>95</td>
      <td>136</td>
      <td>5.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Importing (uploading) data from Onedata</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/onedata-remote-import/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-24</td>
      <td>2025-04-08</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>136</td>
      <td>158</td>
      <td>3.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Exporting to Onedata remote</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/onedata-remote-export/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-03-24</td>
      <td>2025-04-08</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>25</td>
      <td>27</td>
      <td>0.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Name tags for following complex histories</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/name-tags/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-04-08</td>
      <td>2023-11-09</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1578</td>
      <td>2123</td>
      <td>2.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Rule Based Uploader</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/upload-rules/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-07-04</td>
      <td>2024-06-14</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>4273</td>
      <td>5693</td>
      <td>3.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Annotate, prepare tests and publish Galaxy workflows in workflow registries</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/workflow-fairification/tutorial.html</td>
      <td></td>
      <td>Visualisation, Read mapping, Validation, Sequencing quality control</td>
      <td>2025-05-26</td>
      <td>2025-05-26</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>multiqc, falco</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>145</td>
      <td>175</td>
      <td>1.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Use Jupyter notebooks in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/galaxy-intro-jupyter/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-07-02</td>
      <td>2025-05-11</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>interactive_tool_jupyter_notebook</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2311</td>
      <td>3066</td>
      <td>2.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Rule Based Uploader: Advanced</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/upload-rules-advanced/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-05-25</td>
      <td>2024-06-14</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>757</td>
      <td>1015</td>
      <td>4.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Using Galaxy and Managing your Data</td>
      <td>Group tags for complex experimental designs</td>
      <td>https://training.galaxyproject.org//topics/galaxy-interface/tutorials/group-tags/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-03-06</td>
      <td>2023-11-09</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2687</td>
      <td>3643</td>
      <td>3.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Prokka</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/annotation-with-prokka/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation, Coding region prediction, Genome annotation, Gene prediction</td>
      <td>2018-03-06</td>
      <td>2025-03-19</td>
      <td>26</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, prokka</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2741445</td>
      <td>7345317</td>
      <td>4.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Helixer</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/helixer/tutorial.html</td>
      <td>Biodiversity, Genomics, Zoology, Plant biology</td>
      <td>Differential protein expression profiling, Genome visualisation, Genome annotation, Sequence annotation, Gene prediction, Sequence assembly validation</td>
      <td>2024-08-21</td>
      <td>2025-06-04</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>omark, jbrowse, busco, helixer, jcvi_gff_stats, gffread</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1013</td>
      <td>1221</td>
      <td>1.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Secondary metabolite discovery</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/secondary-metabolite-discovery/tutorial.html</td>
      <td></td>
      <td>Data handling, Sequence clustering, Service invocation, Differential gene expression analysis, Gene prediction</td>
      <td>2024-02-21</td>
      <td>2024-09-27</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ctb_np-likeness-calculator, ncbi_acc_download, collapse_dataset, Remove beginning1, interactive_tool_jupyter_notebook, bp_genbank2gff3, ctb_silicos_qed, openbabel_remDuplicates, edu.tamu.cpt.gff3.customGbkToGff, tp_awk_tool, ctb_chemfp_mol2fps, ctb_simsearch, antismash</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>581</td>
      <td>734</td>
      <td>1.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Comparison of two annotation tools - Helixer and Braker3</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/comparison-braker-helixer-annotation/tutorial.html</td>
      <td>Sequence assembly, Genomics</td>
      <td>Differential protein expression profiling, Genome visualisation, Genome annotation, Sequence annotation, Gene prediction, Sequence alignment, Sequence assembly validation</td>
      <td>2025-06-20</td>
      <td>2025-06-25</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>omark, jbrowse, busco, rna_star, braker3, helixer, gffread</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>183</td>
      <td>226</td>
      <td>2.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>From small to large-scale genome comparison</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/hpc-for-lsgc/tutorial.html</td>
      <td></td>
      <td>Multiple sequence alignment</td>
      <td>2021-02-08</td>
      <td>2024-07-31</td>
      <td>8</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>tp_awk_tool, clustalw, chromeister, gecko</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2678499</td>
      <td>7153822</td>
      <td>4.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Masking repeats with RepeatMasker</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/repeatmasker/tutorial.html</td>
      <td>Sequence composition, complexity and repeats, Genomics</td>
      <td>Genome annotation, Editing, RNA-Seq analysis, Repeat sequence detection, Mapping</td>
      <td>2021-11-29</td>
      <td>2025-06-03</td>
      <td>22</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>bedtools_maskfastabed, red, repeatmodeler, repeatmasker_wrapper</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>9624</td>
      <td>12307</td>
      <td>1.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Identification of AMR genes in an assembled bacterial genome</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/amr-gene-detection/tutorial.html</td>
      <td>Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Sequence analysis, Infectious disease, Antimicrobial resistance</td>
      <td>Antimicrobial resistance prediction, Genome visualisation, Genome annotation, Read mapping</td>
      <td>2024-01-23</td>
      <td>2025-06-10</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, Grep1, upload1, bowtie2, tbl2gff3, bakta, staramr_search</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>3098</td>
      <td>4231</td>
      <td>3.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Bacterial Genome Annotation</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-genome-annotation/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis, Whole genome sequencing, Functional genomics, Mobile genetic elements</td>
      <td>Genome assembly, Genome visualisation, Sequence motif recognition, Structural variation detection, Protein feature detection, Multilocus sequence typing, Genome annotation, Nucleic acid feature detection, Scaffolding</td>
      <td>2024-02-01</td>
      <td>2024-10-15</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>plasmidfinder, Grouping1, jbrowse, upload1, tp_replace_in_column, tbl2gff3, bakta, integron_finder, tp_tail_tool, isescan</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>5654</td>
      <td>7598</td>
      <td>2.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (prokaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation</td>
      <td>2021-06-04</td>
      <td>2025-05-15</td>
      <td>22</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>create_account, jbrowse, list_organism, iframe, create_or_update</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2498056</td>
      <td>6537590</td>
      <td>5.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Braker3</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/braker3/tutorial.html</td>
      <td>Sequence assembly, Genomics, Biodiversity, Zoology, Plant biology</td>
      <td>Differential protein expression profiling, Genome visualisation, Genome annotation, Sequence annotation, Gene prediction, Sequence alignment, Sequence assembly validation</td>
      <td>2025-03-17</td>
      <td>2025-06-20</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>omark, jbrowse, busco, rna_star, braker3, fasta-stats, gffread</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1013</td>
      <td>1274</td>
      <td>2.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Introduction to Genome Annotation</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-05-13</td>
      <td>2025-04-23</td>
      <td>5</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2723256</td>
      <td>7314404</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Dataset construction for bacterial comparative genomics</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-comparative-genomics-dataset-construction/tutorial.html</td>
      <td>Genomics, Microbiology</td>
      <td>Data handling, Data retrieval, Sequence database search</td>
      <td>2025-03-14</td>
      <td>2025-03-14</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>Cut1, Remove beginning1, datasets_download_genome, cat1, upload1, sort1, bg_uniq</td>
      <td>UseGalaxy.fr</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>56</td>
      <td>69</td>
      <td>4.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Bacterial genome quality control</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-genome-quality-control/tutorial.html</td>
      <td>Genomics, Microbiology</td>
      <td>Genome comparison, Validation, Statistical calculation, Sequence composition calculation, Sequence assembly validation, Sequencing quality control</td>
      <td>2025-03-14</td>
      <td>2025-03-14</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>checkm2, drep_dereplicate, upload1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>60</td>
      <td>66</td>
      <td>3.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Bacterial pangenomics</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-pangenomics/tutorial.html</td>
      <td>Genomics, Microbiology</td>
      <td>Essential dynamics, Rarefaction, Sequence assembly, Genome annotation</td>
      <td>2025-03-14</td>
      <td>2025-03-14</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>ppanggolin_all, ppanggolin_msa, upload1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>61</td>
      <td>72</td>
      <td>1.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome Annotation</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/genome-annotation/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2015-06-29</td>
      <td>2025-06-10</td>
      <td>19</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>34776</td>
      <td>63514</td>
      <td>1.450000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Creating an Official Gene Set</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/official-gene-set/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-11-02</td>
      <td>2024-02-29</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>38</td>
      <td>44</td>
      <td>1.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Funannotate</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/funannotate/tutorial.html</td>
      <td>Gene and protein families</td>
      <td>Query and retrieval, Genome visualisation, Sequence annotation, Sequence alignment, Formatting, Sequence motif recognition, Homology-based gene prediction, Protein feature detection, Genome annotation, Fold recognition, Information extraction, Sequence assembly validation</td>
      <td>2021-11-29</td>
      <td>2025-06-10</td>
      <td>26</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>interproscan, jbrowse, busco, rna_star, picard_MergeSamFiles, aegean_parseval, eggnog_mapper, funannotate_annotate, funannotate_compare, funannotate_predict</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>10061</td>
      <td>14367</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Comparative gene analysis in unannotated genomes</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/gene-centric/tutorial.html</td>
      <td>Genomics, Gene and protein families, Sequence analysis, Phylogeny, Comparative genomics</td>
      <td>Sequence alignment analysis, De-novo assembly, Transcriptome assembly, Database search, Phylogenetic tree generation, Multiple sequence alignment, Coding region prediction</td>
      <td>2022-09-08</td>
      <td>2025-06-03</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>join1, bg_diamond, Cut1, collapse_dataset, tab2fasta, cat1, regexColumn1, orfipy, rapidnj, rbc_mafft, tp_split_on_column, gops_intersect_1, bg_diamond_makedb, Add_a_column1, Filter1, bg_diamond_view</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1748</td>
      <td>2485</td>
      <td>3.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Maker (short)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/annotation-with-maker-short/tutorial.html</td>
      <td></td>
      <td>Genome visualisation, Genome annotation, Sequence annotation, Sequence assembly validation</td>
      <td>2021-01-12</td>
      <td>2024-02-29</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>maker, jbrowse, busco, jcvi_gff_stats, fasta-stats, maker_map_ids, gffread</td>
      <td></td>
      <td>UseGalaxy.be</td>
      <td>0</td>
      <td></td>
      <td>3803</td>
      <td>5374</td>
      <td>3.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Essential genes detection with Transposon insertion sequencing</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/tnseq/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence analysis, Mobile genetic elements</td>
      <td>Read pre-processing, Primer removal, Sequence trimming, Transposon prediction</td>
      <td>2019-07-02</td>
      <td>2024-03-18</td>
      <td>12</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>Cut1, gff_to_prot, tp_easyjoin_tool, cutadapt, deeptools_bam_coverage, Add_a_column1, transit_gumbel, tp_sort_header_tool, bowtie_wrapper, __EXTRACT_DATASET__, Filter1, bg_find_subsequences</td>
      <td>UseGalaxy.no, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>2731128</td>
      <td>7328762</td>
      <td>4.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Maker</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/annotation-with-maker/tutorial.html</td>
      <td></td>
      <td>Sequence annotation, Operation, Homology-based gene prediction, Genome visualisation, Genome annotation, Ab-initio gene prediction, Gene prediction, Sequence assembly validation</td>
      <td>2018-09-24</td>
      <td>2024-02-29</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>maker, jbrowse, busco, augustus_training, jcvi_gff_stats, snap_training, fasta-stats, maker_map_ids, gffread</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>13542</td>
      <td>24096</td>
      <td>1.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Functional annotation of protein sequences</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/functional/tutorial.html</td>
      <td>Function analysis, Protein sites, features and motifs</td>
      <td>Sequence alignment analysis, Query and retrieval, Sequence motif recognition, Homology-based gene prediction, Protein feature detection, Genome annotation, Fold recognition, Information extraction</td>
      <td>2022-07-20</td>
      <td>2025-06-03</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bg_diamond, interproscan, ncbi_blastp_wrapper, eggnog_mapper</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>3871</td>
      <td>5258</td>
      <td>1.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (eukaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo-euk/tutorial.html</td>
      <td>Genomics</td>
      <td>Genome visualisation</td>
      <td>2022-08-22</td>
      <td>2025-06-03</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>iframe, jbrowse, create_or_update</td>
      <td>UseGalaxy.no</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>1331</td>
      <td>2076</td>
      <td>2.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Long non-coding RNAs (lncRNAs) annotation with FEELnc</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/lncrna/tutorial.html</td>
      <td></td>
      <td>Classification, Transcriptome assembly, RNA-Seq analysis, Sequence annotation, Annotation</td>
      <td>2022-09-23</td>
      <td>2024-09-18</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>stringtie, feelnc, Extract_features1, gffread, tp_cat</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1808</td>
      <td>2430</td>
      <td>2.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>CRISPR screen analysis</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/crispr-screen/tutorial.html</td>
      <td></td>
      <td>Read pre-processing, Genetic variation analysis, Primer removal, Statistical calculation, Sequence composition calculation, Sequencing quality control, Sequence trimming, Validation</td>
      <td>2021-10-15</td>
      <td>2024-08-08</td>
      <td>12</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>fastqc, cutadapt, mageck_count, multiqc, volcanoplot, mageck_test, mageck_mle, tp_awk_tool</td>
      <td>UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2330217</td>
      <td>5948733</td>
      <td>4.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Microbial Variant Calling</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/microbial-variants/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Phylogenetic tree visualisation, Variant calling, Genome visualisation, Phylogenetic tree generation</td>
      <td>2018-02-26</td>
      <td>2025-03-19</td>
      <td>24</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>snippy, jbrowse</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>14795</td>
      <td>23742</td>
      <td>2.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Visualisation</td>
      <td>Genomic Data Visualisation with JBrowse</td>
      <td>https://training.galaxyproject.org//topics/visualisation/tutorials/jbrowse/tutorial.html</td>
      <td></td>
      <td>Genome visualisation</td>
      <td>2020-01-10</td>
      <td>2024-10-15</td>
      <td>17</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>jbrowse</td>
      <td>Galaxy@AuBi, UseGalaxy.be, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@Pasteur, GalaxyTrakr, MISSISSIPPI, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2728128</td>
      <td>7320892</td>
      <td>4.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Quantification of single-molecule RNA fluorescence in situ hybridization (smFISH) in yeast cell lines</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/2D-spot-detection/tutorial.html</td>
      <td></td>
      <td>Data handling, Image analysis, Visualisation, Image annotation</td>
      <td>2025-03-20</td>
      <td>2025-03-20</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ip_histogram_equalization, ip_spot_detection_2d, ip_overlay_images, ip_points_to_label, unzip, ip_binary_to_labelimage, idr_download_by_ids</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>123</td>
      <td>157</td>
      <td>1.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Quantification of electrophoresis gel bands using QuPath and Galaxy imaging tools</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/electrophoresis-gel-bands-image-analysis/tutorial.html</td>
      <td></td>
      <td>Data handling, Image analysis, Visualisation, Image annotation</td>
      <td>2025-08-11</td>
      <td>2025-08-11</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>graphicsmagick_image_convert, ip_points_to_label, ip_imageinfo, interactive_tool_qupath, ip_split_image, __EXTRACT_DATASET__, ip_2d_feature_extraction</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>97</td>
      <td>121</td>
      <td>1.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Using BioImage.IO models for image analysis in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/process-image-bioimageio/tutorial.html</td>
      <td></td>
      <td>Data handling, Image analysis, Visualisation, Image annotation</td>
      <td>2025-04-09</td>
      <td>2025-04-16</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>image_math, ip_overlay_images, ip_binary_to_labelimage, ip_split_image, bioimage_inference, ip_threshold, __EXTRACT_DATASET__</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>150</td>
      <td>181</td>
      <td>3.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>End-to-End Tissue Microarray Image Analysis with Galaxy-ME</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/multiplex-tissue-imaging-TMA/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-02-14</td>
      <td>2025-03-14</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>vitessce_spatial, ip_convertimage, basic_illumination, scimap_phenotyping, palom, unet_coreograph, quantification, gate_finder, celesta, mesmer, squidpy_spatial, ashlar, rename_tiff_channels, scanpy_compute_graph, scimap_mcmicro_to_anndata</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1299</td>
      <td>1772</td>
      <td>1.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Training Custom YOLO Models for Segmentation of Bioimages</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/yolo-segmentation-training/tutorial.html</td>
      <td></td>
      <td>Data handling, Image analysis, Visualisation, Image annotation</td>
      <td>2025-07-25</td>
      <td>2025-07-31</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>json2yolosegment, graphicsmagick_image_convert, ip_histogram_equalization, interactive_tool_anylabeling, unzip, yolo_training</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>119</td>
      <td>142</td>
      <td>0.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Overview of the Galaxy OMERO-suite - Upload images and metadata in OMERO using Galaxy</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/omero-suite/tutorial.html</td>
      <td></td>
      <td>Image analysis</td>
      <td>2024-12-04</td>
      <td>2025-04-08</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>omero_get_id, omero_get_value, omero_roi_import, omero_metadata_import, omero_filter, omero_import</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>149</td>
      <td>191</td>
      <td>3.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Nucleoli segmentation and feature extraction using CellProfiler</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/tutorial-CP/tutorial.html</td>
      <td></td>
      <td>Parsing, Image analysis, Quantification</td>
      <td>2020-07-01</td>
      <td>2024-10-15</td>
      <td>9</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>cp_export_to_spreadsheet, cp_identify_primary_objects, cp_relate_objects, cp_gray_to_color, cp_measure_object_intensity, cp_cellprofiler, cp_image_math, cp_measure_texture, cp_save_images, cp_measure_granularity, cp_mask_image, cp_convert_objects_to_image, cp_common, cp_enhance_or_suppress_features, cp_display_data_on_image, idr_download_by_ids, cp_measure_object_size_shape, cp_measure_image_area_occupied, cp_measure_image_quality, cp_measure_image_intensity</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2727054</td>
      <td>7319973</td>
      <td>3.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Voronoi segmentation</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/voronoi-segmentation/tutorial.html</td>
      <td></td>
      <td>Data handling, Image analysis, Visualisation, Image annotation</td>
      <td>2025-05-13</td>
      <td>2025-07-11</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ip_convertimage, image_math, ip_overlay_images, ip_count_objects, ip_imageinfo, unzip, voronoi_tessellation, repeat_channels, ip_binary_to_labelimage, ip_filter_standard, colorize_labels, ip_threshold, ip_2d_feature_extraction</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>200</td>
      <td>333</td>
      <td>4.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Introduction to Image Analysis using Galaxy</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/imaging-introduction/tutorial.html</td>
      <td></td>
      <td>Data handling, Image analysis, Visualisation, Image annotation</td>
      <td>2019-08-13</td>
      <td>2025-03-20</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ip_histogram_equalization, ip_overlay_images, ip_count_objects, ip_imageinfo, unzip, ip_binary_to_labelimage, ip_filter_standard, ip_threshold</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2572</td>
      <td>3616</td>
      <td>1.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Parameter tuning and optimization - Evaluating nuclei segmentation with Galaxy</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/parameter-tuning/tutorial.html</td>
      <td></td>
      <td>Data handling, Image analysis, Image annotation, Visualisation</td>
      <td>2025-04-28</td>
      <td>2025-04-28</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ip_histogram_equalization, collapse_dataset, param_value_from_file, ip_filter_standard, ip_threshold, tp_split_on_column, ip_segmetrics</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>57</td>
      <td>68</td>
      <td>2.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Tracking of mitochondria and capturing mitoflashes</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/detection-of-mitoflashes/tutorial.html</td>
      <td></td>
      <td>Image analysis</td>
      <td>2024-11-20</td>
      <td>2024-11-20</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ip_spot_detection_2d, ip_curve_fitting, ip_points_association_nn</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>126</td>
      <td>147</td>
      <td>0.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Analyse HeLa fluorescence siRNA screen</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/hela-screen-analysis/tutorial.html</td>
      <td></td>
      <td>Data handling, Image analysis, Visualisation, Image annotation</td>
      <td>2019-08-13</td>
      <td>2024-11-07</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>collapse_dataset, ip_2d_filter_segmentation_by_features, unzip, ip_binary_to_labelimage, ip_filter_standard, ip_threshold, ip_2d_feature_extraction</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>730</td>
      <td>937</td>
      <td>2.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>Object tracking using CellProfiler</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/object-tracking-using-cell-profiler/tutorial.html</td>
      <td></td>
      <td>Parsing, Image analysis, Quantification</td>
      <td>2021-03-17</td>
      <td>2024-10-15</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>cp_measure_object_size_shape, cp_save_images, cp_export_to_spreadsheet, cp_identify_primary_objects, cp_measure_object_intensity, cp_track_objects, unzip, cp_color_to_gray, cp_tile, cp_cellprofiler, cp_common, cp_overlay_outlines</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2106</td>
      <td>2630</td>
      <td>1.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>FAIR Bioimage Metadata</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/bioimage-metadata/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-09-16</td>
      <td>2025-09-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Imaging</td>
      <td>REMBI - Recommended Metadata for Biological Images – metadata guidelines for bioimaging data</td>
      <td>https://training.galaxyproject.org//topics/imaging/tutorials/bioimage-REMBI/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-09-16</td>
      <td>2025-09-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>From peaks to genes</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2016-03-29</td>
      <td>2025-08-29</td>
      <td>64</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Cut1, Grouping1, upload1, tp_replace_in_column, get_flanks1, tp_sort_header_tool, gops_intersect_1, tp_tail_tool, CONVERTER_interval_to_bed_0, Add_a_column1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>19978</td>
      <td>32898</td>
      <td>3.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>Data Manipulation Olympics</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/data-manipulation-olympics/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-07-20</td>
      <td>2025-05-16</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>join1, {{version_cut_columns}}, tp_sorted_uniq, wc_gnu, upload1, {{version_split}}, {{version_join}}, {{version_wc}}, datamash_ops, {{version_replace_text_line}}, Cut1, {{version_filter}}, {{version_datamash}}, {{version_cat}}, {{version_paste}}, Filter1, Show beginning1, {{version_remove_beginning}}, cat1, {{version_sort}}, tp_cat, tp_sort_header_tool, tp_split_on_column, Add_a_column1, {{version_cut_advanced}}, Count1, {{version_unique}}, Grouping1, Remove beginning1, {{version_replace_text_column}}, regexColumn1, {{version_select_first}}, tabular_to_csv, {{version_compute}}, {{version_remove_columns_by_header}}, {{version_count}}</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>5682</td>
      <td>8322</td>
      <td>3.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>Introduction to Genomics and Galaxy</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/galaxy-intro-strands/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-10-18</td>
      <td>2024-10-15</td>
      <td>41</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>cat1, gene2exon1, tp_cat, gops_intersect_1, Filter1</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>MISSISSIPPI</td>
      <td>0</td>
      <td></td>
      <td>19642</td>
      <td>31147</td>
      <td>3.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>NGS data logistics</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/galaxy-intro-ngs-data-managment/tutorial.html</td>
      <td></td>
      <td>Data handling, Sequence contamination filtering, Genome indexing, Sequence alignment, Read mapping, Generation, Formatting, Sequencing quality control, SNP detection, Validation</td>
      <td>2017-02-22</td>
      <td>2025-07-24</td>
      <td>38</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>fastp, bwa, picard_MarkDuplicates, lofreq_call, collapse_dataset, bwa_mem2, lofreq_indelqual, snpSift_extractFields, fasterq_dump, snpEff, snpeff_sars_cov_2, multiqc, samtools_view, bwa_mem, samtools_stats, lofreq_viterbi, Filter1, snpEff_build_gb</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>MISSISSIPPI, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>19494</td>
      <td>33442</td>
      <td>3.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>How to reproduce published Galaxy analyses</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/galaxy-reproduce/tutorial.html</td>
      <td></td>
      <td>Visualisation</td>
      <td>2021-08-25</td>
      <td>2024-10-15</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Cut1, Grouping1, tp_sorted_uniq, Remove beginning1, upload1, csv_to_tabular, ggplot2_point, datamash_ops</td>
      <td>Galaxy@AuBi, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>MISSISSIPPI, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2127</td>
      <td>2850</td>
      <td>4.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>Introduction to Galaxy</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2016-09-30</td>
      <td>2025-03-10</td>
      <td>5</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2723343</td>
      <td>7314516</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>Options for using Galaxy</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/options-for-using-galaxy/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-05-24</td>
      <td>2022-09-23</td>
      <td>14</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2723343</td>
      <td>7314516</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>IGV Introduction</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/igv-introduction/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2016-03-30</td>
      <td>2023-02-17</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>7868</td>
      <td>11154</td>
      <td>1.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>Upload data to Galaxy</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/dummy/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2023-11-09</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>421</td>
      <td>481</td>
      <td>2.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>A short introduction to Galaxy</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/galaxy-intro-short/tutorial.html</td>
      <td></td>
      <td>Sequencing quality control, Statistical calculation, Sequence composition calculation</td>
      <td>2018-08-27</td>
      <td>2025-03-31</td>
      <td>43</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>cshl_fastq_quality_filter, fastqc</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>Galaxy@Pasteur, GalaxyTrakr, MISSISSIPPI, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2774605</td>
      <td>7394355</td>
      <td>7.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>Galaxy Basics for everyone</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/galaxy-intro-101-everyone/tutorial.html</td>
      <td></td>
      <td>Visualisation</td>
      <td>2019-11-14</td>
      <td>2024-12-02</td>
      <td>34</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Cut1, Grouping1, tp_sorted_uniq, Remove beginning1, upload1, csv_to_tabular, ggplot2_point, datamash_ops</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, MISSISSIPPI, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>21598</td>
      <td>34365</td>
      <td>4.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>Galaxy Basics for genomics</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/galaxy-intro-101/tutorial.html</td>
      <td></td>
      <td>Mapping</td>
      <td>2016-12-19</td>
      <td>2025-04-22</td>
      <td>68</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>bedtools_intersectbed, upload1, tp_sort_header_tool, tp_head_tool, datamash_ops, comp1</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>45622</td>
      <td>73791</td>
      <td>3.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Introduction to Galaxy Analyses</td>
      <td>Very Short Introductions: QC</td>
      <td>https://training.galaxyproject.org//topics/introduction/tutorials/vsi_qc/tutorial.html</td>
      <td></td>
      <td>Validation, Sequence contamination filtering, Sequencing quality control</td>
      <td>2024-03-26</td>
      <td>2024-06-25</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>fastp, multiqc</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>MISSISSIPPI, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>278</td>
      <td>336</td>
      <td>3.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Materials Science</td>
      <td>Finding the muon stopping site with pymuon-suite in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/materials-science/tutorials/muon-stopping-sites-muairss-uep/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-01-22</td>
      <td>2024-06-14</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>pm_uep_opt, pm_muairss_read</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>231</td>
      <td>311</td>
      <td>2.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Mass spectrometry: LC-MS analysis</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/lcms/tutorial.html</td>
      <td></td>
      <td>Chromatogram visualisation, Label-free quantification, Chromatographic alignment, Peak detection, Validation, Mass spectrum visualisation, Data filtering</td>
      <td>2019-07-02</td>
      <td>2024-10-15</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Univariate, quality_metrics, Batch_correction, wsdl_hmdb, msnbase_readmsdata, abims_xcms_fillPeaks, abims_xcms_group, abims_xcms_xcmsSet, generic_filter, abims_xcms_retcor, xcms_plot_chromatogram, xcms_merge, abims_CAMERA_annotateDiffreport</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>12325</td>
      <td>20202</td>
      <td>1.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Molecular formula assignment and mass recalibration with MFAssignR package</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/mfassignr/tutorial.html</td>
      <td></td>
      <td>Visualisation</td>
      <td>2025-05-19</td>
      <td>2025-05-19</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mfassignr_recal, mfassignr_snplot, mfassignr_isofiltr, mfassignr_mfassignCHO, mfassignr_histnoise, mfassignr_kmdnoise, mfassignr_recallist, mfassignr_findRecalSeries, mfassignr_mfassign</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>59</td>
      <td>73</td>
      <td>3.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Mass spectrometry imaging: Examining the spatial distribution of analytes</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/msi-analyte-distribution/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-02-21</td>
      <td>2024-10-15</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Filter1, cardinal_quality_report, cardinal_mz_images, cardinal_data_exporter, cardinal_spectra_plots, cardinal_filtering</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2927</td>
      <td>4181</td>
      <td>1.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Mass spectrometry: GC-MS data processing (with XCMS, RAMClustR, RIAssigner, and matchms)</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.html</td>
      <td></td>
      <td>Label-free quantification, Clustering, Peak detection, Chromatographic alignment, Correlation, Standardisation and normalisation, Validation, Imputation, Mass spectrum visualisation, Data filtering, Chromatogram visualisation, Filtering, Spectral library search, Formatting, Format validation</td>
      <td>2023-05-08</td>
      <td>2023-11-09</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ramclustr_define_experiment, waveica, msnbase_readmsdata, abims_xcms_group, abims_xcms_fillPeaks, abims_xcms_xcmsSet, ramclustr, msconvert, matchms_formatter, matchms_similarity, riassigner, matchms_filtering, abims_xcms_retcor, xcms_merge</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2507</td>
      <td>3239</td>
      <td>1.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Mass spectrometry imaging: Finding differential analytes</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/msi-finding-nglycans/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-04-13</td>
      <td>2023-11-09</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>maldi_quant_peak_detection, Summary_Statistics1, cardinal_preprocessing, cardinal_classification, Grep1, cardinal_quality_report, cardinal_mz_images, join_files_on_column_fuzzy, cardinal_segmentations, cardinal_combine, maldi_quant_preprocessing, Filter1</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1928</td>
      <td>2773</td>
      <td>1.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Predicting EI+ mass spectra with QCxMS</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/qcxms-predictions/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-10-01</td>
      <td>2024-10-01</td>
      <td>1</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>qcxms_getres, param_value_from_file, openbabel_compound_convert, tp_cut_tool, qcxms_production_run, xtb_molecular_optimization, ctb_im_conformers, qcxms_neutral_run, tp_cat, split_file_to_collection, __FILTER_FAILED_DATASETS__</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>590720</td>
      <td>1425510</td>
      <td>3.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Mass spectrometry: LC-MS preprocessing with XCMS</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/lcms-preprocessing/tutorial.html</td>
      <td></td>
      <td>Chromatogram visualisation, Label-free quantification, Chromatographic alignment, Peak detection, Validation, Mass spectrum visualisation, Data filtering</td>
      <td>2020-02-04</td>
      <td>2024-10-15</td>
      <td>14</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>msnbase_readmsdata, abims_xcms_group, abims_xcms_fillPeaks, abims_xcms_xcmsSet, abims_xcms_summary, intens_check, idchoice, xcms_export_samplemetadata, abims_xcms_retcor, xcms_plot_chromatogram, xcms_merge, abims_CAMERA_annotateDiffreport</td>
      <td>UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2729260</td>
      <td>7323867</td>
      <td>4.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Introduction to Metabolomics</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2018-11-20</td>
      <td>2023-01-12</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2723343</td>
      <td>7314516</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Mass spectrometry: LC-MS data processing</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/lcms-dataprocessing/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-12-14</td>
      <td>2024-10-15</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>normalization, quality_metrics, Batch_correction, intens_check, Multivariate, generic_filter</td>
      <td>UseGalaxy.fr</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2686</td>
      <td>3641</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Metabolomics</td>
      <td>Mass spectrometry : GC-MS analysis with metaMS package</td>
      <td>https://training.galaxyproject.org//topics/metabolomics/tutorials/gcms/tutorial.html</td>
      <td></td>
      <td>Chromatogram visualisation, Label-free quantification, Chromatographic alignment, Peak detection, Validation, Mass spectrum visualisation, Data filtering</td>
      <td>2021-09-22</td>
      <td>2023-11-09</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>msnbase_readmsdata, abims_xcms_group, abims_xcms_fillPeaks, abims_xcms_xcmsSet, xcms_merge</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1425</td>
      <td>2323</td>
      <td>1.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Introduction to Microbiome Analysis</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-06-22</td>
      <td>2024-08-08</td>
      <td>5</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2723343</td>
      <td>7314517</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Analyses of metagenomics data - The global picture</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/general-tutorial/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Phylogenetic analysis, Sequence read processing, Sequence clustering, Sequencing quality control, DNA barcoding, Visualisation, Taxonomic classification</td>
      <td>2017-06-22</td>
      <td>2024-08-09</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>krona-text, mothur_cluster_split, mothur_align_seqs, mothur_make_group, mothur_make_shared, metaphlan2, mothur_unique_seqs, mothur_filter_seqs, humann2, mothur_make_biom, taxonomy_krona_chart, humann2_regroup_table, metaphlan2krona, mothur_count_seqs, humann2_renorm_table, mothur_summary_seqs, mothur_merge_files, mothur_pre_cluster, mothur_classify_otu, mothur_screen_seqs, mothur_classify_seqs</td>
      <td></td>
      <td>UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>7604</td>
      <td>10354</td>
      <td>2.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Data handling, Read pre-processing, Variant calling, Primer removal, Sequence contamination filtering, Sequence assembly validation, Sequence alignment analysis, Sequence assembly visualisation, Genome assembly, Statistical calculation, Sequence file editing, Formatting, Sequence composition calculation, Sequencing quality control, Read mapping, Visualisation, Sequence trimming, Local alignment</td>
      <td>2022-12-05</td>
      <td>2025-06-16</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>quast, filter_tabular, bamtools, fastqc, seqtk_subseq, cutadapt, collection_column_join, megahit, coverm_contig, metaspades, bowtie2, ngsutils_bam_filter, bandage_info, tp_cat, random_lines1, bandage_image, megahit_contig2fastg, bg_uniq</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>3894</td>
      <td>5423</td>
      <td>2.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Query an annotated mobile genetic element database to identify and annotate genetic elements (e.g. plasmids) in metagenomics data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metaplasmidome_query/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence analysis, Mobile genetic elements</td>
      <td>Pairwise sequence alignment, Visualisation</td>
      <td>2025-01-07</td>
      <td>2025-01-07</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>join1, Cut1, Grouping1, tp_sorted_uniq, histogram_rpy, count_gff_features, ggplot2_histogram, tab2fasta, cat1, CONVERTER_fasta_to_tabular, tp_replace_in_column, minimap2, Add_a_column1, MQoutputfilter, add_column_headers, tp_tail_tool, sort1, Filter1</td>
      <td>UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>236</td>
      <td>276</td>
      <td>1.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Read pre-processing, Phylogenetic inference, Primer removal, Aggregation, Phylogenetic tree editing, Validation, Species frequency estimation, Sequence similarity search, Phylogenetic analysis, Sequence alignment analysis, Phylogenetic tree visualisation, Statistical calculation, Sequencing quality control, Sequence composition calculation, Visualisation, Sequence trimming, Conversion, Sequence comparison, Taxonomic classification</td>
      <td>2019-11-21</td>
      <td>2025-06-16</td>
      <td>6</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>humann_renorm_table, graphlan_annotate, fastqc, Grep1, multiqc, humann, humann_unpack_pathways, humann_regroup_table, Cut1, metaphlan, cutadapt, export2graphlan, fastq_paired_end_interlacer, taxonomy_krona_chart, tp_find_and_replace, tp_sort_header_tool, Grouping1, humann_split_stratified_table, combine_metaphlan_humann, humann_rename_table, graphlan, bg_sortmerna</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>2729297</td>
      <td>7322156</td>
      <td>4.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Clinical Metaproteomics 2: Discovery</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/clinical-mp-2-discovery/tutorial.html</td>
      <td></td>
      <td>Protein quantification, Filtering, Heat map generation, Clustering, Statistical calculation, Principal component plotting, Formatting, Standardisation and normalisation, Visualisation, Imputation</td>
      <td>2025-09-16</td>
      <td>2025-09-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Cut1, Grouping1, fasta_merge_files_and_filter_unique_sequences, maxquant, ident_params, fasta2tab, Remove beginning1, search_gui, filter_tabular, Grep1, query_tabular, msconvert, dbbuilder, tp_cat, peptide_shaker, fasta_cli, Filter1</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Clinical Metaproteomics 4: Quantitation</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/clinical-mp-4-quantitation/tutorial.html</td>
      <td></td>
      <td>Protein quantification, Heat map generation, Clustering, Statistical calculation, Principal component plotting, Standardisation and normalisation, Visualisation, Imputation</td>
      <td>2025-09-16</td>
      <td>2025-09-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>maxquant, Cut1, Grep1, Grouping1</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Identification of the micro-organisms in a beer using Nanopore sequencing</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/beer-data-analysis/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Visualisation, Aggregation, Statistical calculation, Sequence contamination filtering, Sequence composition calculation, Taxonomic classification, Sequencing quality control</td>
      <td>2022-09-29</td>
      <td>2024-11-29</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>fastp, krakentools_kreport2krona, fastqc, taxonomy_krona_chart, porechop, kraken2, Filter1</td>
      <td>GalaxyTrakr, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2438</td>
      <td>3557</td>
      <td>2.883333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Building an amplicon sequence variant (ASV) table from 16S data using DADA2</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/dada-16S/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>DNA barcoding, Variant calling, Visualisation, Deposition, Analysis</td>
      <td>2024-06-05</td>
      <td>2025-07-03</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>dada2_mergePairs, phyloseq_from_dada2, dada2_learnErrors, __SORTLIST__, interactive_tool_phyloseq, dada2_dada, dada2_seqCounts, tp_replace_in_line, dada2_filterAndTrim, dada2_plotQualityProfile, tp_head_tool, __UNZIP_COLLECTION__, dada2_removeBimeraDenovo, dada2_assignTaxonomyAddspecies, cat1, tp_replace_in_column, Add_a_column1, Grouping1, collection_element_identifiers, dada2_makeSequenceTable</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>5951</td>
      <td>8388</td>
      <td>2.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Clinical Metaproteomics 5: Data Interpretation</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/clinical-mp-5-data-interpretation/tutorial.html</td>
      <td></td>
      <td>Visualisation, Prediction and recognition</td>
      <td>2025-09-16</td>
      <td>2025-09-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>unipept, Grep1, msstatstmt</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial analysis with Nanopore data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/nanopore-16S-metagenomics/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Visualisation, Validation, Statistical calculation, Sequence contamination filtering, Sequence composition calculation, Taxonomic classification, Sequencing quality control</td>
      <td>2020-11-24</td>
      <td>2024-07-31</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>fastp, fastqc, Remove beginning1, multiqc, taxonomy_krona_chart, datamash_reverse, tp_replace_in_line, porechop, kraken2</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>7170</td>
      <td>9857</td>
      <td>2.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Clinical Metaproteomics 1: Database-Generation</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/clinical-mp-1-database-generation/tutorial.html</td>
      <td></td>
      <td>de Novo sequencing, Tag-based peptide identification, Protein identification, Target-Decoy, Expression analysis</td>
      <td>2025-09-16</td>
      <td>2025-09-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>dbbuilder, fasta_merge_files_and_filter_unique_sequences, uniprotxml_downloader, metanovo</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Identifying Mycorrhizal Fungi from ITS2 sequencing using LotuS2</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/lotus2-identifying-fungi/tutorial.html</td>
      <td></td>
      <td>Sequence feature detection, DNA barcoding</td>
      <td>2024-09-30</td>
      <td>2024-09-30</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>lotus2</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>UseGalaxy.be</td>
      <td>0</td>
      <td></td>
      <td>592</td>
      <td>747</td>
      <td>1.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Pathogen detection from (direct Nanopore) sequencing data using Galaxy - Foodborne Edition</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html</td>
      <td>Metagenomics, Public health and epidemiology, Taxonomy, Sequence assembly, Pathology, Sequence analysis</td>
      <td>Data handling, Phylogenetic tree generation (from molecular sequences), Box-Whisker plot plotting, Variant calling, Aggregation, Sequence contamination filtering, Sequence assembly, De-novo assembly, Pairwise sequence alignment, Phylogenetic tree generation, Phylogenetic tree generation (maximum likelihood and Bayesian methods), Multilocus sequence typing, Validation, Phylogenetic tree analysis, Mapping, Sequence alignment analysis, Scatter plot plotting, Multiple sequence alignment, Sequence assembly visualisation, Phylogenetic tree reconstruction, Antimicrobial resistance prediction, Genome assembly, Statistical calculation, Cross-assembly, Mapping assembly, Sequencing quality control, Sequence composition calculation, Visualisation, Base-calling, Taxonomic classification</td>
      <td>2023-01-26</td>
      <td>2024-09-27</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>barchart_gnuplot, krakentools_kreport2krona, tp_sorted_uniq, bamtools_split_mapped, fasttree, fastqc, regex1, Grep1, ggplot2_heatmap, multiqc, __BUILD_LIST__, __FILTER_EMPTY_DATASETS__, bedtools_getfastabed, table_compute, tp_multijoin_tool, Cut1, clustalw, add_line_to_file, collapse_dataset, tab2fasta, nanoplot, tp_head_tool, Paste1, mlst, split_file_to_collection, param_value_from_file, samtools_coverage, tp_replace_in_column, CONVERTER_gz_to_uncompressed, taxonomy_krona_chart, medaka_consensus_pipeline, tp_find_and_replace, bandage_image, flye, tp_split_on_column, Add_a_column1, tp_cut_tool, __FILTER_FAILED_DATASETS__, Count1, fastp, fasta_merge_files_and_filter_unique_sequences, Grouping1, bcftools_norm, compose_text_param, bcftools_consensus, fasta2tab, Remove beginning1, clair3, collection_element_identifiers, collection_column_join, snpSift_extractFields, snpSift_filter, regexColumn1, abricate, minimap2, porechop, samtools_fastx, krakentools_extract_kraken_reads, newick_display, samtools_depth, kraken2</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>4415</td>
      <td>6668</td>
      <td>3.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>MGnify v5.0 Amplicon Pipeline</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mgnify-amplicon/tutorial.html</td>
      <td>Metagenomics, Taxonomy, Sequence analysis</td>
      <td>Read pre-processing, Sequence contamination filtering, k-mer counting, Validation, Alignment, Mapping, Nucleic acid design, Comparison, Statistical calculation, Formatting, Sequence composition calculation, Sequencing quality control, DNA barcoding, Visualisation, Sequence trimming, Sequence feature detection, Nucleic acid feature detection, Analysis</td>
      <td>2025-07-30</td>
      <td>2025-07-30</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>wc_gnu, fastqc, CONVERTER_uncompressed_to_gz, seqprep, multiqc, __FILTER_EMPTY_DATASETS__, __MERGE_COLLECTION__, bedtools_getfastabed, fastq_dl, filter_tabular, infernal_cmsearch, lotus2, cshl_fasta_formatter, __UNZIP_COLLECTION__, biom_convert, ampvis2_heatmap, param_value_from_file, query_tabular, mapseq, prinseq, gops_concat_1, taxonomy_krona_chart, CONVERTER_gz_to_uncompressed, tp_find_and_replace, ampvis2_load, cmsearch_deoverlap, fastq_to_fasta_python, Add_a_column1, fastp, collapse_dataset, Grouping1, trimmomatic, bedtools_maskfastabed, collection_element_identifiers, collection_column_join, fastq_filter, tp_awk_tool, __FILTER_FROM_FILE__, mgnify_seqprep</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>57</td>
      <td>73</td>
      <td>3.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Taxonomic Profiling and Visualization of Metagenomic Data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/taxonomic-profiling/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Genome annotation, Visualisation, Aggregation, Statistical calculation, Taxonomic classification</td>
      <td>2023-05-03</td>
      <td>2025-04-24</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>__UNZIP_COLLECTION__, krakentools_kreport2krona, metaphlan, interactive_tool_phinch, est_abundance, taxonomy_krona_chart, kraken_biom, kraken2, interactive_tool_pavian</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>10949</td>
      <td>14680</td>
      <td>2.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Calculating α and β diversity from microbiome taxonomic data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/diversity/tutorial.html</td>
      <td></td>
      <td>Aggregation, Visualisation</td>
      <td>2024-08-01</td>
      <td>2025-06-17</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>krakentools_alpha_diversity, krakentools_beta_diversity</td>
      <td>Galaxy@Pasteur, GalaxyTrakr, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi</td>
      <td>0</td>
      <td></td>
      <td>5642</td>
      <td>6815</td>
      <td>1.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Antibiotic resistance detection</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/plasmid-metagenomics-nanopore/tutorial.html</td>
      <td>Metagenomic sequencing, Public health and epidemiology, Metagenomics, Microbiology, Infectious disease, Sequence analysis, Antimicrobial resistance</td>
      <td>Scatter plot plotting, Box-Whisker plot plotting, Sequence assembly visualisation, Aggregation, Genome assembly, Antimicrobial resistance prediction, Sequence analysis, Mapping assembly, De-novo assembly, Pairwise sequence alignment</td>
      <td>2019-06-25</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>minimap2, nanoplot, PlasFlow, bandage_image, gfa_to_fa, miniasm, unicycler, staramr_search, racon</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2966</td>
      <td>4160</td>
      <td>2.900000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics-short/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Read pre-processing, Phylogenetic inference, Primer removal, Phylogenetic tree editing, Validation, Species frequency estimation, Sequence similarity search, Phylogenetic analysis, Sequence alignment analysis, Phylogenetic tree visualisation, Statistical calculation, Sequencing quality control, Sequence composition calculation, Visualisation, Sequence trimming, Conversion, Sequence comparison, Taxonomic classification</td>
      <td>2020-02-13</td>
      <td>2025-06-16</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>humann_renorm_table, graphlan_annotate, fastqc, Grep1, multiqc, humann, humann_unpack_pathways, combine_metaphlan2_humann2, humann_regroup_table, Cut1, metaphlan, cutadapt, export2graphlan, fastq_paired_end_interlacer, taxonomy_krona_chart, tp_find_and_replace, tp_sort_header_tool, humann_split_stratified_table, humann_rename_table, graphlan, bg_sortmerna</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>661</td>
      <td>845</td>
      <td>2.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Clinical Metaproteomics 3: Verification</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/clinical-mp-3-verification/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-09-16</td>
      <td>2025-09-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Cut1, Grouping1, fasta_merge_files_and_filter_unique_sequences, collapse_dataset, Remove beginning1, pepquery2, query_tabular, dbbuilder, uniprotxml_downloader, tp_cat, Filter1</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Moving Pictures</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-moving-pictures/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td></td>
      <td>2024-03-14</td>
      <td>2024-03-14</td>
      <td>1</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>487</td>
      <td>753</td>
      <td>2.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Binning of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-binning/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Genome assembly, Statistical calculation, Validation, Sequence composition calculation, Sequence assembly validation, Sequencing quality control</td>
      <td>2023-12-05</td>
      <td>2025-05-15</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>megahit, checkm_lineage_wf</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>4996</td>
      <td>6155</td>
      <td>1.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Cancer Microbiome Intervention</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-cancer-microbiome-intervention/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td></td>
      <td>2024-02-12</td>
      <td>2024-03-14</td>
      <td>3</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>368</td>
      <td>595</td>
      <td>3.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Indexing and profiling microbes with MetaSBT</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metasbt/tutorial.html</td>
      <td>Microbial ecology, Taxonomy</td>
      <td></td>
      <td>2025-08-25</td>
      <td>2025-08-25</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>19</td>
      <td>24</td>
      <td>5.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (extended)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Phylogenetic analysis, Sequence read processing, Sequence clustering, Phylogenetic tree reconstruction, Sequencing quality control, DNA barcoding, Visualisation, Phylogenetic tree generation, Phylogenetic tree analysis, Taxonomic classification</td>
      <td>2017-02-12</td>
      <td>2024-03-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mothur_rarefaction_single, mothur_sub_sample, mothur_get_groups, mothur_remove_lineage, newick_display, mothur_cluster_split, mothur_align_seqs, mothur_venn, mothur_make_shared, XY_Plot_1, mothur_unique_seqs, mothur_filter_seqs, mothur_make_biom, mothur_remove_seqs, mothur_seq_error, taxonomy_krona_chart, mothur_summary_single, mothur_dist_shared, mothur_count_seqs, mothur_summary_seqs, mothur_tree_shared, mothur_pre_cluster, mothur_taxonomy_to_krona, mothur_classify_otu, mothur_count_groups, mothur_dist_seqs, mothur_heatmap_sim, mothur_chimera_vsearch, mothur_classify_seqs, mothur_cluster, mothur_screen_seqs, mothur_remove_groups, mothur_make_contigs</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>10981</td>
      <td>16121</td>
      <td>2.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop-short/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Phylogenetic analysis, Sequence read processing, Sequence clustering, Phylogenetic tree reconstruction, Sequencing quality control, DNA barcoding, Visualisation, Phylogenetic tree generation, Phylogenetic tree analysis, Taxonomic classification</td>
      <td>2019-05-13</td>
      <td>2024-06-14</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>mothur_rarefaction_single, mothur_sub_sample, mothur_remove_lineage, newick_display, mothur_cluster_split, mothur_venn, mothur_make_shared, XY_Plot_1, mothur_unique_seqs, mothur_filter_seqs, mothur_remove_seqs, mothur_summary_single, mothur_dist_shared, mothur_count_seqs, mothur_summary_seqs, mothur_tree_shared, mothur_pre_cluster, collapse_dataset, mothur_classify_otu, mothur_screen_seqs, mothur_count_groups, mothur_dist_seqs, mothur_heatmap_sim, mothur_chimera_vsearch, mothur_classify_seqs, mothur_cluster, mothur_remove_groups</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>4791</td>
      <td>7974</td>
      <td>4.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>M. tuberculosis Variant Analysis</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence assembly, Genetic variation, Public health and epidemiology, Infectious disease</td>
      <td>Sequence alignment analysis, Global alignment, Variant calling, Antimicrobial resistance prediction, Genome visualisation, Phylogenetic tree visualisation, Statistical calculation, Sequence contamination filtering, Sequence alignment, Sequence analysis, Sequence composition calculation, Sequencing quality control, Phylogenetic tree generation, Validation, Local alignment, Taxonomic classification</td>
      <td>2020-07-25</td>
      <td>2024-07-26</td>
      <td>30</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>tbvcfreport, fastqc, snippy, multiqc, upload1, tb_variant_filter, tp_sed_tool, jbrowse, qualimap_bamqc, EMBOSS: seqret84, __FLATTEN__, mosdepth, tb_profiler_profile, EMBOSS:%20seqret84, fastp, bcftools_consensus, jvarkit_wgscoverageplotter, samtools_stats, tp_awk_tool, kraken2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>7324</td>
      <td>11367</td>
      <td>3.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Pox virus genome analysis from tiled-amplicon sequencing data</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/pox-tiled-amplicon/tutorial.html</td>
      <td></td>
      <td>Sequence alignment analysis, Global alignment, Multiple sequence alignment, Primer removal, Sequence contamination filtering, Sequence analysis, Sequence alignment, Generation, Genome indexing, Read mapping, Sequencing quality control, Local alignment</td>
      <td>2023-05-15</td>
      <td>2024-03-20</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Grep1, upload1, EMBOSS: maskseq51, samtools_view, bwa_mem, datamash_ops, Cut1, fasta_regex_finder, qualimap_bamqc, fatovcf, ivar_consensus, fasta_compute_length, ivar_trim, rbc_mafft, param_value_from_file, mimodd_info, Add_a_column1, fastp, compose_text_param, samtools_merge, __ZIP_COLLECTION__, __APPLY_RULES__</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>928</td>
      <td>1107</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Avian influenza viral strain analysis from gene segment sequencing data</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/aiv-analysis/tutorial.html</td>
      <td></td>
      <td>Phylogenetic analysis, Sequence alignment analysis, Data handling, Multiple sequence alignment, Data retrieval, Primer removal, Sequence contamination filtering, Sequence analysis, Sequence alignment, Read mapping, De-novo assembly, Sequencing quality control, Generation, Genome indexing, Sequence file editing</td>
      <td>2022-11-21</td>
      <td>2024-10-04</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>Show beginning1, fastp, vapor, collapse_dataset, iqtree, seqtk_subseq, Grep1, __RELABEL_FROM_FILE__, qualimap_bamqc, upload1, samtools_view, bwa_mem, tp_find_and_replace, ivar_consensus, bamtools_split_ref, rbc_mafft</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1879</td>
      <td>2655</td>
      <td>2.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Whole transcriptome analysis of Arabidopsis thaliana</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/mirna-target-finder/tutorial.html</td>
      <td></td>
      <td>Read pre-processing, Primer removal, RNA-Seq analysis, Validation, Sequence composition calculation, Sequencing quality control, Read mapping, Gene expression analysis, RNA-Seq quantification, Visualisation, Sequence trimming, Differential gene expression analysis</td>
      <td>2021-04-08</td>
      <td>2025-08-10</td>
      <td>18</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>sample_seqs, Cut1, rbc_mirdeep2_mapper, targetfinder, cat1, salmon, trim_galore, filter_by_fasta_ids, deseq2, __MERGE_COLLECTION__, multiqc, rbc_mirdeep2_quantifier, falco, sort1, Filter1</td>
      <td>UseGalaxy.cz</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2583461</td>
      <td>6810757</td>
      <td>5.116667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Identification and Evolutionary Analysis of Transcription-Associated Proteins in Streptophyte algae and Land plants</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/tapscan-streptophyte-algae/tutorial.html</td>
      <td></td>
      <td>Phylogenetic analysis, Phylogenetic tree editing, Multiple sequence alignment, Phylogenetic inference (from molecular sequences), Sequence analysis</td>
      <td>2025-03-26</td>
      <td>2025-04-27</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Cut1, iqtree, Remove beginning1, quicktree, trimal, filter_by_fasta_ids, rbc_mafft, tapscan_classify, ete_treeviewer, Filter1</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>180</td>
      <td>228</td>
      <td>1.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>NCBI BLAST+ against the MAdLand</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.html</td>
      <td></td>
      <td>Sequence alignment analysis</td>
      <td>2023-01-16</td>
      <td>2024-07-31</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>bg_diamond, ncbi_blastp_wrapper, ncbi_blastx_wrapper</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>1537</td>
      <td>1942</td>
      <td>2.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Analysis of plant scRNA-Seq Data with Scanpy</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-plant/tutorial.html</td>
      <td></td>
      <td>Expression profile clustering, Differential gene expression analysis, Principal component visualisation</td>
      <td>2021-04-08</td>
      <td>2025-02-13</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>scanpy_cluster_reduce_dimension, scanpy_inspect, scanpy_remove_confounders, scanpy_normalize, scanpy_filter, scanpy_plot, anndata_import, anndata_manipulate</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2581</td>
      <td>3227</td>
      <td>1.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Statistical analysis of DIA data</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/DIA_Analysis_MSstats/tutorial.html</td>
      <td></td>
      <td>Differential protein expression profiling, Tag-based peptide identification, Spectrum calculation</td>
      <td>2020-12-02</td>
      <td>2023-11-09</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>msstats, Grep1, histogram_rpy, Filter1</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1285</td>
      <td>1814</td>
      <td>2.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>EncyclopeDIA</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/encyclopedia/tutorial.html</td>
      <td></td>
      <td>Filtering, Formatting</td>
      <td>2021-06-21</td>
      <td>2025-05-19</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>encyclopedia_searchtolib, encyclopedia_quantify, msconvert</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1920</td>
      <td>2631</td>
      <td>1.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Protein FASTA Database Handling</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/database-handling/tutorial.html</td>
      <td>Proteomics</td>
      <td></td>
      <td>2017-02-14</td>
      <td>2025-04-08</td>
      <td>30</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>fasta_merge_files_and_filter_unique_sequences, DecoyDatabase, fasta2tab, tab2fasta, dbbuilder, addValue</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>4588</td>
      <td>7322</td>
      <td>2.133333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Neoantigen 5b: IEDB binding PepQuery Validated Neopeptides</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/neoantigen-hla-binding-novel-peptides/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-01-14</td>
      <td>2025-07-02</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Cut1, Remove beginning1, pepquery2, query_tabular, table_compute, iedb_api, Filter1</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>19</td>
      <td>21</td>
      <td>2.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Label-free data analysis using MaxQuant</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/maxquant-label-free/tutorial.html</td>
      <td></td>
      <td>Protein quantification, Differential protein expression profiling, Heat map generation, Tag-based peptide identification, Clustering, Statistical calculation, Principal component plotting, Standardisation and normalisation, Visualisation, Imputation, Spectrum calculation</td>
      <td>2020-04-29</td>
      <td>2025-05-21</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>msstats, Cut1, maxquant, histogram_rpy, Grep1, tp_sort_header_tool, Add_a_column1, Filter1</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>19539</td>
      <td>28586</td>
      <td>1.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Peptide Library Data Analysis</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/peptide-library-data-analysis/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-01-22</td>
      <td>2023-11-09</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>pdaug_tsvtofasta, pdaug_fishers_plot, pdaug_peptide_data_access, pdaug_addclasslabel, pdaug_peptide_sequence_analysis, pdaug_merge_dataframes, pdaug_basic_plots, pdaug_sequence_property_based_descriptors</td>
      <td>UseGalaxy.be</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>699</td>
      <td>954</td>
      <td>2.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Peptide and Protein ID using OpenMS tools</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/protein-id-oms/tutorial.html</td>
      <td></td>
      <td>Filtering, Formatting</td>
      <td>2017-10-16</td>
      <td>2025-04-08</td>
      <td>32</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ConsensusID, IDScoreSwitcher, TextExporter, PeakPickerHiRes, MSGFPlusAdapter, FidoAdapter, Grep1, FalseDiscoveryRate, msconvert, IDMerger, XTandemAdapter, PeptideIndexer, IDPosteriorErrorProbability, IDFilter, FileInfo</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3303</td>
      <td>5568</td>
      <td>2.900000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Annotating a protein list identified by LC-MS/MS experiments</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/proteome_annotation/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-09-12</td>
      <td>2024-06-14</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>cluter_profiler, IDconverter, MQoutputfilter, retrieve_from_hpa, Jvenn, reactome_analysis, rna_abbased_data</td>
      <td>UseGalaxy.be</td>
      <td>UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3005</td>
      <td>4382</td>
      <td>2.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Mass spectrometry imaging: Loading and exploring MSI data</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/mass-spectrometry-imaging-loading-exploring-data/tutorial.html</td>
      <td>Proteomics</td>
      <td></td>
      <td>2018-11-27</td>
      <td>2025-04-08</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_sort_header_tool, cardinal_data_exporter, Grep1, cardinal_quality_report</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2660</td>
      <td>4137</td>
      <td>1.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Clinical Metaproteomics 2: Discovery</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/clinical-mp-2-discovery/tutorial.html</td>
      <td></td>
      <td>Protein quantification, Filtering, Heat map generation, Clustering, Statistical calculation, Principal component plotting, Formatting, Standardisation and normalisation, Visualisation, Imputation</td>
      <td>2024-02-06</td>
      <td>2025-08-10</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Cut1, Grouping1, fasta_merge_files_and_filter_unique_sequences, maxquant, ident_params, fasta2tab, Remove beginning1, search_gui, filter_tabular, Grep1, query_tabular, msconvert, dbbuilder, tp_cat, peptide_shaker, fasta_cli, Filter1</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>313</td>
      <td>424</td>
      <td>6.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Neoantigen 5a: Predicting HLA Binding</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/neoantigen-predicting-hla-binding/tutorial.html</td>
      <td></td>
      <td>Genetic variation analysis, Read mapping</td>
      <td>2025-01-14</td>
      <td>2025-07-02</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_awk_tool, query_tabular, optitype, seq2hla</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>32</td>
      <td>36</td>
      <td>0.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Single Cell Proteomics data analysis with bioconductor-scp</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/bioconductor-scp/tutorial.html</td>
      <td></td>
      <td>Visualisation, Expression analysis</td>
      <td>2025-02-07</td>
      <td>2025-02-10</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>Grouping1, bioconductor_scp</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>153</td>
      <td>200</td>
      <td>1.200000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Neoantigen 3: PepQuery2 Verification</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/neoantigen-peptide-verification/tutorial.html</td>
      <td></td>
      <td>Filtering, Formatting</td>
      <td>2025-01-14</td>
      <td>2025-07-02</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>pepquery2, query_tabular, tab2fasta, msconvert, ncbi_blastp_wrapper</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>28</td>
      <td>34</td>
      <td>3.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 2: Function</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-function/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics</td>
      <td>Query and retrieval, Heat map generation, Functional clustering, Quantification, Differential protein expression analysis, Filtering, Indexing, Statistical inference, Principal component visualisation, Visualisation</td>
      <td>2020-10-29</td>
      <td>2024-03-15</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_filter, metaquantome_sample, metaquantome_expand, metaquantome_db, metaquantome_stat, metaquantome_viz</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>402</td>
      <td>572</td>
      <td>4.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Clinical Metaproteomics 4: Quantitation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/clinical-mp-4-quantitation/tutorial.html</td>
      <td></td>
      <td>Protein quantification, Heat map generation, Clustering, Statistical calculation, Principal component plotting, Standardisation and normalisation, Visualisation, Imputation</td>
      <td>2024-02-06</td>
      <td>2025-08-10</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>maxquant, Cut1, Grep1, Grouping1</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>167</td>
      <td>234</td>
      <td>4.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>DIA Analysis using OpenSwathWorkflow</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/DIA_Analysis_OSW/tutorial.html</td>
      <td></td>
      <td>Filtering, Formatting</td>
      <td>2020-12-02</td>
      <td>2025-05-20</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Grep1, msconvert, pyprophet_score, pyprophet_protein, pyprophet_export, pyprophet_merge, pyprophet_peptide, OpenSwathWorkflow</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2957</td>
      <td>4168</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 1: Data creation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-data-creation/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Label-free quantification, Filtering, Visualisation, Prediction and recognition, Formatting</td>
      <td>2020-10-16</td>
      <td>2024-03-14</td>
      <td>20</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>unipept, Cut1, Remove beginning1, regex1, query_tabular, Grep1, msconvert, tp_replace_in_column, tp_replace_in_line, peptide_shaker, search_gui, flashlfq, Filter1</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>865</td>
      <td>1475</td>
      <td>3.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Neoantigen 1a: Fusion-Database-Generation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/neoantigen-fusion-database-generation/tutorial.html</td>
      <td></td>
      <td>Sequence alignment</td>
      <td>2025-01-14</td>
      <td>2025-07-02</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>arriba_get_filters, rna_star, regex1, query_tabular, tab2fasta, CONVERTER_gz_to_uncompressed, arriba, tp_awk_tool</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>95</td>
      <td>122</td>
      <td>1.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Machine Learning Modeling of Anticancer Peptides</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/ml-modeling-of-anti-cancer-peptides/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-01-22</td>
      <td>2023-11-09</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>pdaug_ml_models, pdaug_addclasslabel, pdaug_merge_dataframes, pdaug_basic_plots, pdaug_sequence_property_based_descriptors</td>
      <td>UseGalaxy.be</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1001</td>
      <td>1503</td>
      <td>2.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Clinical Metaproteomics 5: Data Interpretation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/clinical-mp-5-data-interpretation/tutorial.html</td>
      <td></td>
      <td>Visualisation, Prediction and recognition</td>
      <td>2024-02-06</td>
      <td>2025-08-10</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>unipept, Grep1, msstatstmt</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>260</td>
      <td>349</td>
      <td>2.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Neoantigen 4: Variant Annotation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/neoantigen-variant-annotation/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-01-14</td>
      <td>2025-07-02</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>query_tabular, tab2fasta, regexColumn1, Convert characters1, pep_pointer</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>47</td>
      <td>59</td>
      <td>1.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Proteogenomics 2: Database Search</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/proteogenomics-dbsearch/tutorial.html</td>
      <td></td>
      <td>Conversion, Peptide database search</td>
      <td>2018-11-20</td>
      <td>2025-05-08</td>
      <td>24</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>ident_params, query_tabular, mz_to_sqlite, tab2fasta, peptide_shaker, search_gui</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1485</td>
      <td>2218</td>
      <td>4.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Clinical Metaproteomics 1: Database-Generation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/clinical-mp-1-database-generation/tutorial.html</td>
      <td></td>
      <td>de Novo sequencing, Tag-based peptide identification, Protein identification, Target-Decoy, Expression analysis</td>
      <td>2024-02-06</td>
      <td>2025-07-30</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>dbbuilder, fasta_merge_files_and_filter_unique_sequences, uniprotxml_downloader, metanovo</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>523</td>
      <td>809</td>
      <td>4.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Library Generation for DIA Analysis</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/DIA_lib_OSW/tutorial.html</td>
      <td></td>
      <td>Protein quantification, Heat map generation, Clustering, Statistical calculation, Principal component plotting, Standardisation and normalisation, Visualisation, Imputation</td>
      <td>2020-12-02</td>
      <td>2025-06-02</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>diapysef, maxquant, OpenSwathAssayGenerator, OpenSwathDecoyGenerator, TargetedFileConverter, Filter1</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>4450</td>
      <td>5815</td>
      <td>1.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Proteogenomics 3: Novel peptide analysis</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-11-20</td>
      <td>2025-05-15</td>
      <td>20</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>query_tabular, ncbi_blastp_wrapper, peptide_genomic_coordinate, pep_pointer</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>1568</td>
      <td>2246</td>
      <td>2.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Secretome Prediction</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/secretome-prediction/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-04-13</td>
      <td>2025-04-08</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_sorted_uniq, tp_easyjoin_tool, tp_cut_tool, Grep1, bg_uniq, uniprot, Convert characters1, get_subontology_from, term_id_vs_term_name, comp1, Filter1, wolf_psort</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>624</td>
      <td>942</td>
      <td>1.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Peptide and Protein ID using SearchGUI and PeptideShaker</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/protein-id-sg-ps/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-06-12</td>
      <td>2025-04-08</td>
      <td>28</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>PeakPickerHiRes, Grep1, FileConverter, peptide_shaker, search_gui</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2973</td>
      <td>4823</td>
      <td>1.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Multiomics data analysis using MultiGSEA</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/multiGSEA-tutorial/tutorial.html</td>
      <td></td>
      <td>Aggregation, Gene-set enrichment analysis, Pathway analysis</td>
      <td>2025-03-10</td>
      <td>2025-03-11</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>multigsea, tp_grep_tool, Filter1</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>191</td>
      <td>207</td>
      <td>2.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>MaxQuant and MSstats for the analysis of TMT data</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/maxquant-msstats-tmt/tutorial.html</td>
      <td></td>
      <td>Protein quantification, Heat map generation, Clustering, Statistical calculation, Principal component plotting, Standardisation and normalisation, Visualisation, Imputation</td>
      <td>2021-08-13</td>
      <td>2025-01-08</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>maxquant, msstatstmt</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>4112</td>
      <td>5514</td>
      <td>1.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Neoantigen 1b: Non-Reference-Database-Generation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/neoantigen-non-reference-database-generation/tutorial.html</td>
      <td></td>
      <td>Transcriptome assembly, Variant calling, Statistical calculation, RNA-Seq analysis, Sequence annotation, Sequence alignment</td>
      <td>2025-01-14</td>
      <td>2025-07-02</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>fasta_merge_files_and_filter_unique_sequences, gffcompare, filter_tabular, freebayes, hisat2, fasta2tab, custom_pro_db, query_tabular, sqlite_to_tabular, stringtie, tab2fasta, regexColumn1, CONVERTER_gz_to_uncompressed, gffcompare_to_bed, tp_cat, translate_bed, bed_to_protein_map</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>42</td>
      <td>48</td>
      <td>0.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Metaproteomics tutorial</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaproteomics/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Visualisation, Prediction and recognition</td>
      <td>2017-06-28</td>
      <td>2024-06-14</td>
      <td>41</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>unipept, sqlite_to_tabular, query_tabular, peptide_shaker, search_gui</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2825</td>
      <td>4518</td>
      <td>2.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 3: Taxonomy</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-taxonomy/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics, Taxonomy</td>
      <td>Query and retrieval, Heat map generation, Functional clustering, Quantification, Differential protein expression analysis, Filtering, Indexing, Statistical inference, Principal component visualisation, Visualisation</td>
      <td>2020-10-29</td>
      <td>2024-03-15</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_filter, metaquantome_sample, metaquantome_expand, metaquantome_db, metaquantome_stat, metaquantome_viz</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>433</td>
      <td>571</td>
      <td>2.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Proteogenomics 1: Database Creation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/proteogenomics-dbcreation/tutorial.html</td>
      <td></td>
      <td>Transcriptome assembly, Variant calling, Statistical calculation, RNA-Seq analysis, Sequence annotation, Sequence alignment</td>
      <td>2018-11-20</td>
      <td>2025-05-08</td>
      <td>29</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>fasta_merge_files_and_filter_unique_sequences, gffcompare, filter_tabular, freebayes, hisat2, fasta2tab, custom_pro_db, query_tabular, sqlite_to_tabular, stringtie, tab2fasta, cat_multi_datasets, regexColumn1, tp_replace_in_column, gffcompare_to_bed, translate_bed, bed_to_protein_map</td>
      <td>UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>5646</td>
      <td>9312</td>
      <td>2.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Peptide and Protein Quantification via Stable Isotope Labelling (SIL)</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/protein-quant-sil/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-03-17</td>
      <td>2025-04-08</td>
      <td>32</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>TextExporter, IDScoreSwitcher, histogram_rpy, Summary_Statistics1, FileInfo, ProteinQuantifier, FidoAdapter, Grep1, FalseDiscoveryRate, MultiplexResolver, XTandemAdapter, PeptideIndexer, IDPosteriorErrorProbability, IDConflictResolver, FeatureFinderMultiplex, tp_tail_tool, IDMapper</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1316</td>
      <td>1853</td>
      <td>1.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Clinical Metaproteomics 3: Verification</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/clinical-mp-3-verification/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-02-06</td>
      <td>2025-08-10</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Cut1, Grouping1, fasta_merge_files_and_filter_unique_sequences, collapse_dataset, Remove beginning1, pepquery2, query_tabular, dbbuilder, uniprotxml_downloader, tp_cat, Filter1</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>177</td>
      <td>241</td>
      <td>6.183333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>MaxQuant and MSstats for the analysis of label-free data</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/maxquant-msstats-dda-lfq/tutorial.html</td>
      <td></td>
      <td>Protein quantification, Differential protein expression profiling, Heat map generation, Tag-based peptide identification, Clustering, Statistical calculation, Principal component plotting, Standardisation and normalisation, Visualisation, Imputation, Spectrum calculation</td>
      <td>2021-02-17</td>
      <td>2025-05-14</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>msstats, Cut1, maxquant, tp_easyjoin_tool, Summary_Statistics1, fasta2tab, Grep1, uniprot, tp_replace_in_column, ggplot2_heatmap2, datamash_ops, Filter1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>8248</td>
      <td>13278</td>
      <td>2.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Biomarker candidate identification</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/biomarker_selection/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-06-13</td>
      <td>2023-11-09</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>retr_pepatlas1, IDconverter, prot_features, MQoutputfilter, retrieve_from_hpa, Jvenn, rna_abbased_data, tp_cut_tool</td>
      <td></td>
      <td>UseGalaxy.be, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1590</td>
      <td>2164</td>
      <td>2.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Neoantigen 2: Database merge and FragPipe discovery</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/neoantigen-fragpipe-discovery/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-01-14</td>
      <td>2025-07-02</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>fasta_merge_files_and_filter_unique_sequences, collapse_dataset, Remove beginning1, query_tabular, fragpipe, Grep1, validate_fasta_database</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>87</td>
      <td>109</td>
      <td>1.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Detection and quantitation of N-termini (degradomics) via N-TAILS</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/ntails/tutorial.html</td>
      <td>Proteomics</td>
      <td></td>
      <td>2017-02-14</td>
      <td>2025-04-08</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ConsensusID, MSGFPlusAdapter, IDFilter, MzTabExporter, FileFilter, FalseDiscoveryRate, IDMerger, FileMerger, PeptideIndexer, HighResPrecursorMassCorrector, IDConflictResolver, FeatureFinderMultiplex, IDMapper</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>815</td>
      <td>1133</td>
      <td>1.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Introduction to proteomics, protein identification, quantification and statistical modelling</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2017-12-01</td>
      <td>2024-08-08</td>
      <td>3</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2723343</td>
      <td>7314519</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Label-free versus Labelled - How to Choose Your Quantitation Method</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/labelfree-vs-labelled/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-03-17</td>
      <td>2025-05-27</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>7579</td>
      <td>12282</td>
      <td>0.900000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Quality Control</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/quality-control/tutorial.html</td>
      <td></td>
      <td>Read pre-processing, Scatter plot plotting, Box-Whisker plot plotting, Primer removal, Statistical calculation, Sequencing quality control, Sequence composition calculation, Sequence trimming, Validation</td>
      <td>2016-10-04</td>
      <td>2025-03-29</td>
      <td>48</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>fastqe, fastqc, cutadapt, multiqc, pycoqc, nanoplot</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2826541</td>
      <td>7503960</td>
      <td>5.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Quality and contamination control in bacterial isolate using Illumina MiSeq Data</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/quality-contamination-control/tutorial.html</td>
      <td>Whole genome sequencing, Genomics, Microbiology, Microbial ecology</td>
      <td>Cross-assembly, Statistical calculation, Sequence contamination filtering, Expression analysis, Sequencing quality control, Read mapping, Visualisation, Taxonomic classification</td>
      <td>2024-07-15</td>
      <td>2025-03-20</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>fastp, upload1, est_abundance, recentrifuge, falco, kraken2</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1386</td>
      <td>1807</td>
      <td>2.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Mapping</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/mapping/tutorial.html</td>
      <td>Mapping</td>
      <td>Sequence alignment analysis, Read pre-processing, Data handling, Primer removal, Genome visualisation, Statistical calculation, Sequencing quality control, Sequence composition calculation, Read mapping, Sequence trimming, Validation</td>
      <td>2016-10-04</td>
      <td>2025-03-31</td>
      <td>43</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, fastqc, trim_galore, multiqc, bowtie2, samtools_stats, bamFilter</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2787204</td>
      <td>7422255</td>
      <td>5.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Primer and primer scheme design for pan-specific detection and sequencing of viral pathogens across genotypes</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/viral_primer_design/tutorial.html</td>
      <td></td>
      <td>PCR primer design, Sequence alignment analysis, Global alignment, Sequence analysis, Multiple sequence alignment, Local alignment, Sequence alignment</td>
      <td>2025-03-20</td>
      <td>2025-03-20</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>varvamp, EMBOSS: degapseq20, fasta_compute_length, rbc_mafft, ncbi_makeblastdb, split_file_to_collection</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>401</td>
      <td>474</td>
      <td>1.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Clean and manage Sanger sequences from raw files to aligned consensus</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.html</td>
      <td></td>
      <td>Phylogenetic analysis, Sequence read processing, Data handling, Sequence clustering, Sequencing quality control, Demultiplexing, DNA barcoding, Visualisation, Sequence file editing, Taxonomic classification</td>
      <td>2024-01-08</td>
      <td>2024-03-05</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ncbi_blastn_wrapper, mothur_degap_seqs, regex1, ab1_fastq_converter, fastq_masker_by_quality, qiime_align_seqs, cshl_fastq_to_fasta, __SORTLIST__, tab2fasta, unzip, seqtk_mergepe, cshl_fastx_reverse_complement, fastq_groomer, aligned_to_consensus, mothur_merge_files, collection_element_identifiers, seqtk_trimfq, filter_by_fasta_ids, __FILTER_FROM_FILE__, fastq_to_tabular</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2988</td>
      <td>3761</td>
      <td>1.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Screening assembled genomes for contamination using NCBI FCS</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/ncbi-fcs/tutorial.html</td>
      <td></td>
      <td>Sequence trimming, Sequence contamination filtering, Sequence assembly validation</td>
      <td>2024-04-16</td>
      <td>2024-04-16</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ncbi_fcs_adaptor, ncbi_fcs_gx</td>
      <td>GalaxyTrakr, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1631</td>
      <td>2223</td>
      <td>2.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Pre-processing of 10X Single-Cell ATAC-seq Datasets</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scatac-preprocessing-tenx/tutorial.html</td>
      <td></td>
      <td>Statistical calculation, Imputation, Sequence alignment, Genome indexing, Sequence composition calculation, Gene regulatory network analysis, Sequencing quality control, Enrichment analysis, Expression profile clustering, Principal component visualisation, Generation, Read mapping, Peak calling, Differential gene expression analysis, Mapping</td>
      <td>2023-04-24</td>
      <td>2025-02-13</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_sorted_uniq, sinto_fragments, fastqc, anndata_inspect, macs2_callpeak, scanpy_plot, sinto_barcode, bedtools_sortbed, bwa_mem, episcanpy_preprocess, episcanpy_build_matrix</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>5734</td>
      <td>6759</td>
      <td>1.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Single-cell quality control with scater</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-scater-qc/tutorial.html</td>
      <td></td>
      <td>Sequence visualisation, Read pre-processing, Sequencing quality control</td>
      <td>2019-10-23</td>
      <td>2025-02-14</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>scater_plot_dist_scatter, scater_create_qcmetric_ready_sce, scater_filter, scater_plot_pca</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>675</td>
      <td>764</td>
      <td>0.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Bulk RNA Deconvolution with MuSiC</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/bulk-music/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-02-11</td>
      <td>2025-02-14</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>music_inspect_eset, music_construct_eset, music_deconvolution</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3719</td>
      <td>4728</td>
      <td>1.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Converting between common single cell data formats</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-data-ingest/tutorial.html</td>
      <td></td>
      <td>Community profiling, Loading</td>
      <td>2024-02-13</td>
      <td>2025-03-11</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>join1, monocle3_create, Cut1, dropletutils, Filter1, scanpy_filter_cells, tabular_to_fastq, regexColumn1, dropletutils_read_10x, anndata_inspect, seurat_read10x, anndata_ops, datamash_transpose, sceasy_convert, tp_tail_tool, tp_cut_tool, fastq_to_tabular</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>2260</td>
      <td>2696</td>
      <td>1.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Pre-processing of 10X Single-Cell RNA Datasets</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-preprocessing-tenx/tutorial.html</td>
      <td></td>
      <td>Community profiling, Validation, Sequence alignment, Loading, Sequencing quality control</td>
      <td>2019-09-11</td>
      <td>2025-04-29</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>dropletutils, multiqc, rna_starsolo</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>11853</td>
      <td>15319</td>
      <td>1.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>GO Enrichment Analysis on Single-Cell RNA-Seq Data</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/GO-enrichment/tutorial.html</td>
      <td></td>
      <td>Gene-set enrichment analysis</td>
      <td>2024-09-17</td>
      <td>2025-04-08</td>
      <td>8</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>Cut1, tp_split_on_column, goenrichment, gprofiler_gost</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>615986</td>
      <td>1488276</td>
      <td>3.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Pseudobulk Analysis with Decoupler and EdgeR</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/pseudobulk-analysis/tutorial.html</td>
      <td></td>
      <td>Expression profile clustering, Differential gene expression analysis, Principal component visualisation</td>
      <td>2025-02-12</td>
      <td>2025-05-09</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>edger, anndata_inspect, scanpy_filter, tp_replace_in_column, decoupler_pseudobulk, volcanoplot, tp_replace_in_line, tp_awk_tool, column_remove_by_header, anndata_manipulate, __EXTRACT_DATASET__</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1254</td>
      <td>1459</td>
      <td>0.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Generating a single cell matrix using Alevin</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_alevin/tutorial.html</td>
      <td></td>
      <td>Gene expression analysis, RNA-Seq quantification, Sequence composition calculation</td>
      <td>2021-03-03</td>
      <td>2025-05-07</td>
      <td>24</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>join1, _salmon_kallisto_mtx_to_10x, Cut1, dropletutils_empty_drops, _dropletBarcodePlot, dropletutils_read_10x, anndata_inspect, sceasy_convert, _ensembl_gtf2gene_list, anndata_manipulate, alevin</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3329</td>
      <td>5067</td>
      <td>3.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Pre-processing of Single-Cell RNA Data</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-preprocessing/tutorial.html</td>
      <td></td>
      <td>Sequence alignment analysis, Data handling, RNA-Seq quantification, Validation, Read summarisation, Sequence alignment, Sequencing quality control</td>
      <td>2019-02-22</td>
      <td>2025-04-08</td>
      <td>5</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>featurecounts, __FLATTEN__, rna_star, collection_column_join, multiqc, umi_tools_extract, samtools_stats, tp_sed_tool, umi_tools_count, bamFilter</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2729452</td>
      <td>7322470</td>
      <td>4.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>An introduction to scRNA-seq data analysis</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-intro/slides.html</td>
      <td></td>
      <td></td>
      <td>2021-01-29</td>
      <td>2025-02-13</td>
      <td>8</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2690891</td>
      <td>7200479</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Generating a single cell matrix using Alevin and combining datasets (bash + R)</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/alevin-commandline/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-12-08</td>
      <td>2025-04-22</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>580</td>
      <td>815</td>
      <td>3.900000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Combining single cell datasets after pre-processing</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_alevin-combine-datasets/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-08</td>
      <td>2025-04-22</td>
      <td>22</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Cut1, anndata_inspect, tp_replace_in_column, anndata_ops, Paste1, anndata_manipulate</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>2408</td>
      <td>3365</td>
      <td>2.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Removing the effects of the cell cycle</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_cell-cycle/tutorial.html</td>
      <td></td>
      <td>Expression profile clustering, Differential gene expression analysis, Principal component visualisation</td>
      <td>2023-01-25</td>
      <td>2025-03-11</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>scanpy_cluster_reduce_dimension, scanpy_inspect, tp_easyjoin_tool, cat1, anndata_inspect, scanpy_plot, addValue, table_compute, scanpy_regress_variable, anndata_manipulate, sort1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2021</td>
      <td>2387</td>
      <td>0.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Filter, plot, and explore single cell RNA-seq data with Seurat (R)</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-10-02</td>
      <td>2025-03-11</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3090</td>
      <td>3534</td>
      <td>0.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Comparing inferred cell compositions using MuSiC deconvolution</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/bulk-music-4-compare/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-01-20</td>
      <td>2025-02-13</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>music_compare</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>264</td>
      <td>407</td>
      <td>1.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Importing files from public atlases</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/EBI-retrieval/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-11-14</td>
      <td>2025-03-11</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>scanpy_read_10x, tp_grep_tool, hca_matrix_downloader, regexColumn1, seurat_read10x, anndata_ops, tp_find_and_replace, sceasy_convert, retrieve_scxa</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>516</td>
      <td>688</td>
      <td>4.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Understanding Barcodes</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-umis/tutorial.html</td>
      <td></td>
      <td>Statistical calculation, Sequence composition calculation, Sequencing quality control</td>
      <td>2019-02-20</td>
      <td>2025-02-14</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>seq_filter_by_id, umi_tools_extract, fastqc</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>6121</td>
      <td>7635</td>
      <td>1.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Single-cell ATAC-seq standard processing with SnapATAC2</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scatac-standard-processing-snapatac2/tutorial.html</td>
      <td></td>
      <td>Gene expression profiling, Essential dynamics, Sequence alignment, Genome indexing, Dimensionality reduction, Expression profile clustering, Principal component visualisation, Generation, Read mapping, Differential gene expression analysis</td>
      <td>2024-07-11</td>
      <td>2025-07-02</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>scanpy_inspect, Cut1, snapatac2_plotting, snapatac2_preprocessing, snapatac2_clustering, replace_column_with_key_value_file, anndata_inspect, scanpy_filter, scanpy_normalize, scanpy_plot, bwa_mem, anndata_ops, fasta_compute_length, anndata_manipulate, snapatac2_peaks_and_motif</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1359</td>
      <td>1653</td>
      <td>1.116667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Clustering 3K PBMCs with Scanpy</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-scanpy-pbmc3k/tutorial.html</td>
      <td></td>
      <td>Expression profile clustering, Differential gene expression analysis, Principal component visualisation</td>
      <td>2019-12-19</td>
      <td>2025-04-29</td>
      <td>16</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>scanpy_cluster_reduce_dimension, scanpy_inspect, scanpy_remove_confounders, anndata_inspect, scanpy_filter, scanpy_normalize, scanpy_plot, anndata_import, datamash_ops, anndata_manipulate</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2731067</td>
      <td>7325571</td>
      <td>4.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Downstream Single-cell RNA analysis with RaceID</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-raceid/tutorial.html</td>
      <td></td>
      <td>Expression profile clustering, Expression analysis, Molecular dynamics</td>
      <td>2019-03-25</td>
      <td>2025-02-14</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_sed_tool, raceid_inspecttrajectory, raceid_filtnormconf, raceid_clustering, datamash_transpose, raceid_trajectory, raceid_inspectclusters, tp_head_tool, datamash_ops</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1039</td>
      <td>1230</td>
      <td>1.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Filter, plot and explore single-cell RNA-seq data with Scanpy (Python)</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case-jupyter_basic-pipeline/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-08-25</td>
      <td>2025-03-11</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3090</td>
      <td>3668</td>
      <td>1.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Scanpy Parameter Iterator</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scanpy_parameter_iterator/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-07-19</td>
      <td>2025-02-13</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>scanpy_run_umap, scanpy_plot_embed, scanpy_run_tsne, scanpy_find_cluster, scanpy_parameter_iterator, scanpy_compute_graph</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>259</td>
      <td>293</td>
      <td>1.200000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Converting NCBI Data to the AnnData Format</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-ncbi-anndata/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-12-13</td>
      <td>2025-03-11</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Cut1, scanpy_filter_cells, unzip, anndata_inspect, tp_replace_in_column, anndata_ops, anndata_import, Paste1, anndata_manipulate</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>403</td>
      <td>476</td>
      <td>1.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Filter, plot and explore single-cell RNA-seq data with Scanpy</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_basic-pipeline/tutorial.html</td>
      <td></td>
      <td>Expression profile clustering, Differential gene expression analysis, Principal component visualisation</td>
      <td>2021-03-24</td>
      <td>2025-04-23</td>
      <td>24</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>join1, scanpy_inspect, scanpy_filter, scanpy_plot, anndata_manipulate, Cut1, scanpy_normalise_data, scanpy_run_tsne, anndata_inspect, scanpy_find_cluster, scanpy_run_pca, scanpy_run_umap, anndata_ops, retrieve_scxa, scanpy_cluster_reduce_dimension, scanpy_find_markers, scanpy_filter_cells, scanpy_plot_embed, interactive_tool_cellxgene_vip, scanpy_filter_genes, scanpy_normalize, scanpy_compute_graph, scanpy_scale_data, scanpy_find_variable_genes</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>5081</td>
      <td>7752</td>
      <td>2.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Multi-sample batch correction with Harmony and SnapATAC2</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scatac-batch-correction-snapatac2/tutorial.html</td>
      <td></td>
      <td>Gene expression profiling, Essential dynamics, Dimensionality reduction</td>
      <td>2025-02-28</td>
      <td>2025-07-02</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Show beginning1, snapatac2_plotting, snapatac2_preprocessing, snapatac2_peaks_and_motif, collection_element_identifiers, snapatac2_clustering, anndata_inspect, fasta_compute_length, __FILTER_FROM_FILE__, anndata_manipulate, __EXTRACT_DATASET__</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>372</td>
      <td>421</td>
      <td>0.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Evaluating Reference Data for Bulk RNA Deconvolution</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/bulk-deconvolution-evaluate/tutorial.html</td>
      <td></td>
      <td>Visualisation, Annotation</td>
      <td>2025-02-02</td>
      <td>2025-03-11</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>join1, music_deconvolution, __DUPLICATE_FILE_TO_COLLECTION__, table_compute, tp_multijoin_tool, tp_sed_tool, annotatemyids, Cut1, ggplot2_point, Paste1, music_construct_eset, Add_a_column1, tp_cut_tool, Count1, collapse_dataset, tp_easyjoin_tool, Remove beginning1, ggplot2_violin, collection_column_join, datamash_transpose, tp_awk_tool, bg_column_arrange_by_header</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>87</td>
      <td>107</td>
      <td>1.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Clustering 3K PBMCs with Seurat</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-seurat-pbmc3k/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-01-23</td>
      <td>2025-05-09</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Cut1, seurat_create, seurat_plot, seurat_clustering, seurat_reduce_dimension, csv_to_tabular, seurat_data, seurat_preprocessing, Filter1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>MISSISSIPPI</td>
      <td>0</td>
      <td></td>
      <td>993</td>
      <td>1175</td>
      <td>1.183333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Filter, plot, and explore single cell RNA-seq data with Seurat</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_FilterPlotandExplore_SeuratTools/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-04-09</td>
      <td>2025-03-11</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>seurat_run_pca, seurat_plot, seurat_find_neighbours, seurat_find_clusters, seurat_find_markers, seurat_scale_data, seurat_read10x, seurat_normalise_data, seurat_find_variable_genes, seurat_filter_cells, seurat_run_umap, retrieve_scxa</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1697</td>
      <td>2072</td>
      <td>1.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Plates, Batches, and Barcodes</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-plates-batches-barcodes/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-02-16</td>
      <td>2025-02-13</td>
      <td>4</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2723343</td>
      <td>7314521</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Inferring single cell trajectories with Scanpy (Python)</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_JUPYTER-trajectories/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-04-07</td>
      <td>2025-03-11</td>
      <td>20</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>4343</td>
      <td>5152</td>
      <td>1.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Matrix Exchange Format to ESet | Creating a single-cell RNA-seq reference dataset for deconvolution</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/bulk-music-2-preparescref/tutorial.html</td>
      <td></td>
      <td>Annotation</td>
      <td>2023-01-20</td>
      <td>2024-02-13</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>join1, tp_sed_tool, Cut1, annotatemyids, scanpy_read_10x, tp_easyjoin_tool, add_line_to_file, regex1, anndata_inspect, music_manipulate_eset, datamash_transpose, retrieve_scxa, tp_awk_tool, music_construct_eset, sort1, tp_cut_tool, table_compute</td>
      <td></td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>712</td>
      <td>1210</td>
      <td>2.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Inferring single cell trajectories with Scanpy</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_trajectories/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-12-08</td>
      <td>2025-03-11</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>scanpy_plot_trajectory, scanpy_run_paga, scanpy_run_fdg, scanpy_plot_embed, scanpy_run_diffmap, scanpy_find_cluster, scanpy_compute_graph, scanpy_run_dpt, anndata_manipulate</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1190</td>
      <td>1516</td>
      <td>2.283333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Single-cell Formats and Resources</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-data-formats/slides.html</td>
      <td></td>
      <td></td>
      <td>2024-07-26</td>
      <td>2025-02-13</td>
      <td>4</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>685265</td>
      <td>1659292</td>
      <td>2.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Inferring single cell trajectories with Monocle3 (R)</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_monocle3-rstudio/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-04-12</td>
      <td>2025-03-11</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>6037</td>
      <td>6873</td>
      <td>0.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Inferring single cell trajectories with Monocle3</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case_monocle3-trajectories/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-30</td>
      <td>2025-04-23</td>
      <td>24</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>join1, monocle3_create, Cut1, Filter1, monocle3_preprocess, monocle3_partition, monocle3_learnGraph, monocle3_reduceDim, regexColumn1, anndata_inspect, monocle3_topmarkers, datamash_transpose, monocle3_diffExp, monocle3_plotCells, tp_cut_tool, monocle3_orderCells</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3805</td>
      <td>4733</td>
      <td>1.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Bulk matrix to ESet | Creating the bulk RNA-seq dataset for deconvolution</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/bulk-music-3-preparebulk/tutorial.html</td>
      <td></td>
      <td>Annotation</td>
      <td>2023-01-20</td>
      <td>2024-02-13</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_sed_tool, annotatemyids, tp_easyjoin_tool, regex1, music_manipulate_eset, tp_awk_tool, column_remove_by_header, music_construct_eset, sort1, tp_cut_tool, table_compute</td>
      <td></td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>653</td>
      <td>1058</td>
      <td>1.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Trajectory analysis</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-trajectories/slides.html</td>
      <td></td>
      <td></td>
      <td>2022-09-30</td>
      <td>2025-02-13</td>
      <td>4</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1767525</td>
      <td>4228207</td>
      <td>2.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Single Cell</td>
      <td>Automated Cell Annotation</td>
      <td>https://training.galaxyproject.org//topics/single-cell/tutorials/scrna-case-cell-annotation/slides.html</td>
      <td></td>
      <td></td>
      <td>2023-09-04</td>
      <td>2025-02-27</td>
      <td>7</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1165288</td>
      <td>2780199</td>
      <td>2.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Machine learning: classification and regression</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/classification_regression/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-03-07</td>
      <td>2025-04-08</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>plotly_regression_performance_plots, sklearn_ensemble, sklearn_svm_classifier, plotly_ml_performance_plots</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>5635</td>
      <td>10167</td>
      <td>0.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Clustering in Machine Learning</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/clustering_machinelearning/tutorial.html</td>
      <td></td>
      <td>Visualisation</td>
      <td>2020-05-08</td>
      <td>2025-04-08</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>csv_to_tabular, ggplot2_point, sklearn_numeric_clustering</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>8369</td>
      <td>13926</td>
      <td>0.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Building the LORIS LLR6 PanCancer Model Using PyCaret</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/loris_model/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-12-13</td>
      <td>2025-05-08</td>
      <td>2</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>PyCaret_Model_Comparison</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>473952</td>
      <td>1118780</td>
      <td>4.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Unsupervised Analysis of Bone Marrow Cells with Flexynesis</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/flexynesis_unsupervised/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-08-10</td>
      <td>2025-08-13</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>flexynesis, flexynesis_plot, __EXTRACT_DATASET__, flexynesis_utils</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>16</td>
      <td>20</td>
      <td>10.200000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Introduction to deep learning</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/intro_deep_learning/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-03-26</td>
      <td>2025-04-08</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>keras_model_config, keras_model_builder, model_prediction, ml_visualization_ex, keras_train_and_eval</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2373</td>
      <td>3044</td>
      <td>1.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Deep Learning (Part 1) - Feedforward neural networks (FNN)</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/FNN/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-04-28</td>
      <td>2024-06-14</td>
      <td>12</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>plotly_regression_performance_plots, keras_model_config, keras_model_builder, model_prediction, keras_train_and_eval</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2585675</td>
      <td>6771589</td>
      <td>3.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Deep Learning (Part 2) - Recurrent neural networks (RNN)</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/RNN/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-02-23</td>
      <td>2025-05-10</td>
      <td>15</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>keras_model_config, keras_model_builder, model_prediction, ml_visualization_ex, keras_train_and_eval</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2656309</td>
      <td>7064071</td>
      <td>3.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Age prediction using machine learning</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/age-prediction-with-ml/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-01-25</td>
      <td>2025-04-08</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>plotly_regression_performance_plots, plotly_parallel_coordinates_plot, sklearn_ensemble, sklearn_build_pipeline, sklearn_searchcv</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>13111</td>
      <td>21172</td>
      <td>0.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Modeling Breast Cancer Subtypes with Flexynesis</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/flexynesis_classification/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-08-13</td>
      <td>2025-09-02</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Show beginning1, join1, tp_easyjoin_tool, tabpfn, flexynesis, pick_value, flexynesis_plot, datamash_transpose, tp_sort_header_tool, __EXTRACT_DATASET__, tp_cut_tool, table_compute</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>27</td>
      <td>51</td>
      <td>3.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Deep Learning (Part 3) - Convolutional neural networks (CNN)</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/CNN/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-04-19</td>
      <td>2025-05-09</td>
      <td>19</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>keras_model_config, keras_model_builder, model_prediction, ml_visualization_ex, sklearn_to_categorical, keras_train_and_eval</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2581308</td>
      <td>6782690</td>
      <td>3.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Interval-Wise Testing for omics data</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/iwtomics/tutorial.html</td>
      <td></td>
      <td>Comparison, Differentially-methylated region identification, Peak calling, Genome annotation, Differential gene expression analysis</td>
      <td>2018-06-14</td>
      <td>2023-11-09</td>
      <td>17</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>iwtomics_loadandplot, iwtomics_testandplot, iwtomics_plotwithscale</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>360</td>
      <td>482</td>
      <td>3.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Train and Test a Deep learning image classifier with Galaxy-Ludwig</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/galaxy-ludwig/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-10-28</td>
      <td>2024-12-11</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ludwig_experiment</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>148</td>
      <td>191</td>
      <td>2.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Image classification in Galaxy with fruit 360 dataset</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/fruit_360/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-12-01</td>
      <td>2024-07-09</td>
      <td>8</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>keras_model_config, keras_model_builder, model_prediction, ml_visualization_ex, sklearn_to_categorical, keras_train_and_eval, tp_cut_tool</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2242060</td>
      <td>5685583</td>
      <td>3.200000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Basics of machine learning</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/machinelearning/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-11-05</td>
      <td>2025-05-13</td>
      <td>15</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>sklearn_svm_classifier</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2726016</td>
      <td>7318146</td>
      <td>5.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Text-mining with the SimText toolset</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/text-mining_simtext/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-04-05</td>
      <td>2025-05-07</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>pubmed_by_queries, interactive_tool_simtext_app, pmids_to_pubtator_matrix</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>452</td>
      <td>620</td>
      <td>2.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>A Docker-based interactive Jupyterlab powered by GPU for artificial intelligence in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/gpu_jupyter_lab/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-04-06</td>
      <td>2025-04-08</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Cut1, Filter1, interactive_tool_ml_jupyter_notebook</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1165</td>
      <td>1485</td>
      <td>1.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Regression in Machine Learning</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/regression_machinelearning/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-01-25</td>
      <td>2025-05-14</td>
      <td>19</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>plotly_regression_performance_plots, Remove beginning1, sklearn_ensemble, sklearn_build_pipeline, sklearn_generalized_linear, sklearn_searchcv</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2726536</td>
      <td>7318560</td>
      <td>3.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Identifing Survival Markers of Brain tumor with Flexynesis</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/flexynesis_survival/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-08-13</td>
      <td>2025-08-13</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>__EXTRACT_DATASET__, tp_replace_in_column, flexynesis, flexynesis_plot, datamash_ops, Add_a_column1</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>15</td>
      <td>17</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Prepare Data from CbioPortal for Flexynesis Integration</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/flexynesis_cbio_import/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2025-08-10</td>
      <td>2025-08-13</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>flexynesis_cbioportal_import, flexynesis_utils, tp_sort_header_tool, __EXTRACT_DATASET__, tp_cut_tool, table_compute</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>20</td>
      <td>20</td>
      <td>1.883333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Classification in Machine Learning</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/classification_machinelearning/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-04-30</td>
      <td>2025-05-14</td>
      <td>18</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>plotly_ml_performance_plots, Remove beginning1, sklearn_svm_classifier, sklearn_ensemble, sklearn_build_pipeline, sklearn_generalized_linear, sklearn_searchcv, sklearn_nn_classifier</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2726041</td>
      <td>7318493</td>
      <td>4.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>PAPAA PI3K_OG: PanCancer Aberrant Pathway Activity Analysis</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/aberrant_pi3k_pathway_analysis/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-05-06</td>
      <td>2023-11-09</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>pancancer_pathway_count_heatmaps, pancancer_map_mutation_class, pancancer_targene_summary_figures, pancancer_alternative_genes_pathwaymapper, pancancer_targene_pharmacology, pancancer_external_sample_status_prediction, pancancer_apply_weights, pancancer_visualize_decisions, pancancer_classifier, pancancer_within_disease_analysis, pancancer_targene_cell_line_predictions, pancancer_compare_within_models</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>763</td>
      <td>1201</td>
      <td>3.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Fine tune large protein model (ProtTrans) using HuggingFace</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/fine_tuning_protTrans/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-06-17</td>
      <td>2025-04-08</td>
      <td>4</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>interactive_tool_ml_jupyter_notebook</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>743424</td>
      <td>1800976</td>
      <td>3.566667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Supervised Learning with Hyperdimensional Computing</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/hyperdimensional_computing/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-04-28</td>
      <td>2023-11-09</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>chopin2</td>
      <td>UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>421</td>
      <td>525</td>
      <td>0.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Statistics and machine learning</td>
      <td>Introduction to Machine Learning using R</td>
      <td>https://training.galaxyproject.org//topics/statistics/tutorials/intro-to-ml-with-r/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-05-21</td>
      <td>2024-10-15</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>interactive_tool_rstudio</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>3974</td>
      <td>5963</td>
      <td>1.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Synthetic Biology</td>
      <td>Evaluating and ranking a set of pathways based on multiple metrics</td>
      <td>https://training.galaxyproject.org//topics/synthetic-biology/tutorials/pathway_analysis/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-26</td>
      <td>2024-06-14</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>rpthermo, rpscore, rpranker, rpfba</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>596</td>
      <td>897</td>
      <td>3.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Synthetic Biology</td>
      <td>Designing plasmids encoding predicted pathways by using the BASIC assembly method</td>
      <td>https://training.galaxyproject.org//topics/synthetic-biology/tutorials/basic_assembly_analysis/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-05-12</td>
      <td>2024-06-14</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>selenzy-wrapper, dnabot, rpbasicdesign</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>943</td>
      <td>1465</td>
      <td>2.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Synthetic Biology</td>
      <td>Generating theoretical possible pathways for the production of Lycopene in E.Coli using Retrosynthesis tools</td>
      <td>https://training.galaxyproject.org//topics/synthetic-biology/tutorials/retrosynthesis_analysis/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-06-23</td>
      <td>2024-06-14</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>rpextractsink, retropath2, rp2paths, get_sbml_model, rpcompletion, rrparser</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>514</td>
      <td>747</td>
      <td>2.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Workshop Kickoff</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/workshop-intro/slides.html</td>
      <td></td>
      <td></td>
      <td>2019-06-04</td>
      <td>2024-02-20</td>
      <td>5</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2723343</td>
      <td>7314525</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Overview of the Galaxy Training Material for Instructors</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/gtn-for-instructors/slides.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2024-08-08</td>
      <td>3</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1776965</td>
      <td>4255241</td>
      <td>2.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Set up a Galaxy for Training</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/setup-galaxy-for-training/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-06-04</td>
      <td>2023-11-09</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>296</td>
      <td>341</td>
      <td>3.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Hybrid training</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/hybrid-training/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2024-06-25</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>19</td>
      <td>21</td>
      <td>0.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Running a workshop as instructor</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/running-workshop/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-11-15</td>
      <td>2025-04-08</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>146</td>
      <td>274</td>
      <td>1.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Galaxy Admin Training</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/galaxy-admin-training/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-11-04</td>
      <td>2024-09-17</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>143</td>
      <td>156</td>
      <td>2.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Teaching experiences</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/experiences/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2022-09-23</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>270</td>
      <td>491</td>
      <td>2.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Live Coding is a Skill</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/live-coding/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2023-11-09</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>81</td>
      <td>85</td>
      <td>0.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Asynchronous training</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/gallantries-async/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-11-02</td>
      <td>2023-11-02</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6</td>
      <td>6</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Organizing a workshop</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/organize-workshop/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-11-15</td>
      <td>2023-11-09</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>245</td>
      <td>266</td>
      <td>1.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Teaching online</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/teaching-online/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2022-09-23</td>
      <td>2023-10-23</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>20</td>
      <td>20</td>
      <td>0.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Course Builder</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/gallantries-coursebuilder/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-11-02</td>
      <td>2023-11-02</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>113</td>
      <td>133</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Teaching and Hosting Galaxy training</td>
      <td>Training Infrastructure as a Service</td>
      <td>https://training.galaxyproject.org//topics/teaching/tutorials/setup-tiaas-for-training/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-11-15</td>
      <td>2023-12-20</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>208</td>
      <td>399</td>
      <td>6.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>RNA-RNA interactome data analysis</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-interactome/tutorial.html</td>
      <td></td>
      <td>Read pre-processing, Primer removal, Sequence trimming, Statistical calculation, Sequence composition calculation, Sequencing quality control</td>
      <td>2020-03-23</td>
      <td>2023-11-09</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>chira_merge, fastqc, query_tabular, cutadapt, chira_extract, chira_map, chira_collapse, chira_quantify</td>
      <td>UseGalaxy.be, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1432</td>
      <td>2182</td>
      <td>2.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>1: RNA-Seq reads to counts</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-reads-to-counts/tutorial.html</td>
      <td></td>
      <td>Read pre-processing, Data handling, Primer removal, Statistical calculation, Read summarisation, Sequence alignment, Formatting, Sequence composition calculation, Sequencing quality control, RNA-Seq quantification, Sequence trimming, Validation</td>
      <td>2018-09-23</td>
      <td>2024-08-09</td>
      <td>33</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>picard_MarkDuplicates, hisat2, fastqc, cutadapt, rseqc_geneBody_coverage, collection_column_join, rseqc_infer_experiment, multiqc, rseqc_read_distribution, featurecounts, samtools_idxstats</td>
      <td>UseGalaxy.eu</td>
      <td>MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>76914</td>
      <td>140395</td>
      <td>2.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Differential abundance testing of small RNAs</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/srna/tutorial.html</td>
      <td></td>
      <td>Read pre-processing, Primer removal, Statistical calculation, RNA-Seq analysis, Sequence alignment, Sequencing quality control, Sequence composition calculation, Gene expression analysis, RNA-Seq quantification, Sequence trimming, Differential gene expression analysis</td>
      <td>2017-06-29</td>
      <td>2023-11-09</td>
      <td>26</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>hisat2, fastqc, samtool_filter2, trim_galore, salmon, bedtools_bamtofastq, deseq2, fastq_manipulation, Filter1</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>6829</td>
      <td>12035</td>
      <td>2.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>CLIP-Seq data analysis from pre-processing to motif detection</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/clipseq/tutorial.html</td>
      <td></td>
      <td>Read pre-processing, Primer removal, Statistical calculation, Sequence alignment, Sequence composition calculation, Sequencing quality control, Sequence trimming, Mapping</td>
      <td>2018-08-17</td>
      <td>2023-11-09</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Extract genomic DNA 1, fastqc, rcas, __MERGE_COLLECTION__, deeptools_plot_fingerprint, sort1, __SORTLIST__, wig_to_bigWig, cutadapt, deeptools_plot_correlation, bedtools_slopbed, bedtools_genomecoveragebed_bedgraph, meme_chip, umi_tools_dedup, deeptools_multi_bam_summary, bctools_extract_alignment_ends, peakachu, rna_star, umi_tools_extract, tp_awk_tool</td>
      <td></td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>8081</td>
      <td>16048</td>
      <td>1.883333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Network analysis with Heinz</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/network-analysis-with-heinz/tutorial.html</td>
      <td></td>
      <td>Differential gene expression analysis, Pathway or network analysis, RNA-Seq analysis</td>
      <td>2018-10-11</td>
      <td>2024-01-15</td>
      <td>16</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>heinz, Cut1, heinz_bum, deseq2, heinz_scoring, heinz_visualization</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2725355</td>
      <td>7317672</td>
      <td>5.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>GO Enrichment Analysis</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/goenrichment/tutorial.html</td>
      <td></td>
      <td>Gene-set enrichment analysis</td>
      <td>2019-01-23</td>
      <td>2024-03-05</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>goslimmer, goenrichment, Filter1</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>22816</td>
      <td>37191</td>
      <td>1.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Genome-wide alternative splicing analysis</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.html</td>
      <td></td>
      <td>Data handling, Statistical calculation, RNA-Seq analysis, Sequence annotation, Sequence analysis, Formatting, Sequence composition calculation, De-novo assembly, Sequencing quality control, Protein sequence analysis, Sequence contamination filtering, Sequence alignment, Visualisation, Validation, Transcriptome assembly, Sequence comparison, Sequence assembly validation</td>
      <td>2023-05-17</td>
      <td>2025-03-28</td>
      <td>11</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>fastqc, cpat, rna_quast, multiqc, rseqc_read_distribution, gene2exon1, gffread, stringtie_merge, sort1, Cut1, gffcompare, rseqc_inner_distance, stringtie, addValue, isoformswitchanalyzer, pygenomeTracks, Filter1, rseqc_junction_saturation, __FLATTEN__, tp_grep_tool, rseqc_geneBody_coverage, cat1, pfamscan, rseqc_junction_annotation, tp_cat, Add_a_column1, fastp, gtftobed12, rna_star, tp_uniq_tool, Remove beginning1, rseqc_infer_experiment, collection_element_identifiers, tp_awk_tool, __FILTER_FROM_FILE__</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1368903</td>
      <td>3220067</td>
      <td>4.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>De novo transcriptome assembly, annotation, and differential expression analysis</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/full-de-novo/tutorial.html</td>
      <td></td>
      <td>Gene functional annotation, Statistical calculation, Coding region prediction, Sequence composition calculation, De-novo assembly, Sequencing quality control, de Novo sequencing, Validation, Transcriptome assembly</td>
      <td>2020-02-13</td>
      <td>2023-11-09</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>transdecoder, trinity_align_and_estimate_abundance, trinotate, trimmomatic, trinity_define_clusters_by_cutting_tree, fastqc, trinity_contig_exn50_statistic, trinity_filter_low_expr_transcripts, trinity_samples_qccheck, multiqc, trinity_run_de_analysis, trinity_analyze_diff_expr, describe_samples, trinity, trinity_abundance_estimates_to_matrix</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>14236</td>
      <td>25955</td>
      <td>2.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>RNA-Seq data analysis, clustering and visualisation tutorial</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-analysis-clustering-viz/tutorial.html</td>
      <td></td>
      <td>Gene functional annotation, RNA-Seq analysis, Differential gene expression analysis</td>
      <td>2025-05-28</td>
      <td>2025-06-16</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Show beginning1, Cut1, Grouping1, tp_easyjoin_tool, deg_annotate, cat1, deseq2, volcanoplot, length_and_gc_content, Multivariate, ggplot2_heatmap2, tp_sort_header_tool, goseq, Add_a_column1, Filter1, table_compute</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>569</td>
      <td>687</td>
      <td>1.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Visualization of RNA-Seq results with heatmap2</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-viz-with-heatmap2/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-12-31</td>
      <td>2023-11-09</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>join1, Cut1, datamash_transpose, ggplot2_heatmap2</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>76811</td>
      <td>126811</td>
      <td>1.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>3: RNA-seq genes to pathways</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-genes-to-pathways/tutorial.html</td>
      <td></td>
      <td>Gene-set enrichment analysis, Gene set testing, Gene functional annotation</td>
      <td>2018-12-31</td>
      <td>2023-11-09</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>join1, Cut1, fgsea, egsea, tp_sort_header_tool, goseq, Add_a_column1, tp_cut_tool</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>25702</td>
      <td>41348</td>
      <td>1.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Visualization of RNA-Seq results with Volcano Plot</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-viz-with-volcanoplot/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-12-31</td>
      <td>2025-04-17</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>volcanoplot</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>151131</td>
      <td>248880</td>
      <td>1.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>2: RNA-seq counts to genes</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-counts-to-genes/tutorial.html</td>
      <td></td>
      <td>RNA-Seq analysis, Annotation</td>
      <td>2018-12-31</td>
      <td>2024-04-10</td>
      <td>21</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>annotatemyids, limma_voom, tp_replace_in_line, mergeCols1, tp_cut_tool</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>42723</td>
      <td>72521</td>
      <td>2.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Small Non-coding RNA Clustering using BlockClust</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/small_ncrna_clustering/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-12-14</td>
      <td>2023-11-09</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_sort_header_tool, samtools_sort, blockbuster, blockclust</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1009</td>
      <td>1381</td>
      <td>3.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>RNA-Seq analysis with AskOmics Interactive Tool</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-analysis-with-askomics-it/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2020-07-09</td>
      <td>2023-11-09</td>
      <td>13</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2724331</td>
      <td>7315795</td>
      <td>6.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Introduction to Transcriptomics</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2016-10-05</td>
      <td>2025-04-08</td>
      <td>3</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2723343</td>
      <td>7314529</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Visualization of RNA-Seq results with CummeRbund</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-viz-with-cummerbund/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-10-16</td>
      <td>2023-11-09</td>
      <td>16</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2725856</td>
      <td>7318337</td>
      <td>4.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Visualization of RNA-Seq results with Volcano Plot in R</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-viz-with-volcanoplot-r/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2021-06-14</td>
      <td>2024-06-14</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>volcanoplot, interactive_tool_rstudio</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>30749</td>
      <td>42031</td>
      <td>1.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Reference-based RNAseq data analysis (long)</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rb-rnaseq/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2017-02-22</td>
      <td>2023-02-17</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>21863</td>
      <td>34304</td>
      <td>2.183333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>RNA-seq Alignment with STAR</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-bash-star-align/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-05-15</td>
      <td>2023-11-09</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>10281</td>
      <td>12603</td>
      <td>1.450000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>RNA Seq Counts to Viz in R</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/rna-seq-counts-to-viz-in-r/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2019-10-08</td>
      <td>2024-10-15</td>
      <td>21</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>interactive_tool_rstudio</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>4193</td>
      <td>5378</td>
      <td>1.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Reference-based RNA-Seq data analysis</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/ref-based/tutorial.html</td>
      <td></td>
      <td>Data handling, Read pre-processing, Primer removal, Pathway or network analysis, Read summarisation, Enrichment analysis, RNA-Seq quantification, Validation, Pathway or network visualisation, Gene functional annotation, RNA-Seq analysis, Sequence alignment, Formatting, Read mapping, Sequencing quality control, Exonic splicing enhancer prediction, Visualisation, Sequence trimming, Differential gene expression analysis</td>
      <td>2016-10-05</td>
      <td>2025-06-16</td>
      <td>103</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>join1, pathview, multiqc, rseqc_read_distribution, __MERGE_COLLECTION__, length_and_gc_content, samtools_view, falco, table_compute, samtools_idxstats, __TAG_FROM_FILE__, Cut1, cutadapt, tp_replace_in_line, pygenomeTracks, Filter1, ChangeCase, picard_MarkDuplicates, __FLATTEN__, tp_grep_tool, deg_annotate, rseqc_geneBody_coverage, dexseq_count, cat1, deseq2, tp_cat, tp_sort_header_tool, goseq, Add_a_column1, featurecounts, Grouping1, gtftobed12, rna_star, __FILTER_FROM_FILE__, rseqc_infer_experiment, collection_element_identifiers, dexseq, ggplot2_heatmap2, tp_tail_tool, __EXTRACT_DATASET__</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>180911</td>
      <td>342073</td>
      <td>2.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>Pathway analysis with the MINERVA Platform</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/minerva-pathways/tutorial.html</td>
      <td></td>
      <td>Gene functional annotation, Data handling, Read pre-processing, Primer removal, Statistical calculation, RNA-Seq analysis, Read summarisation, Sequence alignment, Annotation, Sequence composition calculation, Sequencing quality control, RNA-Seq quantification, Sequence trimming, Validation</td>
      <td>2024-03-28</td>
      <td>2024-07-26</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>join1, tp_sorted_uniq, fastqc, Grep1, multiqc, rseqc_read_distribution, annotatemyids, Cut1, cutadapt, tp_replace_in_line, hisat2, fasterq_dump, rseqc_geneBody_coverage, tp_sort_header_tool, goseq, Add_a_column1, featurecounts, limma_voom, collection_column_join, __EXTRACT_DATASET__</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>548</td>
      <td>660</td>
      <td>1.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Transcriptomics</td>
      <td>De novo transcriptome reconstruction with RNA-Seq</td>
      <td>https://training.galaxyproject.org//topics/transcriptomics/tutorials/de-novo/tutorial.html</td>
      <td></td>
      <td>Statistical calculation, RNA-Seq analysis, Sequence annotation, Sequence alignment, Read summarisation, Sequence composition calculation, Sequencing quality control, RNA-Seq quantification, Differential gene expression analysis, Transcriptome assembly</td>
      <td>2017-02-19</td>
      <td>2024-06-14</td>
      <td>36</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>gffcompare, Filter1, trimmomatic, hisat2, fastqc, stringtie, deeptools_bam_coverage, deseq2, stringtie_merge, featurecounts</td>
      <td>Galaxy@AuBi, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>16411</td>
      <td>28387</td>
      <td>2.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Introduction to Variant analysis</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2016-08-19</td>
      <td>2023-03-05</td>
      <td>2</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2723343</td>
      <td>7314529</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Calling very rare variants</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/dunovo/tutorial.html</td>
      <td></td>
      <td>Genome indexing, Variant calling, Read mapping, Sequence alignment, Statistical calculation, Generation, Sequence composition calculation, Sequencing quality control</td>
      <td>2017-02-22</td>
      <td>2024-12-09</td>
      <td>21</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bamleftalign, fastqc, allele_counts_1, align_families, bwa_mem, dunovo, correct_barcodes, sequence_content_trimmer, naive_variant_caller, make_families, Filter1</td>
      <td>UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1917</td>
      <td>3018</td>
      <td>1.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Calling variants in diploid systems</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/dip/tutorial.html</td>
      <td></td>
      <td>Statistical calculation, Variant calling, Genetic variation analysis, Sequence analysis</td>
      <td>2017-02-20</td>
      <td>2025-04-28</td>
      <td>17</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>gemini_db_info, freebayes, snpEff, vcfallelicprimitives, gemini_query, gemini_load</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>UseGalaxy.be, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>15080</td>
      <td>25992</td>
      <td>3.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Calling variants in non-diploid systems</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/non-dip/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Sequence alignment analysis, Data handling, Variant calling, Statistical calculation, Sequence alignment, Formatting, Sequence composition calculation, Sequencing quality control, Generation, Genome indexing, Read mapping</td>
      <td>2017-02-16</td>
      <td>2024-03-15</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Cut1, bamleftalign, picard_MarkDuplicates, freebayes, fastqc, vcffilter2, picard_MergeSamFiles, vcf2tsv, bwa_mem, bamFilter</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>8394</td>
      <td>15230</td>
      <td>2.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Somatic Variant Discovery from WES Data Using Control-FREEC</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/somatic-variant-discovery/tutorial.html</td>
      <td></td>
      <td>Sequence visualisation, Genome alignment, Variant calling, Statistical calculation, Sequence alignment, Genome indexing, Sequence composition calculation, Sequencing quality control, Generation, Read mapping, Visualisation, Validation, Copy number estimation</td>
      <td>2022-10-05</td>
      <td>2025-03-20</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bamleftalign, control_freec, trimmomatic, fastqc, Grep1, __RELABEL_FROM_FILE__, __BUILD_LIST__, samtools_rmdup, circos, samtools_calmd, multiqc, tp_replace_in_line, bwa_mem, samtools_view, falco, tp_text_file_with_recurring_lines, __EXTRACT_DATASET__</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1792</td>
      <td>2308</td>
      <td>2.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Exome sequencing data analysis for diagnosing a genetic disease</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/exome-seq/tutorial.html</td>
      <td></td>
      <td>Genetic variation analysis, Variant calling, Statistical calculation, Sequence alignment, Genome indexing, Sequence composition calculation, Sequencing quality control, Generation, Read mapping, Sequence analysis, Validation</td>
      <td>2016-08-19</td>
      <td>2025-05-11</td>
      <td>29</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>bcftools_norm, gemini_load, freebayes, trimmomatic, fastqc, snpEff, multiqc, samtools_view, bwa_mem, gemini_inheritance, snpEff_download, samtools_rmdup</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.be, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>34150</td>
      <td>62464</td>
      <td>2.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Mapping and molecular identification of phenotype-causing mutations</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/mapping-by-sequencing/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2018-03-07</td>
      <td>2023-11-09</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mimodd_varextract, mimodd_varreport, mimodd_varcall, snpEff, tp_find_and_replace, mimodd_map, snpEff_download, mimodd_vcf_filter</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>4260</td>
      <td>6286</td>
      <td>2.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Trio Analysis using Synthetic Datasets from RD-Connect GPAP</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/trio-analysis/tutorial.html</td>
      <td></td>
      <td>Genetic variation analysis, Sequence analysis</td>
      <td>2022-05-19</td>
      <td>2024-09-17</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>bcftools_norm, tp_grep_tool, bcftools_merge, pyega3, CONVERTER_uncompressed_to_gz, snpEff, regexColumn1, CONVERTER_gz_to_uncompressed, gemini_inheritance, gemini_load, gene_iobio_display_generation_iframe, Filter1</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1878</td>
      <td>3360</td>
      <td>3.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Deciphering Virus Populations - Single Nucleotide Variants (SNVs) and Specificities in Baculovirus Isolates</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/baculovirus-isolate-variation/tutorial.html</td>
      <td></td>
      <td>Data handling, Read pre-processing, Primer removal, Sequence alignment, Genome indexing, Generation, Read mapping, Visualisation, Sequence trimming</td>
      <td>2025-02-24</td>
      <td>2025-02-24</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>ncbi_acc_download, bcftools_call, collapse_dataset, bcftools_mpileup, fastq_dump, trim_galore, vcf2tsv, tp_replace_in_column, tp_replace_in_line, bwa_mem, tp_awk_tool, ggplot2_point, Filter1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>214</td>
      <td>298</td>
      <td>0.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Identification of somatic and germline variants from tumor and normal sample pairs</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/somatic-variants/tutorial.html</td>
      <td></td>
      <td>Sequence alignment analysis, Data handling, Variant calling, Statistical calculation, Sequence alignment, Genome indexing, Sequence composition calculation, Sequencing quality control, Generation, Read mapping, Visualisation, Validation</td>
      <td>2019-03-13</td>
      <td>2025-03-20</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>gemini_annotate, bamleftalign, gemini_load, tp_easyjoin_tool, trimmomatic, fastqc, snpEff, multiqc, samtools_rmdup, samtools_calmd, varscan_somatic, bwa_mem, gemini_query, falco, bg_column_arrange_by_header, bamFilter</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>13908</td>
      <td>25155</td>
      <td>2.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Querying the University of Bradford GDC Beacon Database for Copy Number Variants (CNVs)</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/beacon_cnv_query/tutorial.html</td>
      <td></td>
      <td>Genetic variation analysis, Service discovery, Database search</td>
      <td>2024-08-06</td>
      <td>2024-09-27</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>beacon2_cnv</td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>153</td>
      <td>189</td>
      <td>2.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Working with Beacon V2: A Comprehensive Guide to Creating, Uploading, and Searching for Variants with Beacons</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/beaconise_1000hg/tutorial.html</td>
      <td></td>
      <td>Genetic variation analysis, Service discovery, Database search</td>
      <td>2024-08-06</td>
      <td>2024-09-27</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>cnv_vcf2json, beacon2_cnv, cnv_phenopacket, beacon2_import</td>
      <td></td>
      <td>UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>271</td>
      <td>307</td>
      <td>0.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Visualisation</td>
      <td>Ploting a Microbial Genome with Circos</td>
      <td>https://training.galaxyproject.org//topics/visualisation/tutorials/circos-microbial/tutorial.html</td>
      <td></td>
      <td>Sequence visualisation</td>
      <td>2023-11-08</td>
      <td>2024-09-17</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>circos_interval_to_tile, Cut1, circos_wiggle_to_scatter, deeptools_bam_coverage, gff2bed1, circos, circos_gc_skew, circos_interval_to_text</td>
      <td>UseGalaxy.cz, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, MISSISSIPPI, UseGalaxy.be, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1333</td>
      <td>1690</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Visualisation</td>
      <td>Visualisation with Circos</td>
      <td>https://training.galaxyproject.org//topics/visualisation/tutorials/circos/tutorial.html</td>
      <td></td>
      <td>Sequence visualisation</td>
      <td>2020-01-10</td>
      <td>2024-10-15</td>
      <td>22</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>Cut1, Remove beginning1, Grep1, circos, upload1, random_lines1</td>
      <td>Galaxy@AuBi, Galaxy@Pasteur, MISSISSIPPI, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2753208</td>
      <td>7362332</td>
      <td>4.183333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Visualisation</td>
      <td>Friends Don't Let Friends Make Bad Graphs</td>
      <td>https://training.galaxyproject.org//topics/visualisation/tutorials/bad-graphs/slides.html</td>
      <td></td>
      <td></td>
      <td>2023-11-20</td>
      <td>2023-11-20</td>
      <td>2</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>1081111</td>
      <td>2596227</td>
      <td>2.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Visualisation</td>
      <td>Visualisations in Galaxy</td>
      <td>https://training.galaxyproject.org//topics/visualisation/tutorials/introduction/slides.html</td>
      <td></td>
      <td></td>
      <td>2020-01-10</td>
      <td>2023-01-12</td>
      <td>1</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2723343</td>
      <td>7314529</td>
      <td>2.516667</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

</div>